\documentclass[10pt]{article}

% DOCUMENT LAYOUT
%\usepackage{fullpage}
%\usepackage[cm]{fullpage}
%\usepackage[letterpaper, top=0.5in, bottom=0.5in, left=0.5in, right=0.5in]{geometry} 
%\usepackage[letterpaper]{geometry} 
%\geometry{textwidth=5.7in, textheight=9.0in, marginparsep=7pt, marginparwidth=.6in}
%\geometry{textwidth=6.0in, textheight=9.8in, marginparsep=7pt, marginparwidth=.8in}
%\geometry{textwidth=6in, textheight=9.8in, marginparsep=7pt, marginparwidth=.8in}
\setlength\parindent{0in}

% Dynamically control baselinestretch
\usepackage{setspace}

% Local hyphenation control
\usepackage{hyphenat}

% highlighting
\usepackage{color,soul}

%\addtolength{\oddsidemargin}{-.500in}
%\addtolength{\evensidemargin}{-.500in}
%\addtolength{\textwidth}{2.00in}

%\addtolength{\topmargin}{-.875in}
%\addtolength{\textheight}{1.75in}

\addtolength{\topmargin}{-1.125in}
\addtolength{\textheight}{2.0in}

\addtolength{\oddsidemargin}{-.375in}
\addtolength{\evensidemargin}{-.375in}
\addtolength{\textwidth}{1.75in}
\addtolength{\marginparsep}{-10pt}
\addtolength{\marginparwidth}{+15pt}

\usepackage{wrapfig}

\usepackage{chngpage}
\usepackage[export]{adjustbox}

\usepackage{sidecap}
\usepackage[abs]{overpic}
\usepackage{wrapfig}

% SPECIAL FORMATTING
% ---- CUSTOM AMPERSAND
\newcommand{\amper}{{\selectfont\itshape\&}}
% ---- MARGIN YEARS
%\newcommand{\years}[1]{\marginpar{\scriptsize #1}}
\newcommand{\years}[1]{\marginpar{\quad \small #1}}
% ---- TALK COUNTER
\newcounter{talknumber}
\setcounter{talknumber}{1}
% ---- ARTICLE COUNTER
\newcounter{articlenumber}
\setcounter{articlenumber}{1}
%\newcommand{\article}{\noindent\marginpar{\scriptsize \arabic{articlenumber}}\addtocounter{articlenumber}{1}}
\newcommand{\article}[2]{
\noindent\marginpar{
  \parbox[c]{0.25in}{\scriptsize \arabic{articlenumber}} 
  \parbox[c]{0.25in}{\scriptsize \href{#2}{#1}}
}\addtocounter{articlenumber}{1}}
% PAPER DESCRIPTION AND THUMBNAIL IMAGE
\newcommand{\desc}[2]{
\noindent\marginpar{
  \parbox[c]{0.50in}{{\includegraphics[width=0.9in]{thumbnails/#1}}}
}{\footnotesize\emph{#2}}}
% NEW ARTICLE STYLE WITH IMAGE AND NO NUMBERS
% #1 : thumbnail filename
% #2 : author list
% #3 : article title
% #4 : journal reference
% #5 : DOI, bioR$\chi$v, or ar$\chi$iv
% #6 : DOI, bioR$\chi$v, or arXiv link
% #7 : blurb / description
\newcommand{\newarticle}[7]{
\begin{adjustwidth}{-1in}{-1in}  
\begin{tabular}{p{0.9in}p{7in}}
%\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/{#1}}} & \parbox[c]{6in}{\setstretch{0.9} {\small #4} $\cdot$ \href{#6}{#5} \\ {\scriptsize {#2}} \\ \raggedright { \bf\nohyphens{#3}}  \\ {\footnotesize\emph {#7}}} % authors too small
%\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/#1}} & \parbox[c]{6in}{\setstretch{0.9}  {\footnotesize {#2}} \\ \raggedright { \bf\nohyphens{\href{#6}{#3}}}  \\ {\footnotesize\emph {#7}}} % no journal names
\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/#1}} & \parbox[c]{6in}{\setstretch{0.9} {\small #4} $\cdot$ \href{#6}{#5} \\ {\footnotesize {#2}} \\ \raggedright { \bf\nohyphens{#3}}  \\ {\footnotesize\emph {#7}}} % big authors
%{} & \parbox[c]{6in}{\setstretch{0.9} {\small #4} $\cdot$ \href{#6}{#5} \\ {\footnotesize {#2}} \\ \raggedright { \bf\nohyphens{#3}}  \\ {\footnotesize\emph {#7}}} % no images
\end{tabular}
\end{adjustwidth}
\vspace{0.2in}
}

% COLLABORATION
% #1 : thumbnail filename
% #2 : name
% #3 : dates
% #4 : url
% #5 : blurb / description
\newcommand{\collaboration}[5]{
\begin{adjustwidth}{-1in}{-1in}  
\begin{tabular}{p{0.9in}p{7in}}
%\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/{#1}}} & \parbox[c]{6in}{\setstretch{0.9} {\small #4} $\cdot$ \href{#6}{#5} \\ {\scriptsize {#2}} \\ \raggedright { \bf\nohyphens{#3}}  \\ {\footnotesize\emph {#7}}} % authors too small
%\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/#1}} & \parbox[c]{6in}{\setstretch{0.9}{ \raggedright { \bf\nohyphens{#2}} \\  {\small #3} $\cdot$ \href{#4}{#4} \\ {\footnotesize\emph {#5}} }} % big authors
%\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/#1}} & \parbox[c]{6in}{\setstretch{0.9}  {\footnotesize {#2}} \\ \raggedright { \bf\nohyphens{\href{#6}{#3}}}  \\ {\footnotesize\emph {#7}}} % no journal names
\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/#1}} & \parbox[c]{6in}{\setstretch{0.9}{ \small \raggedright { \bf\nohyphens{#2}} ({#3}) $\cdot$ \href{#4}{#4} \\ {\footnotesize\emph {#5}} }} % big authors
%{} & \parbox[c]{6in}{\setstretch{0.9}{ \small \raggedright { \bf\nohyphens{#2}} ({#3}) $\cdot$ \href{#4}{#4} \\ {\footnotesize\emph {#5}} }} % no images
\end{tabular}
\end{adjustwidth}
\vspace{0.2in}
}

% no images
%\newcommand{\newarticle}[7]{
%\begin{adjustwidth}{-1in}{-1in}  
%\begin{tabular}{p{0.9in}p{7in}}
%%\parbox[c]{0.9in}{\includegraphics[width=0.9in]{thumbnails/{#1}}} & \parbox[c]{6in}{\setstretch{0.9} {\small #4} $\cdot$ \href{#6}{#5} \\ {\scriptsize {#2}} \\ \raggedright { \bf\nohyphens{#3}}  \\ {\footnotesize\emph {#7}}} % authors too small
%\parbox[c]{0.9in}{} & \parbox[c]{6in}{\setstretch{0.9} {\small #4} $\cdot$ \href{#6}{#5} \\ {\footnotesize {#2}} \\ \raggedright { \bf\nohyphens{#3}}  \\ {\footnotesize\emph {#7}}}
%\end{tabular}
%\end{adjustwidth}
%\vspace{0.2in}
%}



% ---- TALKS
\newcommand{\talk}[2]{
\noindent\marginpar{
   \scriptsize \scshape
  #2 \\ #1
}}
\newcommand{\numberedtalk}[2]{
\noindent\marginpar{
  \parbox[c]{0.25in}{\scriptsize \arabic{talknumber}} 
% \parbox[c]{0.5in}{\scriptsize {#2}}
  \parbox[c]{0.5in}{\scriptsize \scshape {#1}}
}\addtocounter{talknumber}{1}}

\newcommand{\newtalk}[3]{
\noindent\marginpar{
   \scriptsize \scshape
  #2 \\ #1
}\spaceTalk\\}

% DEBUG
%\renewcommand{\talk}[2]{
%\textbf{\arabic{talknumber}} $\cdot$  \emph{#1} $\cdot$ \emph{#2}\\
%\addtocounter{talknumber}{1}}

% SPACING
\newcommand{\spaceEd}{\vspace{1ex}}
\newcommand{\spaceRes}{\vspace{1ex}}
\newcommand{\spaceFell}{\vspace{0.5ex}}
\newcommand{\spacePub}{\vspace{0.5ex}}
\newcommand{\spaceTalk}{\vspace{1ex}}
\newcommand{\spaceSoc}{\vspace{0.5ex}}
\newcommand{\spacePost}{\vspace{0.5ex}}

\newcommand{\jdc}{ {\bf \hl{Chodera~JD}} } % highlight
%\newcommand{\jdc}{\underline{\bf Chodera~JD}} % underline

% Modify section spacing
%\usepackage{titlesec}
%\titlespacing*{\section}{0pt}{0.2\baselineskip}{\baselineskip}
%\titlespacing*{\subsection}{0pt}{0.2\baselineskip}{\baselineskip}
%\titlespacing*{\subsubsection}{0pt}{0.2\baselineskip}{\baselineskip}

% GRAPHICS
\usepackage{graphicx}

% HEADINGS
\usepackage{sectsty} 
\usepackage[normalem]{ulem} 

\usepackage{fancyhdr}
\renewcommand{\headrulewidth}{0pt}
\pagestyle{fancy}
\fancyhf{}
%\fancyfoot[LE,CO]{{\small\textsc{John D. Chodera - Sloan Kettering Institute - \thepage}}}%
\rfoot{\small\textsc{John D. Chodera - Sloan Kettering Institute - \thepage}}

% Clean up font spacing with micro type
\usepackage{microtype}

% Use a clean, modern-looking font
\usepackage[default]{lato}
%\usepackage[default]{raleway}
%\usepackage{avant}
%\renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
%\usepackage[sfdefault,light]{roboto}  %% Option 'sfdefault' only if the base font of the document is to be sans serif
\usepackage[T1]{fontenc}

% Change the appearance of section and subsection headings
\sectionfont{\mdseries\large\underline} 
\subsectionfont{\bfseries\scshape\normalsize} 
\subsubsectionfont{\bfseries\upshape\normalsize} 

% Control tightness of spacing
\renewcommand{\baselinestretch}{1.1}

% PDF SETUP
% ---- FILL IN HERE THE DOC TITLE AND AUTHOR
\usepackage[bookmarks, colorlinks, breaklinks, pdftitle={John D. Chodera - vita},pdfauthor={John D. Chodera}]{hyperref}  
\hypersetup{linkcolor=blue,citecolor=blue,filecolor=black,urlcolor=blue} 

% Change the typewriter font to something nicer.
\usepackage[ttdefault]{sourcecodepro}

% DOCUMENT
\begin{document}
\reversemarginpar
% FILL IN NAME HERE
{\fontseries{eb}\selectfont \LARGE John D. Chodera}\\[1cm]

\vspace{-0.5in}

% FILL IN AUTHOR INFORMATION HERE
\begin{minipage}[t]{2.5in}
\includegraphics[width=2.5in,valign=c]{images/john_chodera_wide.jpg} % commented out for MSKCC
\end{minipage}
\quad
\begin{minipage}[t]{3in}
\small
\begin{tabular}{rl}
{\bf url} & \href{http://www.choderalab.org}{http://www.choderalab.org}\\[0.05in]
{\bf email} & \href{mailto:choderaj@mskcc.org}{choderaj@mskcc.org}\\[0.05in] % for MSKCC
%{\bf email} & \href{mailto:john.chodera@choderalab.org}{john.chodera@choderalab.org}\\[0.05in] 
{\bf github} & \href{https://github.com/choderalab}{https://github.com/choderalab}\\[0.05in]
{\bf ORCID iD} & \href{http://orcid.org/0000-0003-0542-119X}{0000-0003-0542-119X}\\[0.05in]
{\bf twitter} & \href{http://twitter.com/jchodera}{@jchodera}\\[0.05in]
{\bf mobile} & +1.415.867.7384\\[0.05in]
%{\bf fax} & 646.422.0717 \\[0.05in]
%{\bf mobile} & 646.737.3319\\[0.05in]
{\bf post} & 
\parbox[t]{3.0in}{%Memorial Sloan Kettering Cancer Center\\
1275 York Ave, ZRC 6-South\\
New York, NY 10065}
\end{tabular}
\end{minipage}

% RESEARCH INTERESTS
%\section*{Research interests}
%\noindent {\bf Computational biophysics}, especially \\
%\noindent {\bf Single-molecule biophysics}, including new techniques for interpreting force microscopy \\
%\noindent {\bf Nonequilibrium statistical mechanics}, as tools for single-molecule experimental probes and the design of new algorithms for computational chemistry \\



\setstretch{1.05} 

\vspace{-0.15in}
\section*{Education and positions}
\vspace{-0.15in}

\small

\noindent\years{2023--}{\bf Member, Memorial Sloan-Kettering Cancer Center}\\  % effective date 6/23/2023
\noindent\years{2019--2023}{\bf Associate Member, Memorial Sloan-Kettering Cancer Center}\\  % effective date 1/25/2019
\noindent\years{2019--}{\bf BIH Einstein Visiting Professor, Charit\'{e}, Berlin}\\
\noindent\years{2013--}{\bf Faculty, Physiology, Biophysics, and Systems Biology Program,\\
Weill Cornell Graduate School of Medical Sciences}\\
\noindent\years{2012--2019}{\bf Assistant Member, Memorial Sloan-Kettering Cancer Center}\\
\noindent\years{2008--2012}{\bf Independent Distinguished Postdoctoral Fellow, California Institute for Quantitative Biosciences (QB3),\\
University of California, Berkeley}\\
Independent research funding, sponsors \href{http://www.cchem.berkeley.edu/plggrp/index.html}{Phillip L. Geissler} and \href{http://zebra.berkeley.edu/}{Susan Marqusee} \\
%\noindent\years{2011--2012}{\bf \href{http://research.google.com/university/exacycle_program.html}{Google Exacycle Visiting Faculty}} \\
\noindent\years{2007--2008}{\bf Postdoctoral researcher, Department of Chemistry, Stanford University}\\
With \href{https://pande.stanford.edu/}{Vijay S. Pande} (head of \href{http://folding.stanford.edu}{Folding@Home} distributed computing project) \\
\noindent\years{1999--2006}{\bf \textsc{Ph.D.} in Biophysics, University of California, San Francisco}\\
%Dissertation: \href{http://www.dillgroup.ucsf.edu/~jchodera/pubs/pdf/jdcthesis.pdf}{\emph{Master equation models of macromolecular dynamics from atomistic simulation.}} \\
Committee: \href{http://www.dillgroup.org}{Ken A. Dill}, \href{http://jacobsonlab.org/}{Matthew P. Jacobson}, \href{https://pande.stanford.edu/}{Vijay S. Pande}\\
\noindent\years{1995--1999}{\bf \textsc{B.S.} in Biology, California Institute of Technology}\\
Undergraduate research with \href{http://www.its.caltech.edu/~phplab/phplab.html}{Paul H.~Patterson} (\emph{molecular neurobiology}) and Jerry E.~Solomon (\emph{computational chemistry})

\vspace{-0.15in}
\section*{Fellowships and awards}
\vspace{-0.15in}

\noindent\years{2022--2024}Einstein Visiting Fellowship\\
\noindent\years{2019--2022}BIH Einstein Visiting Fellowship\\
\noindent\years{2017}Silicon Therapeutics Open Science Fellowship\\
\noindent\years{2013--2016}Louis V.~Gerstner Young Investigator Award\\
\noindent\years{2013--2014}Google Exacycle for External Faculty\\
\noindent\years{2008--2012}QB3-Berkeley Distinguished Postdoctoral Fellowship, University of California, Berkeley\\
%\noindent\years{2008}Berkeley Mini Stat Mech Meeting Best Poster Prize (second place)\\
\noindent\years{2005--2006}IBM Predoctoral Fellowship\\
%\noindent\years{2005}Frank M. Goyan Award for outstanding work in physical chemistry, University of California, San Francisco\\
\noindent\years{2000--2005}Howard Hughes Medical Institute Predoctoral Fellowship
%\noindent\years{1998--1999}Caltech Upperclass Merit Award Scholarship\\
%\noindent\years{1997,$\:$1998}Caltech Summer Undergraduate Research Fellowships


\vspace{-0.15in}
\section*{Research overview}
\vspace{-0.15in}

%{\bf My research aims to reimagine the way we discover small molecule drugs and pair therapeutics with individual patient tumors by developing predictive physical and structure-informed machine learning models for the cancer genomics era.} 
%By combining novel algorithmic advances to achieve orders-of-magnitude efficiency gains with powerful but inexpensive GPU hardware, machine learning, and distributed computing technologies, my lab is developing next-generation approaches and open source software for predicting small molecule binding affinities, designing small molecules with desired properties, predicting drug sensitivity or resistance of clinical mutations, and understanding the detailed structural mechanisms underlying oncogenic mutations.
%Using automated biophysical measurements, my laboratory collects new experimental data targeted to advance the quantitative accuracy of our methodologies and gather new insight into drug susceptibility and resistance in kinases and other cancer targets.
%My work makes extensive use of scalable Bayesian statistical inference and probabilistic programming, structure-based machine learning, and information theoretic principles for designing optimal experiments.
%
%\vfill
%
%{\bf A major current focus of the lab is the development of technologies required to enable fully autonomous decisionmaking in drug discovery.}
%Such technologies empower medicinal chemists to manage a portfolio of correlated drug discovery programs without the need to make decisions about which molecules to synthesize, enabling them to focus on objectives and strategies instead.
%This includes the development of automated, modular decisionmaking frameworks inspired by reinforcement learning; integration of automated synthetic route planning; new architectures for fast structure-enabled machine learning; rearchitecting alchemical free energy calculations for differentiable machine learning frameworks to introduce learnability; and scalable Bayesian inference frameworks for automated experimental design and interpretation.
%
%\vfill
%
%{\bf I am passionate about open science}, disseminating scientific best practices, and maximizing research reproducibility.
%
%\vfill
%
%{\bf I focus on training the next generation of interdisciplinary scientists} equipped to tackle the most challenging problems in health and human disease for industrial drug discovery.

%We co-develop the \href{http://openmm.org}{OpenMM} GPU-powered molecular simulation framework, which powers numerous biomolecular modeling and simulation applications using physical modeling and machine learning. 
%As a core member of the \href{https://foldingathome.stanford.edu/about/the-foldinghome-consortium/}{Folding@home Consortium}, my lab harnesses the largest computing platform in the world---the first to reach an exaFLOP/s---pooling the efforts of a million volunteers around the world to study functional implications of mutations and new opportunities for therapeutic design against cancer targets and global pandemics.
%We are also a core member of the \href{http://openforcefield.org}{Open Force Field Consortium}, an industry-funded open effort to develop better force fields for drug discovery and biomolecular modeling.
%I am a co-founder of the \href{http://postera.ai/covid}{COVID Moonshot}, a radical open science patent-free drug discovery effort aiming to develop an inexpensive small molecule therapy effective against COVID-19 and future coronavirus.
%I co-founded the \href{http://openforcefield.org}{Open Force Field Consortium}, an academic-industry collaboration to drive improvements in biomolecular forcefields with open code, open data, and open science.

\textbf{My lab is developing the infrastructure to enable fully autonomous small molecule drug discovery powered by next-generation hybrid physical/machine learning models to predict potency, selectivity, and resistance in a data-efficient manner.}
We develop advanced predictive computational methodologies for drug discovery in frontier areas where these models are lacking, working with industry collaborators to deploy those solutions in real drug discovery programs.
We build open source tools---such as \href{http://openmm.org}{OpenMM}, which has been downloaded over 1.2 million times, run on millions of machines, and powers advances such as AlphaFold---that enable these algorithms to have impact across the drug discovery and biomolecular modeling fields.
We create open science communities---such as (1) the \href{http://openforcefield.org}{Open Force Field Initiative}, (2) the \href{http://postera.ai/covid}{COVID Moonshot} (which delivered a patent-free COVID antiviral preclinical candidate into an IND-enabling preclinical program funded by the Wellcome Trust), (3) the \href{https://foldingathome.org/about-2/the-foldinghome-consortium/?lng=en}{Folding@home Consortium} which harnesses hundreds of thousands of volunteer computers around the world to accelerate biomolecular simulation and discovery, and (4) the \href{http://asapdiscovery.org}{AI-driven Structure-enabled Antiviral Program} that received \$68M in initial funding from the NIH for open science antiviral drug discovery.
We apply these tools to cancer, where we develop models for (1) designing small molecule kinase inhibitors with targeted polypharmacology, (2) predicting drug sensitivity or resistance of clinical mutations, and (3) understanding the detailed structural mechanisms underlying oncogenic mutations.
And we train the next generation of interdisciplinary scientists equipped to tackle the most challenging problems in health and human disease for industrial drug discovery.

\setstretch{1.00} 

\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% MAJOR COLLABORATIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Major Research Collaborations}

%Collaborative research projects can accomplish things together that no individual laboratory can in isolation.\\
%I lead or participate in several major collaborative research efforts:\\

\setstretch{1.05} 

\collaboration{covid-moonshot-logo-small}{The COVID Moonshot}{2020--}{https://covid.postera.ai/covid}{I co-founded the {\bf COVID Moonshot}, a global open science collaboration leveraging our free alchemical energy calculations on Folding@home to discovery a patent-free oral antiviral useful against COVID-19. Starting from an X-ray fragment screen, we produced SARS-CoV-2 main viral protease (Mpro) inhibitor preclinical candidates in just 18 months (\href{https://www.nature.com/articles/d41586-021-01571-1}{Nature Comment}). Together with the \href{http://dndi.org}{Drugs for Neglected Diseases initiative (DND$i$)}---the world's leading nonprofit developing new therapies for neglected diseases---we were \href{https://dndi.org/press-releases/2021/covid-moonshot-funded-by-wellcome-to-rapidly-develop-safe-globally-accessible-affordable-antiviral-pill/}{awarded \$11M from the WHO Access to COVID Tools Accelerator (ACT-A) via the Wellcome Trust} to carry out an accelerated preclinical program, with clinical trials expected in early 2024. Our antiviral is expected to be free of the drug-drug interactions that present difficulties to widespread use of Paxlovid, and has differentiated resistance liabilities. We are working to partner with generics manufacturers to make our antiviral available for globally equitable and affordable access.}

\collaboration{asap-logo-small}{AI-driven Structure-enabled Antiviral Platform (ASAP)}{2022--}{https://asapdiscovery.org}{Together with \href{http://dndi.org}{DNDi} and \href{http://postera.ai}{PostEra}, I lead the \href{http://asapdiscovery.org}{AI-driven Structure-enabled Antiviral Platform}, which \href{https://dndi.org/press-releases/2022/consortium-to-discover-antivirals-covid19-receives-nih-funding-develop-globally-accessible-treatments-pandemics/}{uses artificial intelligence and computational chemistry to accelerate structure-based open science antiviral drug discovery and deliver oral antivirals for pandemics with the goal of global, equitable, and affordable access}. I am the Contact PI on the \href{https://www.niaid.nih.gov/research/antiviral-drug-discovery-centers-pathogens-pandemic-concern}{NIH NIAID Antiviral Drug Discovery (AViDD) Center} that currently funds ASAP, currently funded by a \href{https://www.niaid.nih.gov/research/antiviral-drug-discovery-centers-pathogens-pandemic-concern-award-recipients}{\$68M NIH award for the first three-year phase of a five-year \$110M grant}. ASAP is actively pursuing additional funding, including a \href{https://www.sprind.org/en/challenges/transmission-blocking-antivirals/}{EUR22M SPRIND proposal} submitted in Mar 2023.}

\collaboration{foldingathome-logo-small}{Folding@home Consortium}{2012--}{http://foldingathome.org}{I am a founding member of the \href{https://foldingathome.org/about-2/the-foldinghome-consortium/}{Folding@home Consortium}, a worldwide network of laboratories that run the \href{http://foldingathome.org}{Folding@home global distributed computing project} where hundreds of thousands of volunteers around the world (mostly gamers with GPUs) contribute idle computing cycles to enable us to study molecular mechanisms of disease and search for new potential therapeutics. As part of the COVID Moonshot effort, \href{https://www.nature.com/articles/s41557-021-00707-0}{Folding@home became the world's first exascale distributed computing platform in 2020}. Using our alchemical free energy methodologies on Folding@home, we accelerated lead optimization for preclinical candidates for the COVID Moonshot that are expected to enter clinical trials in 2023.}

\collaboration{openmm7-logo}{OpenMM molecular simulation framework}{2006--}{http://openmm.org}{I co-lead the \href{https://openmm.org/development}{OpenMM Consortium}, which develops an open source GPU-accelerated molecular simulation package used both as a stand-alone tool for biomolecular simulation and as a library to enable other tools to use biomolecular simulation, such as \href{https://isolde.cimr.cam.ac.uk/}{Isolde} and \href{https://www.deepmind.com/research/highlighted-research/alphafold}{AlphaFold2}. OpenMM is the fastest and most widely deployed GPU-accelerated biomolecular simulation package globally, and has been downloaded over 1.2 million times. OpenMM is currently \href{https://openmm.org/development}{funded by the NIH} to effect a transformation in the biomolecular simulation community to enable the use of fast, accurate machine-learning force fields.}

\collaboration{openforcefield-logo-small}{Open Force Field Initiative and Consortium}{2019--}{http://openforcefield.org}{I co-founded the \href{https://openforcefield.org/about/organization/\#open-force-field-initiative}{Open Force Field Initiative} (an NIH-funded collaboration) and related \href{https://openforcefield.org/about/organization/\#open-force-field-consortium}{Open Force Field Consortium} (funded by over a dozen industry partners), a scientific collaboration consisting of dozens of scientists around the world working to develop modern open source infrastructure for building and applying high-quality biomolecular force fields for accelerating drug discovery. These force fields and tools are used by pharma and academic labs around the world, and recently aided the discovery of a new preclinical candidate by the COVID Moonshot.}

\collaboration{samplchallenges-logo-small}{Statistical Assessment of the Modeling of Proteins and Ligands (SAMPL)}{2012--}{https://samplchallenges.github.io}{I co-organize the \href{https://samplchallenges.github.io/}{SAMPL Challenges}, which are iterative blind predictive modeling challenges for the computational chemistry community aimed at accelerating the generation of solutions to major obstacles limiting the progress of computer-aided drug discovery, focusing the community on solving these challenges, providing critical experimental data to computational researchers that lack access to data generation capabilities, and emphasizing knowledge sharing. The learnings from these challenges have been reported in over \href{https://www.samplchallenges.org/history/references/}{14 special issues of the Journal for Computer-Aided Molecular Design}. In some areas, we have managed to move computational methods for wide disagreement among methods to tightly-grouped predictions that can identify when the experimental data is wrong.}

\collaboration{cache}{Critical Assessment of Computational Hit-finding Experiments (CACHE)}{2022--}{https://cache-challenge.org/}{I co-founded the \href{https://cache-challenge.org/}{CACHE Challenge}, which aims to be the "CASP for hit-finding in drug discovery" by providing a new generation of AI- and physics-driven approaches to assess and refine their hit identification approaches against experimental data. Each CACHE Challenge focuses on identifying new chemical matter against an orphan target protein of pharmaceutical relevance, and all experimental data and compounds are disclosed into the public domain to accelerate drug discovery against these targets.}

\collaboration{omsf-logo-small}{Open Molecular Software Foundation (OMSF)}{2021--}{http://openforcefield.org}{I work closely with the \href{http://omsf.io}{Open Molecular Software Foundation (OMSF)}, a 501(c)3 nonprofit organization that aims to be thoughtful stewards for the biomolecular modeling and computer-aided drug discovery open source software communities. Using a funding mechanism I helped pioneer, OMSF can act as a fiscal sponsor that works with open source software communities in the biomolecular sciences to aid them in becoming self-governing organizations that serve the community with fiscally sustainable support from industrial, federal, and philanthropic sources. OMSF currently hosts the \href{http://openforcefield.org}{Open Force Field Initiative and Consortium}, the \href{http://openfree.energy}{Open Free Energy Consortium}, and \href{https://openfold.io/}{OpenFold Consortium}.}

%\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% SIGNIFICANT PUBLICATIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\section*{Selected Significant Publications}
%
%{\footnotesize \underline{Underline} denotes Chodera lab trainee; $\ddag$ denotes co-corresponding authorship; \jdc highlighted for convenience.}\\
%
%{\bf Selected publications and preprints following promotion to Associate Member} \\
%
%\newarticle{covid-moonshot-logo-small}{\href{https://docs.google.com/spreadsheets/d/1OnPzj5IvYtD5uu6tqmlhvOX47__WgN1VrbzMH_kqns0/edit?pli=1#gid=0}{The COVID Moonshot Consortium} (includes trainees \underline{Bruce Macdonald HE}, \underline{Rufa D}, \underline{Glass WG}, \underline{Zhang I}), \jdc$\ddag$, Lee A$\ddag$, London N$\ddag$, von Delft F$\ddag$}{Open science discovery of oral non-covalent SARS-CoV-2 main protease inhibitors}{\emph{Preprint}}{bioRxiv}{https://doi.org/10.1101/2020.10.29.339317}{We report the discovery of a new oral antiviral non-covalent SARS-CoV-2 main protease inhibitor developed by the COVID Moonshot, a global open science collaboration leveraging free energy calculations on Folding@home and ML-accelerated synthesis planning, now in accelerated preclinical studies funded by an \$11M grant from the WHO ACT-A program via the Wellcome Trust. 
%We are currently in discussions with generics manufacturers about partnering with us throughout clinical trials (expected to begin in 2023) to ensure we can scale up production for global equitable and affordable access once approved by regulatory agencies.\\
%\underline{Contributions:} My lab led computer-aided drug discovery aspects of this project, and performed tens of thousands of free energy calculations to prioritize which compounds were synthesized.}
%
%\newarticle{qml-explicit-free-energy}{\underline{Wieder M}, \underline{Fass J}, and \jdc}{Teaching free energy calculations to learn from experimental data}{\emph{Preprint}}{bioRxiv}{https://doi.org/10.1101/2021.08.24.457513}{In this pioneering work, we show, for the first time, how we can use machine learning to systematically improve explicit solvent alchemical free energy calculations using fully machine learning force fields by fitting directly to experimental free energies. This is also the first work to show how to compute alchemical free energies in explicit solvent using fully machine learning force fields, and the first to show how this can also involve breaking and forming covalent bonds.\\
%\underline{Contributions:} My lab led all work.
%}
%
%\newarticle{hybrid-qm-mm.jpg}{\underline{Rufa DA}, \underline{Bruce Macdonald, HE}, \underline{Fass J}, \underline{Wieder M}, Grinaway PB, Roitberg AE, Isayev O, and \jdc}{Towards chemical accuracy for alchemical free energy calculations with hybrid physics-based machine learning / molecular mechanics potentials}{\emph{Preprint}}{bioR$\chi$iv}{https://doi.org/10.1101/2020.07.29.227959}{In this first use of hybrid machine learning / molecular mechanics (ML/MM) potentials for alchemical free energy calculations, we demonstrate how the improved modeling of intramolecular ligand energetics offered by the quantum machine learning potential ANI-2x can significantly improve the accuracy in predicting kinase inhibitor binding free energy by reducing the error from 0.97~kcal/mol to 0.47~kcal/mol, which could drastically reduce the number of compounds that must be synthesized in lead optimization campaigns for minimal additional computational cost.\\
%\underline{Contributions:} My lab led all work, consulting Adrian Roitberg and Olexandr Isayev on appropriate use of the ANI potentials in alchemical free energy calculations.
%}
%
%\newarticle{espaloma.jpg}{\underline{Wang Y}, \underline{Fass J}, and \jdc}{End-to-end differentiable molecular mechanics force field construction}{\emph{Chemical Science} 13:12016, 2022}{DOI}{https://doi.org/10.1039/D2SC02739A}{Molecular mechanics force fields have been a workhorse for computational chemistry and drug discovery. Here, we propose a new approach to force field parameterization in which graph convolutional networks are used to perceive chemical environments and assign molecular mechanics (MM) force field parameters. The entire process of chemical perception and parameter assignment is differentiable end-to-end with respect to model parameters, allowing new force fields to be easily constructed from MM or QM force fields, extended, and applied to arbitrary biomolecules.\\
%\underline{Contributions:} My lab led all work.
%}
%
%% new
%\newarticle{abl-mutant-kinetics}{Lyczek A, Tilman-Berger B, Rangwala AM, Paung Y, Tom J, Philipose H, \underline{Guo J}, \underline{Albanese SK}, Robers MB, Knapp S, \jdc, Seeliger MA}{Mutation in Abl kinase with altered drug binding kinetics indicates a novel mechanism of imatinib resistance}{\emph{Proceedings of the National Academy of Sciences} 118:e2111451118, 2021}{DOI}{https://doi.org/10.1073/pnas.2111451118}{Here, we characterize the biophysical mechanisms underlying mutants of Abl kinase associated with clinical drug resistance to targeted cancer therapies. 
%We uncover a surprising novel mechanism of mutational resistance to kinase inhibitor therapy in which the off-rate for inhibitor unbinding is increased without affecting inhibitor affinity.\\
%\underline{Contributions:} My lab curated the clinical cancer mutations, coordinated the production of initial protein constructs, and led all simulation work.}
%
%%\newarticle{foldingathome-covid-cryptic-pockets}{Zimmerman MI, Porter JR, Ward MD, Singh S, Vithani N, Meller A, Mallimadugula UL, Kuhn CE, Borowsky JH, \underline{Wiewiora RP}, Hurley MFD, Harbison AM, Fogarty CA, Coffland JE, Fadda E, Voelz VA, \jdc, Bowman GR}{SARS-CoV-2 simulations go exascale to predict dramatic spike opening and cryptic pockets across the proteome}{\emph{Nature Chemistry} 13:651, 2021}{DOI}{https://doi.org/10.1038/s41557-021-00707-0}{To accelerate a multitude of drug development activities to combat the global threat posed by COVID-19, over a million citizen scientists have banded together through the Folding@home distributed computing project to create the world’s first Exascale computer and simulate protein dynamics. 
%%An unprecedented 0.1 seconds of simulation of the viral proteome reveal how the spike complex uses conformational masking to evade an immune response, conformational changes implicated in the function of other viral proteins, and cryptic pockets that are absent in experimental structures. These structures and mechanistic insights present new targets for the design of therapeutics.\\
%%\underline{Contributions:} My lab contributed key software infrastructure to power this effort, and led the initial drive for volunteers to join our effort to study COVID.
%%}
%
%%\newarticle{vir-cell-xray.jpg}{Thompson EC, Rosen LE, Shepherd JG, Spreafico R, da Silva Filipe A, Wojcechowskyj JA, Davis C, Piccoli L, Pascall DJ, Dillen J, Lytras S, Czudnochowski N, Shah R, Meury M, Jesudason N, De Marco A, Li K, Bassi J, O'Toole A, Pinto D, Colquhoun RM, Culap K, Jackson B, Zatta F, Rambaut A, Jaconi S, Sreenu VB, Nix J, \underline{Zhang I}, Jarrett RF, \underline{Glass WG}, Beltramello M, Nomikou K, Pizzuto M, Tong L, Cameroni E, Cross TI, Johnson N, Di Iulio J, Wickenhagen A, Ceschi A, Harbison AM, Mair D, Ferrari P, Smollett K, Sallusto F, Carmichael S, Garzoni C, Nichols J, Galli M, Hughes J, Riva A, Ho A, Schiuma M, Semple MG, Openshaw PJM, Fadda E, Baillie JK, \jdc, Rihn SJ, Lycet SJ, Virgin HW, Telenti A, Corti D, Robertson DL, Snell J, ISARIC4C Investigators}{Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity}{\emph{Cell} 184:1171, 2021}{DOI}{https://doi.org/10.1016/j.cell.2021.01.037}{New mutations that enhance the affinity of SARS-CoV-2 spike protein for human ACE2---and potentially pose threats to antibody-based therapeutics and vaccines for COVID-19---are already emerging in the wild. We characterize and describe sentinel mutations of SARS-CoV-2 in the wild that herald challenges for combatting COVID-19, and use simulations of the RBD-ACE2 interface on Folding@home to biophysically characterize why these mutations can lead to enhanced affinity.\\
%%\underline{Contributions:} My lab led all the molecular simulation work reported in this paper.
%%}
%
%%{\bf Selected publications prior to promotion to Associate Member} \\
%%
%%\newarticle{setd8-msm}{\underline{Rafal P. Wiewiora}$^\dag$, Shi Chen$^\dag$, Fanwang Meng, Nicolas Babault, Anqi Ma, Wenyu Yu, Kun Qian, Hao Hu, Hua Zou, Junyi Wang, Shijie Fan, Gil Blum, Fabio Pittella-Silva, \underline{Kyle A. Beauchamp}, Wolfram Tempel, Hualing Jiang, Kaixian Chen, Robert Skene, Y. George Zheng, Peter J. Brown, Jian Jin, \jdc$^\ddag$, and Minkui Luo$^\ddag$}{The dynamic conformational landscapes of the protein methyltransferase SETD8}{\emph{eLife} 8:e45403, 2019}{DOI}{http://doi.org/10.7554/eLife.45403}{In this work, we show how targeted X-ray crystallography using covalent inhibitors and depletion of native ligands to reveal structures of low-population hidden conformations can be combined with massively distributed molecular simulation to resolve the functional dynamic landscape of the protein methyltransferase SETD8 in unprecedented atomistic detail. Using an aggregate of six milliseconds of fully atomistic simulation from Folding\@home, we use Markov state models to illuminate the conformational dynamics of this important epigenetic protein.\\
%%\underline{Contributions:} My lab led all the molecular simulation work reported in this paper.
%%}
%%
%%\newarticle{ddr1-dfg-pmf.jpg}{\underline{Hanson SM}$^*$, Georghiou G$^*$, Miller WT, Rest JS, \jdc$\ddag$, and Seeliger MA$\ddag$}{What makes a kinase promiscuous for inhibitors?}{\emph{Cell Chemical Biology} 26:390, 2019}{DOI}{https://doi.org/10.1016/j.chembiol.2018.11.005}{Using a combination of chemogenomics, structural biology, and molecular simulation approaches, we identify a set of human kinases that are especially promiscuous binders of small molecule kinase inhibitors, and show that a prototypical member of this class, DDR1, achieves this promiscuity by virtue of its more stable Asp-DFG-out conformation.
%%\underline{Contributions:} My lab led all the molecular simulation work reported in this paper.
%%}
%%
%%\newarticle{kinase-mutations}{Hauser K, Negron C, \underline{Albanese SK}, Ray S, Steinbrecher T, Abel R, and \jdc, and Wang L}{Predicting resistance of clinical Abl mutations to targeted kinase inhibitors using alchemical free-energy calculations}{\emph{Communications Biology} 1:70, 2018}{DOI}{https://doi.org/10.1038/s42003-018-0075-x}{The emergence of resistance mutations are a significant problem in precision cancer therapy. With cancer centers now routinely collecting tumor sequencing panels, it is apparent the vast majority of missense mutations in kinases---for which more than 36 small molecule inhibitors are available---are very rare, and no information will be available as to whether these mutations might reduce therapeutic efficacy for some inhibitors by inducing drug resistance. Here, we take the first steps toward using physical modeling to predict individualized therapeutic response, assessing how accurately the impact of clinical Abl mutations on inhibitor binding affinity can be predicted. Our results suggest free energy calculations are a promising tool for aiding therapeutic decisionmaking.\\
%%\underline{Contributions:} My lab conceived of this study, design the experiment, helped analyze the data, and helped write the paper.
%%}
%%
%%\newarticle{aurka-phosphorylation}{Ruff EF, Muretta JM, Thompson A, Lake E, Cyphers S, \underline{Albanese SK}, \underline{Hanson SM}, \underline{Behr JM}, Thomas DT, \jdc, and Levinson NM}{A dynamic mechanism for allosteric activation of Aurora kinase A by activation loop phosphorylation}{\emph{eLife} 7:e32766, 2018}{DOI}{https://doi.org/10.7554/eLife.32766}{Kinase phosphorylation is canonically believed to induce a conformational change of the DFG loop that borders the ATP binding site from out to in. Surprisingly, it seems that not all kinases behave this way. Through a combination of FRET, IR, and EPR labeling and large-scale molecular dynamics simulations, we show that phorphorylation activates Aurora kinase by a novel mechanism that does not simply correspond to a DFG-out to DFG-in population shift, but rather reorganization of DFG-in subpopulations. This paper follows our prior report in \emph{Nature Chemical Biology} on the role of Tpx2 in activating a cellular subpopulation of Aurora by another surprising mechanism.\\
%%\underline{Contributions:} My lab led all the molecular simulation work reported in this paper.
%%}
%%
%%%\newarticle{mtor-mutations.pdf}{Xu~J, Pham~CG, Albanese~SK, Dong~Y, Oyama~T, Lee~CH, Rodrik-Outmezguine~V, Yao~Z, Han~S, Chen~D, Parton~DL, \jdc, Rosen~N, Cheng~EH, and Hsieh~JJ}{Mechanistically distinct cancer-associated mTOR activation clusters predict sensitivity to rapamycin}{\emph{Journal of Clinical Investigation} 126:3529, 2016}{DOI}{https://doi.org/10.1172/JCI86120}{We use massively parallel distributed molecular simulations on Folding\@home to probe the mechanism activating mutations of the mTOR kinase identified in clinical populations.  {\color{red}Expand to paragraph.}}
%%
%%\newarticle{kinome-expression-tree}{\underline{Albanese SK}*, \underline{Parton DL}*, \underline{Isik M}$\dag$, \underline{Rodr\'{i}guez-Laureano L}$\dag$, \underline{Hanson SM}, Gradia S, Jeans C, Levinson NM, Seeliger M, and \jdc}{An open library of human kinase domain constructs for automated bacterial expression}{\emph{Biochemistry} 57:4675, 2018}{DOI}{http://doi.org/10.1021/acs.biochem.7b01081}{To establish a tractable experimental system for studying the biophysical determinants of selective kinase inhibitor resistance in clinical cancer mutations, we engineer a library of human kinase domains with useful bacterial expression with phosphatase coexpression. Over 350 requests have been made for these plasmids.\\
%%\underline{Contribtuions:} My lab designed the experiments, directed cloning and expression at the QB3 MacroLab (Gradia and Jeans), and performed the characterization experiments in our laboratory (with guidance from Levinson and Seliger).
%%}
%%
%%%\newarticle{ensembler.pdf}{Parton DL, Grinaway PB, Hanson SM, Beauchamp KA, and \jdc}{Ensembler: Enabling high-throughput molecular simulations at the superfamily scale}{\emph{PLoS Computational Biology} 12:e1004728, 2016}{DOI}{http://dx.doi.org/10.1371/journal.pcbi.1004728} {Traditionally, biomolecular simulations has focused on the study of one protein at a time. In the modern genome-enabled, high-throughput world, however, there is much to be gained by studying entire families, superfamilies, or mutational variants at the same time. To overcome the lack of tools in our field to make this possible, we developed a new high-throughput pipeline to enable massively parallel molecular simulation studies of biomolecular dynamics at the superfamily scale, illustrating its application to protein tyrosine kinases, an important class of drug targets in cancer.}
%%
%%%\newarticle{identifying-ligand-binding-sites.pdf}{Wang K, \jdc, Yang Y, and Shirts MR}{Identifying ligand binding sites and poses using GPU-accelerated Hamiltonian replica exchange molecular dynamics}{\emph{Journal of Computer Aided Molecular Design} 27:989--1007, 2013}{DOI}{http://dx.doi.org/10.1007/s10822-013-9689-8}{This paper was the first to demonstrate how GPU-accelerated molecular dynamics can be used to compute binding free energies, using Hamiltonian replica exchange methodologies that are now in standard use in drug discovery through both academic codes and commercial products, such as Schrodinger FEP+. In addition, we demonstrated how we could use this technique to identify binding sites that may be previously unknown.}
%%
%%%\newarticle{ncmc.pdf}{Nilmeier JP, Crooks GE, Minh DDL, and \jdc}{Nonequilibrium candidate Monte Carlo is an efficient tool for equilibrium simulation}{\emph{Proceedings of the National Academy of Sciences} 108:E1009, 2011}{DOI}{http://dx.doi.org/10.1073/pnas.1106094108}{We present a significant generalization of Monte Carlo methods that provide an enormously useful tool for enhancing the efficiency of molecular simulations and enabling molecular design. This technique allows astronomical increases in acceptance rates---such as efficiency gains of 10$^{30}$---allowing efficient Monte Carlo proposals for part of the system to be effectively mixed with molecular dynamics. This approach is now allowing our lab to dynamically treat protonation states, insert/delete counterions, rapidly reorient ligands in binding sites, and sample over chemical species within a single simulation.}
%
%\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% SCIENTIFIC ADVISORY BOARDS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Scientific Advisory Board Memberships}

One way I work to translate my research into impact is via participating in Scientific Advisory Boards (SABs) for companies using computational chemistry to accelerate drug discovery.
Some of the SABs I have served on include: \\

\collaboration{schrodinger-logo-small}{Schrodinger}{2013--2018}{http://schrodinger.com}{I joined the Schrodinger SAB shortly after their first release of the \href{https://www.schrodinger.com/products/fep}{FEP+} alchemical free energy calculation tool, aiming to ensure the first commercial turnkey product version of my field of alchemical free energy calculations had maximum impact for structure-enabled drug discovery. This effort was highly successful, with FEP+ near-universally used by industry to accelerate structure-enabled drug discovery efforts. I left in 2018 to focus my efforts on new, more scalable technologies and ways to use these tools for frontier problems in drug discovery.}

\collaboration{openeye-logo-small}{OpenEye Scientific}{2018--}{https://www.eyesopen.com}{I joined the OpenEye SAB excited to help enable an ecosystem of open source computer-aided drug discovery tools and academic research that could rapidly be translated into impactful deployment within both industry and academia using their \href{https://www.eyesopen.com/orion}{Orion cloud-based drug discovery platform}. OpenEye was recenly acquired by Cadence for \$500M.}

\collaboration{interline-logo-small}{Interline Therapeutics}{2021--}{https://www.interlinetx.com}{I am a founding SAB member and scientific cofounder of Interline Therapeutics, which has raised \$92M since its inception in 2021. Interline seeks to harness open source alchemical free energy methodologies that I developed to tackle problems in target and conformational selectivity to enable the design of small molecules that modulate protein communities.}

\collaboration{ventus-logo-small}{Ventus Therapeutics}{2021--}{https://www.ventustx.com}{Ventus is a rapidly growing small molecule discovery company that has raised over \$300M in three funding rounds. In addition to feating extraordinary medicinal chemistry talent, Ventus has developed a novel computational platform, ReSOLVE, to accelerate novel chemotype hit discovery for new targets. I joined the SAB to advise on the integration of structure-enabled open source computer-aided drug discovery workflows to accelerate hit-to-candidate progression.}

\collaboration{redesign-logo-small}{Redesign Science}{2020--}{https://www.redesignscience.com}{Redesign Science is a NYC-based startup aiming to commercialize technologies I helped develop as a graduate student---the construction of Markov state models of coarse-grained biomolecular dynamics from distributed biomolecular simulations---to use as a platform to accelerate drug discovery.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% SOFTWARE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\vspace{0.25in}

\section*{Open Source Software}

% TODO: Expand with list of major software products

\setstretch{1.05} 

All software developed by our lab can be found at \url{http://github.com/choderalab}

%Major codes developed by our lab can be found at \url{http://choderalab.org/code}

All software is licensed under permissive \href{http://opensource.org}{OSI}-approved open source licenses and available on \href{http://github.com/choderalab}{GitHub}

\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% ALL PUBLICATIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{All Publications}

\setstretch{1.05} 

\begin{minipage}[t]{3.3in}
All publications: \href{http://choderalab.org/publications}{http://choderalab.org/publications} \\
Google Scholar statistics: \href{http://goo.gl/qO0JW}{http://goo.gl/qO0JW} \hspace{0.2in} \\
MyNCBI Bibliography: \href{http://goo.gl/e3kjgK}{http://goo.gl/e3kjgK} \\
bioR$\chi$iv preprints: \href{https://bit.ly/2LHGwxJ}{https://bit.ly/2LHGwxJ} \\
ar$\chi$iv preprints: \href{https://bit.ly/2Rp2Y3T}{https://bit.ly/2Rp2Y3T} \\
chemRxiv preprints: \href{https://chemrxiv.org/search?q=chodera}{https://chemrxiv.org/search?q=chodera} \\
%Open source software: \href{http://choderalab.org/code}{http://choderalab.org/code}\\
Open source software: \href{http://github.com/choderalab}{http://github.com/choderalab}\\
Major datasets: \href{http://choderalab.org/data}{http://choderalab.org/data} \\
{\small \it h-index: 59 / i10-index: 120 / citations: 20151 (12 Oct 2023)} \\
{\scriptsize $^*$ denotes co-first-authors \\
$\dag$ denotes co-second-authors \\
$\ddag$ denotes co-corresponding authors}
\end{minipage}
\quad
\begin{minipage}[t]{3in}

\includegraphics[width=2.8in,valign=t]{thumbnails/citations-2023-10-12.pdf}

\vspace{0.05in}
{\small annual citation counts via Google Scholar (12 Oct 2023)}
\end{minipage}

\setstretch{1.00} 

%%%%%%%%%%%%

%\section*{Submitted and Under Review}

%\vspace{-0.2in}

%%%%%%%%%%%%
\section*{Preprints}

{\footnotesize \underline{Underline} denotes Chodera lab trainee; $\ddag$ denotes co-corresponding authorship; \jdc highlighted for convenience.}\\

% new
\newarticle{kinase-docking}{\underline{Schaller D}, Christ CD, \jdc, Volkamer A}{Benchmarking cross-docking strategies for structure-informed machine learning in kinase drug discovery}{\emph{Preprint}}{bioRxiv}{https://doi.org/10.1101/2023.09.11.557138}{We assess strategies for predicting useful docked ligand poses for structure-informed machine learning for kinase inhibitor drug discovery.}

% new
\newarticle{sake.jpg}{\underline{Wang Y}, \jdc}{SAKE: Spatial attention kinetic networks with E(n)-equivariance}{\emph{Preprint}}{ar$\chi$iv}{https://arxiv.org/abs/2301.08893}{We present a new architecture for E(n)-equivariant machine learning models that provides an excellent balance of speed and accuracy for predicting molecular properties like energies, as well as constructing normalizing flows.}

% new
\newarticle{espaloma-0.3.pdf}{Takaba K$\ddag$, Pulido I, Behara PK, Henry M, MacDermott-Opeskin H, \jdc$\ddag$, Wang Y$\ddag$}{Machine-learned molecular mechanics force field for the simulation of protein-ligand systems and beyond}{\emph{Preprint}}{ar$\chi$iv}{https://arxiv.org/abs/2307.07085}{We present a new self-consistent MM force field trained on $>$1.1M quantum chemical calculations that uses graph nets to achieve high accuracy and produce accurate protein-ligand binding free energies.}

% new
\newarticle{espaloma-charge.jpg}{\underline{Wang Y}$\ddag$, Pulido I, Takaba K, Kaminow B, Scheen J, Wang L, \jdc$\ddag$}{EspalomaCharge: Machine learning-enabled ultra-fast partial charge assignment}{\emph{Preprint}}{ar$\chi$iv}{https://doi.org/10.48550/arXiv.2302.06758}{We present a drop-in replacement for generating AM1-BCC ELF10 charges based on graph convolutional nets that is orders of magnitude faster than standard methods for both small molecules and biomolecules.}

\newarticle{qml-explicit-free-energy}{\underline{Wieder M}, \underline{Fass J}, and \jdc}{Teaching free energy calculations to learn from experimental data}{\emph{Preprint}}{bioRxiv}{https://doi.org/10.1101/2021.08.24.457513}{In this pioneering work, we show, for the first time, how we can use machine learning to systematically improve explicit solvent alchemical free energy calculations using fully machine learning force fields by fitting directly to experimental free energies. This is also the first work to show how to compute alchemical free energies in explicit solvent using fully machine learning force fields, and the first to show how this can also involve breaking and forming covalent bonds..}

\newarticle{hybrid-qm-mm.jpg}{\underline{Rufa DA}, \underline{Bruce Macdonald HE}, \underline{Fass J}, \underline{Wieder M}, \underline{Grinaway PB}, Roitberg AE, Isayev O, and \jdc}{Towards chemical accuracy for alchemical free energy calculations with hybrid physics-based machine learning / molecular mechanics potentials}{\emph{Preprint}}{bioR$\chi$iv}{https://doi.org/10.1101/2020.07.29.227959}{In this first use of hybrid machine learning / molecular mechanics (ML/MM) potentials for alchemical free energy calculations, we demonstrate how the improved modeling of intramolecular ligand energetics offered by the quantum machine learning potential ANI-2x can significantly improve the accuracy in predicting kinase inhibitor binding free energy by reducing the error from 0.97~kcal/mol to 0.47~kcal/mol, which could drastically reduce the number of compounds that must be synthesized in lead optimization campaigns for minimal additional computational cost.}

\newarticle{fragmenter}{\underline{Stern CD}, Bayly CI, Smith DGA, \underline{Fass J}, Wang L-P, Mobley DL, and \jdc}{Capturing non-local through-bond effects when fragmenting molecules for quantum chemical torsion scans}{\emph{Preprint}}{bioR$\chi$iv}{https://doi.org/10.1101/2020.08.27.270934}{We show how the Wiberg Bond Order (WBO) can be used to construct small molecule fragmentation schemes that will avoid disrupting the chemical environment around torsions. The resulting fragmentation scheme powers the QCSubmit tool used to fragment and inject small molecule datasets into the QCFractal computation pipeline for deposition into the QCArchive quantum chemistry archive the Open Force Field Initiative uses for constructing force fields, as well as powering bespoke torsion refitting for individual molecules.}

\newarticle{biphenyl-wbo}{\underline{Stern CD}, Maat J, Dotson DL, Bayly CI, Smith DGA, Mobley DL, and \jdc}{Capturing non-local through-bond effects in molecular mechanics force fields: II. Using fractional bond orders to fit torsion parameters}{\emph{Preprint}}{bioR$\chi$iv}{https://www.biorxiv.org/content/10.1101/2022.01.17.476653v1.abstract}{We show how the Wiberg Bond Order (WBO) can be used to accurately interpolate torsional profiles for molecular mechanics force fields, which holts the potential for drastically reducing the complexity of these force fields while increasing their ability to generalize and accurately treat complex druglike molecules such as kinase inhibitors.}

\newarticle{gimlet}{\underline{Wang Y}, \underline{Fass J}, \underline{Stern CD}, and \jdc}{Graph nets for partial charge prediction}{\emph{Preprint}}{ar$\chi$iv}{https://arxiv.org/abs/1909.07903}{Graph convolutional and message-passing networks can be a powerful tool for predicting physical properties of small molecules when coupled to a simple physical model that encodes the relevant invariances. Here, we show the ability of graph nets to predict partial atomic charges for use in molecular dynamics simulations and physical docking.}

%{\bf Experimental best practices}\\

\newarticle{itc-worksheet}{\underline{Boyce SE}, Tellinghuisen JT, and \jdc}{Avoiding accuracy-limiting pitfalls in the study of protein-ligand interactions with isothermal titration calorimetry}{\emph{Preprint}}{bioR$\chi$iv}{http://dx.doi.org/10.1101/018036}{We demonstrate how to avoid accuracy-limiting problems in standard isothermal calorimetry experiments as well as capture the primary sources of uncertainty in thermodynamic parameters.}

\newarticle{bhmm.pdf}{\jdc, No\'{e} F, Hinrichs NS, Keller B, Elms PJ, Kaiser CM, Ewall-Wice A, Marqusee S, and Bustamante C}{Bayesian hidden Markov model analysis of single-molecule biophysical experiments}{\emph{Preprint}}{ar$\chi$iv}{http://arxiv.org/find/all/1/all:+chodera/0/1/0/all/0/1}{We present a Bayesian hidden Markov model analysis scheme that allows biomolecular conformational dynamics---and the corresponding uncertainty due to limited data---to be inferred from single-molecule trajectories. This approach was developed for a single-molecule study examining folding dynamics of nascent proteins exiting the ribosome [Science 334:1723, 2011 $\cdot$ \href{http://dx.doi.org/10.1126/science.1209740}{DOI}].}

\newarticle{robust-rate-estimates.pdf}{\jdc, Elms PJ, Swope WC, Prinz J-H, Marqusee S, Bustamante C, No\'{e} F, and Pande VS}{A robust approach to estimating rates from time-correlation functions}{\emph{Preprint}}{ar$\chi$iv}{http://arxiv.org/abs/1108.2304}{We present a simple, robust approach to estimating two-state rate constants from experimental or simulation data.}

\vspace{-0.2in}
%\eject

%%%%%%%%%%%%

\section*{Reviews and Commentaries}

{\bf Open science and globally equitable drug discovery}\\

\newarticle{covid-moonshot-nature-commentary}{von Delft F, Calmiano M, \jdc, Griffen E, Lee A, London N, Matviuk T, Perry B, Robinson M, and von Delft A}{A white-knuckle ride of open COVID drug discovery}{\emph{Nature} 594:330, 2021}{DOI}{https://doi.org/10.1038/d41586-021-01571-1}{The COVID Moonshot is an open science effort to discover a direct-acting SARS-CoV-2 oral antiviral. Here, we share lessons from this effort, including the missed opportunity to develop a phase 2 ready drug more than a decade ago that could have halted the COVID-19 pandemic in its tracks.}

% new
\newarticle{knowable-covid-moonshot}{Lee A, \jdc, von Delft F}{Why we are developing a patent-free COVID antiviral therapy}{\emph{Knowable Magazine} 09.27.2021}{web}{https://knowablemagazine.org/article/health-disease/2021/why-we-are-developing-patentfree-covid-antiviral-therapy}{In this Opinion, we present the case for a new open science driven model for drug discovery for pandemics.}

{\bf Best practices for computer-aided drug discovery}\\

%new
\newarticle{affinity-benchmark-best-practices}{Hahn DF, Bayly CI, \underline{Boby ML}, \underline{Bruce Macdonald HE}, \jdc, Mey ASJS, Mobley DL, Perez Benito L, Schindler CEM, Tresadern G, Warren GL}{Best practices for constructing, preparing, and evaluating protein-ligand binding affinity benchmarks}{\emph{Living Journal of Computational Molecular Science} 4:1497, 2022}{DOI}{https://doi.org/10.33011/livecoms.4.1.1497}{This living best practices paper for the Living Journal of Computational Molecular Sciences describes the current community consensus in how to curate experimental benchmark data for assessing predictive affinity models for drug discovery, how to prepare these systems for affinity calculations, and how to assess the results to compare performance.}

%new
\newarticle{alchemical-best-practices}{Mey ASJS$^\ddag$, Allen B, \underline{Bruce Macdonald HE}, \jdc$^\ddag$, Kuhn M, Michel J, Mobley DL$^\ddag$, Naden LN, Prasad S, Rizzi A, Scheen J, Shirts MR$^\ddag$, Tresadern G, and Xu H}{Best practices for alchemical free energy calculations}{\emph{Living Journal of Computational Molecular Science} 2:18378, 2020}{DOI}{https://doi.org/10.33011/livecoms.2.1.18378}{This living review for the Living Journal of Computational Molecular Sciences (LiveCoMS) covers the essential considerations for running alchemical free energy calculations for rational molecular design for drug discovery.}

\newarticle{alchemical-methods.jpg}{\jdc, Mobley DL, Shirts MR, Dixon RW, Branson KM, and Pande VS}{Free energy methods in drug discovery and design: Progress and challenges}{\emph{Current Opinion in Structural Biology} 21:150, 2011}{DOI}{http://dx.doi.org/10.1016/j.sbi.2011.01.011}{A review of current opportunities and challenges for alchemical free energy calculations in drug discovery and design.}

\newarticle{binding-free-energy.pdf}{Shirts MR, Mobley DL, \jdc}{Alchemical free energy calculations: Ready for prime time?}{\emph{Annual Reports in Computational Chemistry} 3:41, 2007}{DOI}{http://dx.doi.org/10.1016/S1574-1400(07)03004-6}{A review of current alchemical free energy methodologies and their potential for use in drug discovery and ligand design.}

{\bf Coarse-graining for biomolecular dynamics}\\

\newarticle{coarse-graining}{Gkeka P, Stoltz G, Farimani AB, Belkacemi Z, Ceriotti M, \jdc, Dinner AR, Ferguson A, Maillet JB, Minoux H, Peter C, Pietrucci F, \underline{Silveira A}, Tkatchenko A, Trstanova Z, Wiewiora R, Leliévre T}{Machine learning force fields and coarse-grained variables in molecular dynamics: application to materials and biological systems}{\emph{Journal of Chemical Theory and Computation} 16:4757, 2020}{DOI}{https://doi.org/10.1021/acs.jctc.0c00355}{We review the state of the art in applying machine learning to coarse grain force fields in space and time to study mutliscale dynamics.}

\newarticle{msm-projection.pdf}{\jdc and No\'{e} F}{Markov state models of biomolecular conformational dynamics}{\emph{Current Opinion in Structural Biology} 25:135, 2014}{DOI}{http://dx.doi.org/10.1016/j.sbi.2014.04.002}{A review of exciting recent developments in the stochastic modeling of biomolecular dynamics using techniques I originally co-developed to study protein folding.}

\newarticle{entropy-enthalpy-compensation.pdf}{\jdc and Mobley DL}{Entropy-enthalpy compensation: Role and ramifications for rational ligand design}{\emph{Annual Reviews in Biophysics} 42:121, 2013}{DOI}{http://dx.doi.org/10.1146/annurev-biophys-083012-130318}{Entropy-enthalpy compensation is likely a universal phenomena, but not as severe as widely thought, and irrelevant for drug discovery and ligand design.}

\newarticle{markov-model-generation-3.pdf}{\underline{Prinz JH}, Wu H, Sarich M, Keller B, Fischbach M, Held M, \jdc, Sch\"{u}tte, and No\'{e} F}{Markov models of molecular kinetics: Generation and validation}{\emph{Journal of Chemical Physics} 134:174105, 2011}{DOI}{http://dx.doi.org/10.1063/1.3565032}{Current best practices for the generation and validation of Markov state models for describing the stochastic dynamics of biomolecular systems.}

\newarticle{social-network.pdf}{\jdc and Pande VS}{The Social Network (of protein conformations)}{\emph{Proceedings of the National Academy of Sciences} 108:12969, 2011}{DOI}{http://dx.doi.org/10.1073/pnas.1109571108}{A new methodology for mapping protein conformational spaces is reminiscent of how we use two-dimensional maps to navigate a three-dimensional world.}

{\bf Protein folding}\\

\newarticle{folding-funnel.pdf}{Dill KA, Ozkan SB, Weikl TR, \jdc, and Voelz VA}{The protein folding problem: When will it be solved?}{\emph{Current Opinion in Structural Biology} 17(3):342, 2007}{DOI}{http://dx.doi.org/10.1016/j.sbi.2007.06.001}{A review of the current state of the protein folding problem.}

\eject

%%%%%%%%%%%%

\section*{All Published and In Press Publications}

%{\bf New publications since promotion to Associate Member}\\

% new
\newarticle{multicompound-multitarget-scoring.jpg}{Outhwaite IR, \underline{Singh S}, Berger B-T, Knapp S, \jdc, Seeliger MA}{Death by a thousand cuts through kinase inhibitor combinations that maximize selectivity and enable rational multitargeting}{eLife 12:e86189, 2024}{DOI}{https://doi.org/10.7554/eLife.86189}{We show how combinations of kinase inhibitors can achieve selectivity gains for rational kinase polypharmacology.}

% new
\newarticle{covid-moonshot-logo-small}{\underline{Boby ML}, Fearon D, Ferla M, Filep M, Robinson MC, \href{https://docs.google.com/spreadsheets/d/1OnPzj5IvYtD5uu6tqmlhvOX47__WgN1VrbzMH_kqns0}{The COVID Moonshot Consortium}, \jdc$\ddag$, Lee A$\ddag$, London N$\ddag$, von Delft F$\ddag$}{Open science discovery of potent noncovalent SARS-CoV-2 main protease inhibitors}{\emph{Science} 382:eabo7201, 2023}{DOI}{https://doi.org/10.1126/science.abo7201}{We report the discovery of a new oral antiviral non-covalent SARS-CoV-2 main protease inhibitor developed by the COVID Moonshot, a global open science collaboration leveraging free energy calculations on Folding@home and ML-accelerated synthesis planning, now in accelerated preclinical studies funded by an \$11M grant from the WHO ACT-A program via the Wellcome Trust. We are currently in discussions with generics manufacturers about partnering with us throughout clinical trials to ensure we can scale up production for global equitable and affordable access once approved by regulatory agencies.}

% new
\newarticle{mutations.jpg}{\underline{Zhang I}, \underline{Rufa DA}, \underline{Pulido I}, \underline{Henry MM}, Rosen LE, Hauser K, \underline{Singh S}, \jdc$\ddag$}{Identifying and overcoming the sampling challenges in relative binding free energy calculations of a model protein:protein complex}{\emph{Journal of Chemical Theory and Computaiton} 19:4863, 2023}{DOI}{https://doi.org/10.1021/acs.jctc.3c00333}{We assess what is required for alchemical free energy calculations to be able to make high-quality predictions of the impact of interfacial mutations on protein-protein binding.}

% new
\newarticle{openmm7-logo}{Eastman P, Galvelis R, Peláez RP, Abreu CRA, Farr SE, Gallicchio E, Gorenko A, Henry MH, Hu F, Huang J, Kr\"{a}mer A, Michel J, Mitchell J, Pande VS, Rodrigues JPGLM, \underline{Rodriguez-Guerra J}, Simmonett AC, Swails J, Turner P, \underline{Wang Y}, \underline{Zhang I}, \jdc, De Fabritiis G, Markland TE}{OpenMM 8: Molecular Dynamics Simulation with Machine Learning Potentials}{\emph{Journal fo Physical Chemistry B} 128:109, 2024}{DOI}{https://doi.org/10.1021/acs.jpcb.3c06662}{We present OpenMM 8, which includes GPU-accelerated support for simulating hybrid ML/MM systems that use machine learning (ML) potentials to achieve high accuracy with minimal loss in speed.}

% new
\newarticle{nnp-mm}{Galvelis R, Varela-Rial A, Doerr S, Fino R, Eastman P, Markland TE,  \jdc, and de~Fabritiis G}{NNP/MM: Fast molecular dynamics simulations with machine learning potentials and molecular mechanics}{\emph{Journal of Chemical Information and Modeling}}{\emph{Journal of Chemical Information and Modeling} 63:5701, 2023}{https://doi.org/10.1021/acs.jcim.3c00773}{We demonstrate that a new generation of quantum machine learning (QML) potentials based on neural networks---which can achieve quantum chemical accuracy at a fraction of the cost---can be implemented efficiently in the OpenMM molecular dynamics simulation engine as part of hybrid machine learning / molecular mechanics (ML/MM) potentials that promise to deliver superior accuracy for modeling protein-ligand interactions.}

% new
% Automated high throughput pKa and distribution coefficient measurements of pharmaceutical compounds for the SAMPL8 blind prediction challenge
% https://doi.org/10.1007/s10822-021-00427-0

% new
\newarticle{men1.jpg}{Perner F, Stein EM, Wenge DV, \underline{Singh S}, Kim J, Apazidis A, Rahnamoun H, Anand D, Marinaccio C, Hatton C, Wen Y, Stone RM, Schaller D, Mowla S, Xiao W, Gamlen HA, Stonestrom AJ, Persaud S, Ener E, Cutler JA, Doench JG, McGeehan GM, Volkamer A, \jdc, Nowak RP, Fischer ES, Levine RL, Armstrong SA, Cai SF}{MEN1 mutations mediate clinical resistance to menin inhibition}{\emph{Nature} 615:913, 2023}{DOI}{https://doi.org/10.1038/s41586-023-05755-9}{We describe how mutants that confer therapeutic resistance to menin inhibition impact small molecule binding but not interactions with the natural ligand MLL1.}

% new
\newarticle{sage}{\underline{Boothroyd S}, Behara PK, Madin OC, Hahn DF, Jang H, Gapsys V, Wagner JR, Horton JT, Dotson DL, Thompson MW, Maat J, Gokey T, Wang L-P, Cole DJ, Gilson MK$^\ddag$, Chodera JD$^\ddag$, Bayly CI, Shirts MR$^\ddag$, Mobley DL$^\ddag$}{Development and benchmarking of Open Force Field 2.0.0---the Sage small molecule force field}{\emph{Journal of Chemical Theory and Computation} 19:3251, 2023}{DOI}{https://doi.org/10.1021/acs.jctc.3c00039}{We present a new generation of small molecule force field for molecular design from the Open Force Field Initiative fit to both quantum chemical and experimental liquid mixture data.}

% new
\newarticle{kadi-fragments.jpg}{Saar KL, McCorkindale W, Fearon D, \underline{Boby M}, Barr H, Ben-Shmuel A, COVID Moonshot Consortium, London N, von Delft F, \jdc, Lee AA}{Turning high-throughput structural biology into predictive inhibitor design}{\emph{Proceedings of the National Academy of Sciences} 120:e2214168120, 2023}{DOI}{https://doi.org/10.1073/pnas.2214168120}{We demonstrate how potent inhibitors can be predicted from high-throughput structural biology, demonstrating this approach against the SARS-CoV-2 main viral protease (Mpro).}

\newarticle{spice-dataset.pdf}{Eastman P, Behara PK, Dotson DL, Galvelis R, \underline{Herr JE}, Horton JT, Mao Y, \jdc, Pritchard BP, \underline{Wang Y}, De Fabritiis G, and Markland TE}{SPICE, a dataset of drug-like molecules and peptides for training machine learning potentials}{\emph{Scientific Data} 10:11, 2023}{DOI}{https://doi.org/10.1038/s41597-022-01882-6}{To remedy the lack of large, open quantum chemical datasets for training accurate general machine learning potentials and molecular mechanics force fields for druglike small molecules and biomolecules, we produce the open SPICE dataset, and show how it can be used to build extremely accurate machine learning potentials.}

\newarticle{bespokefit.pdf}{Horton JT, \underline{Boothroyd S}, Wagner W, Mitchell JA, Gokey T, Dotson DL, Behara PK, Ramaswamy VK, Mackey M, \jdc, Anwar J, Mobley DL, and Cole DJ}{Open Force Field BespokeFit: Automating bespoke torsion parametrization at scale}{\emph{Journal of Chemical Informatics and Modeling} 62:22, 2022}{DOI}{https://doi.org/10.1021/acs.jcim.2c01153}{We describe an automated pipeline for generating tailored force field parameters for small molecules using quantum chemical or quantum machine learning potentials.}

% updated
\newarticle{espaloma.jpg}{\underline{Wang Y}, \underline{Fass J}, and \jdc}{End-to-end differentiable molecular mechanics force field construction}{\emph{Chemical Science} 13:12016, 2022}{DOI}{https://doi.org/10.1039/D2SC02739A}{Molecular mechanics force fields have been a workhorse for computational chemistry and drug discovery. Here, we propose a new approach to force field parameterization in which graph convolutional networks are used to perceive chemical environments and assign molecular mechanics (MM) force field parameters. The entire process of chemical perception and parameter assignment is differentiable end-to-end with respect to model parameters, allowing new force fields to be easily constructed from MM or QM force fields, extended, and applied to arbitrary biomolecules.}

% new
\newarticle{openff-evaluator}{\underline{Boothroyd S}, Wang L-P, Mobley DL, \jdc, and Shirts MR}{The Open Force Field Evaluator: An automated, efficient, and scalable framework for the estimation of physical properties from molecular simulation}{\emph{Journal of Chemical Theory and Computation} 18:3566, 2022}{DOI}{https://doi.org/10.1021/acs.jctc.1c01111}{We describe a new software framework for automated evaluation of physical properties for the benchmarking and optimization of small molecule force fields according to best practices.}

% new
\newarticle{openff-evaluator}{\underline{Boothroyd S}, Madin OC, Mobley DL, Wang L-P, \jdc, and Shirts MR}{Improving force field accuracy by training against condensed-phase mixture properties}{\emph{Journal of Chemical Theory and Computation} 18:3577, 2022}{DOI}{https://doi.org/10.1021/acs.jctc.1c01268}{We use a new automated framework for physical property evaluation and fitting to show how molecular mechanics force fields can be systematically improved by fitting to condensed phase properties.}

% new
\newarticle{cache}{Ackloo S, Al-awar R, Amaro RE, Arrowsmith CH, Azevedo H, Batey RA, Bengio Y, Betz UAK, Bologa CG, \jdc, Cornell WD, Dunham I, Ecker GF, Edfeldt K, Edwards AM, Gilsom MK, Gordijo CR, Hessler G, Hillisch A, Hogner A, Irwin JJ, Jansen JM, Kuhn D, Leach AR, Lee AA, Lessel U, Moult J, Muegge I, Oprea TI, Perry BG, Riley, Singh Saikantendu K, Santhakumar V, Schapira M, Scholten C, Todd MH, Vedadi M, Volkamer A, and Wilson TM}{CACHE (Critical Assessment of Computational Hit-finding Experiments): A public-private partnership benchmarking initiative to enable the development of computational methods for hit-finding}{\emph{Nature Reviews Chemistry}, 6:287, 2022}{DOI}{https://doi.org/10.1038/s41570-022-00363-z}{We describe CACHE: A new public-private partnership that aims to transform computer-aided drug discovery much the way that CASP transformed protein structure prediction into a reproducible, accurate engineering discipline.}

%new
\newarticle{sampl7-phip2}{Grosjean H, \underline{Isik M}, Aimon A, Mobley D, \jdc, von Delft F, and Biggin PC}{SAMPL7 protein-ligand challenge: A community-wide evaluation of computational methods against fragment screening and pose-prediction}{\emph{Journal of Computer-Aided Molecular Design} 36:291, 2022}{DOI}{https://doi.org/10.1007/s10822-022-00452-7}{We field a blind community challenge to assess how well state of the art computational chemistry methods can predict the binding modes of small druglike fragments to a protein target for which no chemical matter is known, PHIP2, using fragment screening at the Diamond Light Source.}

%new
\newarticle{ink4-cdk4.jpg}{Li Q, Jiang B, \underline{Guo J}, Shao H, Del Priore IS, Chang Q, Kudo R, Li Z, Razavi P, Liu B, Boghossian AS, Rees MG, Ronan MM, Roth JA, Donovan KA, Palafox M, Reis-Filho JS, de Stanchina E, Fischer ES, Rosen N, Serra V, Koff A, \jdc, Gray NS, and Chandarlapaty S}{INK4 tumor suppressor proteins mediate resistance to CDK4/6 kinase inhibitors}{\emph{Cancer Discovery} 12:356, 2022}{DOI}{http://dx.doi.org/10.1158/2159-8290.CD-20-1726}{We demonstrate CDK6 causes drug resistance by binding INK4 proteins, and develop bifunctional degraders conjugating palbociclib with E3 ligands to overcome this mechanism of resistance.}

% new
\newarticle{gcn2-activation}{Mellinghoff I, Tang CP, Clark O, Ferrarone J, Campos C, Lalani AS, \jdc, Intlekofer AM, and Elemento O}{GCN2 kinase activation by ATP-competitive kinase inhibitors}{\emph{Nature Chemical Biology}, 18:207, 2022}{DOI}{https://doi.org/10.1038/s41589-021-00947-8}{We describe paradoxical activation of GCN2 kinase activity by the kinase inhibitor neratinib, and propose a model for how inhibitor-induced dimerization might cause this unusual activity.}

% new
\newarticle{2CLJQ}{Madin OC, \underline{Boothroyd S}, Messerly RA, \jdc, \underline{Fass J}, Shirts MR}{Bayesian inference-driven model parameterization and model selection for 2CLJQ fluid models}{\emph{Journal of Chemical Informatics and Modeling}, 62:874, 2022}{DOI}{https://doi.org/10.1021/acs.jcim.1c00829}{Here, we show how Bayesian inference can be used to automatically perform model selection and fit parameters for a molecular mechanics force field.}

% new
\newarticle{parsley.pdf}{Qiu Y, Smith DGA, \underline{Boothroyd S}, Jang H, Wagner J, Bannan C, Gokey T, Lim VT, \underline{Stern CD}, \underline{Rizzi A}, Lucas X, Tjanaka B, Shirts MR$^\ddag$, Gilson MK$^\ddag$, \jdc$^\ddag$, Bayly CI$^\ddag$, Mobley DL$^\ddag$, Wang L-P$^\ddag$}{Development and benchmarking of Open Force Field v1.0.0, the Parsley small molecule force field}{\emph{Journal of Chemical Theory and Computation}, 17:6262, 2021}{DOI}{https://doi.org/10.1021/acs.jctc.1c00571}{We present a new, modern small molecule force field for molecular design from the Open Force Field Initiative, a large industry-academic collaboration that focuses on open science, open data, and modern open source infrastructure.}

% new
\newarticle{abl-mutant-kinetics}{Lyczek A, Tilman-Berger B, Rangwala AM, Paung Y, Tom J, Philipose H, \underline{Guo J}, \underline{Albanese SK}, Robers MB, Knapp S, \jdc, Seeliger MA}{Mutation in Abl kinase with altered drug binding kinetics indicates a novel mechanism of imatinib resistance}{\emph{Proceedings of the National Academy of Sciences} 118:e2111451118}{DOI}{https://doi.org/10.1073/pnas.2111451118, 2021}{Here, we characterize the biophysical mechanisms underlying mutants of Abl kinase associated with clinical drug resistance to targeted cancer therapies. We uncover a surprising novel mechanism of mutational resistance to kinase inhibitor therapy in which the off-rate for inhibitor unbinding is increased without affecting inhibitor affinity.}

% new
\newarticle{qcarchive}{Smith DGA, Lolinco AT, Glick ZL, Lee J, Alenaizan A, Barnes TA, Borca CH, Di Remigio R, Dotson DL, Ehlert S, Heide AG, Herbst MF, Hermann J, Hicks CB, Horton JT, Hurtado AG, Kraus P, Kruse P, Lee SJR, Misiewicz JP, Naden LN, Ramezanghorbani F, Scheurer M, Shriber JB, Simmonett AC, Steinmetzer J, Wagner JR, Ward L, Welborn M, Altarawy D, Anwar J, \jdc, Dreuw A, Kulik HJ, Liu F, Martinez TJ, Matthews DA, Schaefer III HF, Sponer J, Turney JM, Wang L-P, De Silva N, King RA, Stanton JF, Gordon MS, Windus TL, Sherrill CD, Burns LA}{Quantum chemistry common driver and databases (QCDB) and quantum chemistry engine (QCEngine): Automation and interoperability among computational chemistry programs}{\emph{Journal of Chemical Physics} 155:204801, 2021}{DOI}{https://doi.org/10.1063/5.0059356}{We describe a new community-wide approach to interoperability for quantum chemistry packages that will enable large-scale applications such as next-generation machine learning for chemistry and automated force field construction for drug discovery.}

\newarticle{vir-antibodies-escape}{Starr TN, Czudnochowski N, Zatta F, Park YJ, Liu Z, Addetia A, Pinto D, Beltramello M, Hernandez P, Greaney AJ, Marzi R, \underline{Glass WG}, \underline{Zhang I}, Dingens AS, Bowen JE, Wojcechowskyj JA, De Marco A, Rosen LE, Zhou J, Montiel-Ruiz M, Kaiser H, Tucker H, Housley MP, di Iulio J, Lombardo G, Agostini M, Sprugasci N, Culap K, Jaconi S, Meury M, Dellota E, Cameroni E, Croll TI, Nix JC, Havenar-Daughton C, Telenti A, Lempp FA, Pizzuto MS, \jdc, Hebner CM, Whelan SPJ, Virgin HW, Veesler D, Corti D, Bloom JD, Snell G}{SARS-CoV-2 RBD antibodies that maximize breadth and resistance to escape}{\emph{Nature} 597:97, 2021}{DOI}{https://doi.org/10.1038/s41586-021-03807-6}{We comprehensively characterize escape, breadth, and potency across a panel of SARS-CoV-2 antibodies targeting the receptor binding domain, including the parent antibody of the recently approved Vir antibody drug (Sotrovimab), illuminating escape mutations with structural and dynamic insight into their mechanism of action.}

\newarticle{neutromeratio-vacuum.jpg}{\underline{Wieder M}, \underline{Fass J}, and \jdc}{Fitting quantum machine learning potentials to experimental free energy data: Predicting tautomer ratios in solution}{\emph{Chemical Science} 12:11364, 2021}{DOI}{http://doi.org/10.1039/D1SC01185E}{We demonstrate, for the first time, how alchemical free energy calculations can performed on systems simulated entirely with quantum machine learning potentials and how these potentials can be retrained on experimental free energies to generalize to new molecules from limited training data. We apply this approach to a difficult problem in small molecule drug discovery: Predicting accurate tautomer ratios in solution.}

\newarticle{foldingathome-covid-cryptic-pockets.jpg}{Zimmerman MI, Porter JR, Ward MD, Singh S, Vithani N, Meller A, Mallimadugula UL, Kuhn CE, Borowsky JH, \underline{Wiewiora RP}, Hurley, MFD, Harbison AM, Fogarty CA, Coffland JE, Fadda E, Voelz VA, \jdc, Bowman GR}{SARS-CoV-2 simulations go exascale to predict dramatic spike opening and cryptic pockets across the proteome}{\emph{Nature Chemistry} 13:651, 2021}{DOI}{https://doi.org/10.1038/s41557-021-00707-0}{To accelerate a multitude of drug development activities to combat the global threat posed by COVID-19, over a million citizen scientists have banded together through the Folding@home distributed computing project to create the world’s first Exascale computer and simulate protein dynamics. An unprecedented 0.1 seconds of simulation of the viral proteome reveal how the spike complex uses conformational masking to evade an immune response, conformational changes implicated in the function of other viral proteins, and cryptic pockets that are absent in experimental structures. These structures and mechanistic insights present new targets for the design of therapeutics.}

\newarticle{postera-ml-model}{Morris A, McCorkindale W, the COVID Moonshot Consortium, Drayman N, \jdc, Tay S, London N, Lee AA}{Discovery of SARS-CoV-2 main protease inhibitors using a synthesis-directed de novo design model}{\emph{Chemical Communications} 57:5909, 2021}{DOI}{https://doi.org/10.1039/D1CC00050K}{We show how a machine learning models of ligand affinity can be coupled to synthetic enumeration models to rapidly generate potent inhibitors of the SARS-CoV-2 main viral protease.}

\newarticle{what-can-msms-do}{Su\'{a}rez E, \underline{Wiewiora RP}, Wehmeyer C, No\'{e} F, \jdc$^\ddag$, Zuckerman DM$^\ddag$}{What Markov State Models can and cannot do: Correlation versus path-based observables in protein-folding models}{\emph{Journal of Chemical Theory and Computation} 17:3119, 2021}{DOI}{https://doi.org/10.1021/acs.jctc.0c01154}{Markov state models are now well-established for describing the long-time conformational dynamics of proteins. Here, we take a critical look of what properties can reliably be extracted from these coarse-grained models.}

\newarticle{vir-cell-xray.jpg}{Thompson EC, Rosen LE, Shepherd JG, Spreafico R, da Silva Filipe A, Wojcechowskyj JA, Davis C, Piccoli L, Pascall DJ, Dillen J, Lytras S, Czudnochowski N, Shah R, Meury M, Jesudason N, De Marco A, Li K, Bassi J, O'Toole A, Pinto D, Colquhoun RM, Culap K, Jackson B, Zatta F, Rambaut A, Jaconi S, Sreenu VB, Nix J, \underline{Zhang I}, Jarrett RF, \underline{Glass WG}, Beltramello M, Nomikou K, Pizzuto M, Tong L, Cameroni E, Cross TI, Johnson N, Di Iulio J, Wickenhagen A, Ceschi A, Harbison AM, Mair D, Ferrari P, Smollett K, Sallusto F, Carmichael S, Garzoni C, Nichols J, Galli M, Hughes J, Riva A, Ho A, Schiuma M, Semple MG, Openshaw PJM, Fadda E, Baillie JK, \jdc, Rihn SJ, Lycet SJ, Virgin HW, Telenti A, Corti D, Robertson DL, Snell J, ISARIC4C Investigators}{Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity}{\emph{Cell} 184:1171, 2021}{DOI}{https://doi.org/10.1016/j.cell.2021.01.037}{New mutations that enhance the affinity of SARS-CoV-2 spike protein for human ACE2—and potentially pose threats to antibody-based therapeutics and vaccines for COVID-19—are already emerging in the wild. We characterize and describe sentinel mutations of SARS-CoV-2 in the wild that herald challenges for combatting COVID-19, and use simulations of the RBD-ACE2 interface on Folding@home to biophysically characterize why these mutations can lead to enhanced affinity.}

\newarticle{covid-moonshot-logo-small}{\jdc$^\ddag$, Lee AA$^\ddag$, London N$^\ddag$, and von Delft F$^\ddag$}{Crowdsourcing drug discovery for pandemics}{\emph{Nature Chemistry} 12:581, 2020}{DOI}{https://doi.org/10.1038/s41557-020-0496-2}{The COVID-19 pandemic has left the world scrambling to find effective therapies to stem the tidal wave of death and put an end to the worldwide disruption caused by SARS-CoV-2. In this Correspondence, we argue for the need for a new open, collaborative drug discovery model (exemplified by our COVID Moonshot collaboration) that breaks free of the limitations of industry-led competitive drug discovery efforts that necessarily restrict information flow and hinder rapid progress by prioritizing profits and patent protection over human lives.}

\newarticle{sampl6-pKa-assessment}{\underline{Isik M}, \underline{Rustenburg AS}, Gunner MR, Mobley DL, \jdc}{Overview of the SAMPL6 pKa challenge: evaluating small molecule microscopic and macroscopic pKa predictions}{\emph{Journal of Computer Aided Molecular Design} 35:131, 2021}{DOI}{https://doi.org/10.1007/s10822-020-00362-6}{The SAMPL6 pKa challenge assessed the ability of the computational chemistry community to predict macroscopic and microscopic pKas for a set of druglike molecules resembling kinase inhibitors. This paper reports on the overall performance and lessons learned, including the surprising finding that many tools predict reasonably accurate macroscopic pKas corresponding to the wrong microscopic protonation sites.}

\newarticle{sampl6-logP-assessment}{\underline{Isik M}, Bergazin TD, Fox T, \underline{Rizzi A}, \jdc, and Mobley DL}{Assessing the accuracy of octanol-water partition coefficient predictions in the SAMPL6 Part II log P Challenge}{\emph{Journal of Computer Aided Molecular Design} 34:335, 2020}{DOI}{http://doi.org/10.1007/s10822-020-00295-0}{We report the performance assessment of the 91 methods that were submitted to the SAMPL6 blind challenge for predicting octanol-water partition coefficient (logP) measurements. The average RMSE of the most accurate five MM-based, QM-based, empirical, and mixed approach methods based on RMSE were 0.92$\pm$0.13, 0.48$\pm$0.06, 0.47$\pm$0.05, and 0.50$\pm$0.06, respectively.}

\newarticle{pka-free-energies.jpg}{Gunner MR, Murakami T, \underline{Rustenburg AS}, \underline{Isik M}, \jdc}{Standard state free energies, not pK$_a$s, are ideal for describing small molecule protonation and tautomeric states}{\emph{Journal of Computer Aided Molecular Design} 34:561, 2020}{DOI}{https://doi.org/10.1007/s10822-020-00280-7}{Here, we demonstrate how the physical nature of protonation and tautomeric state effects means that the standard state free energies of each microscopic protonation/tautomeric state at a single pH is sufficient to describe the complete pH-dependent microscopic and macroscopic populations. We introduce a new kind of diagram that uses this concept to illustrate a variety of pH-dependent phenomena, and show how it can be used to identify common issues with protonation state prediction algorithms. As a result, we recommend future blind prediction challenges utilize microstate free energies at a single reference pH as the minimal sufficient information for assessing prediction accuracy and utility.}

\newarticle{kinase-selectivity-paper}{\underline{Albanese SK}, \jdc, Volkamer A, Keng S, Abel R, and Wang L}{Is structure based drug design ready for selectivity optimization?}{\emph{Journal of Chemical Informatics and Modeling} 60:6211, 2020}{DOI}{https://doi.org/10.1021/acs.jcim.0c00815}{We asked whether the similarity of binding sites in related kinases might result in a fortuitous cancellation of errors in using alchemical free energy calculations to predict kinase inhibitor selectivities. Surprisingly, we find that even distantly related kinases have sufficient correlation in their errors that predicting changes in selectivity can be much more accurate than predicting changes in potency due to this effect, and show how this could lead to large reductions in the number of molecules that must be synthesized to achieve a desired selectivity goal.}

\newarticle{sampl6-sampling}{\underline{Rizzi A}, Jensen T, Slochower DR, Aldeghi M, Gapsys V, Ntekoumes D, Bosisio S, Papadourakis M, Henriksen NM, de Groot BL, Cournia Z, Dickson A, Michel J, Gilson MK, Shirts MR, Mobley DL, and \jdc}{The SAMPL6 SAMPLing challenge: Assessing the reliability and efficiency of binding free energy calculations}{\emph{Journal of Computer Aided Molecular Design} 34:601, 2020}{DOI}{http://dx.doi.org/10.1007/s10822-020-00290-5}{To assess the relative efficiencies of alchemical binding free energy calculations, the SAMPL6 SAMPLing challenge asked participants to submit predictions as a function of computer effort for the same force field and charge model. Surprisingly, we found that most molecular simulation codes cannot agree on the binding free energy was, even for the same force field.}

\newarticle{sampl6-part2-logP}{\underline{Isik M}, Levorse D, Mobley DL, Rhodes T, and \jdc}{Octanol-water partition coefficient measurements for the SAMPL6 Blind Prediction Challenge}{\emph{Journal of Computer Aided Molecular Design} 34:405, 2020}{DOI}{http://doi.org/10.1007/s10822-019-00271-3}{We describe the design and data collection (and associated challenges) for the SAMPL6 part II logP octanol-water blind prediction challenge, where the goal was to benchmark the accuracy of force fields for druglike molecules (here, molecules resembling kinase inhibitors).}

\newarticle{slochower-host-guest}{Slochower DR, Hendrikson NM, Wang LP, \jdc, Mobley DL, and Gilson MK}{Binding thermodynamics of host-guest systems with SMIRNOFF99Frosst 1.0.5 from the Open Force Field Initiative}{\emph{Journal of Chemical Theory and Computation} 15:6225, 2019}{DOI}{https://doi.org/10.1021/acs.jctc.9b00748}{We assess the accuracy of the SMIRNOFF99Frosst 1.0.5 force field in reproducing host-guest binding thermodynamics in comparison with the GAFF force field, demonstrating how the SMIRNOFF format for compactly specifying force fields provide comparable accuracy with 20x fewer parameters.}

\newarticle{sharing-data.pdf}{Abraham MJ, Apostolov R, Barnoud J, Bauer P, Blau C, Bonvin AMMJ, Chavent M, \jdc, Condic-Jurkic K, Delemotte L, Grubm\"{u}ller H, Howard RJ, Jordan J, Lindahl E, Ollila S, Selent J, Smith D, Stansfeld PJ, Tiemann J, Trellet M, Woods C, and Zhumov A}{Sharing data from molecular simulations}{\emph{Journal of Chemical Informatics and Modeling} 59:4093, 2019}{DOI}{https://doi.org/10.1021/acs.jcim.9b00665}{There is a dire need to establish standards for sharing data in the molecular sciences. Here, we review the findings of a workshop held in Stockholm in Nov 2018 to discuss this need.}

\newarticle{id1}{Wojnarowicz PM, Lima e Silva R, Ohnaka M, Lee SB, Chin Y, Kulukian A, Chang SH, Desai B, Escolano MG, Shah R, Garcia-Cao M, Xu S, Kadam R, Goldgur Y, Miller MA, Ouerfelli O, Yang G, Arakawa T, \underline{Albanese SK}, Garland WA, Stoller G, Chaudhary J, Norton L, Soni RK, Philip J, Hendrickson RC, Iavarone A, Dannenberg AJ, \jdc, Pavletich N, Lasorella A, Campochiaro PA, Benezra R}{A Small-Molecule Pan-Id Antagonist Inhibits Pathologic Ocular Neovascularization}{\emph{Cell Reports} 29:62, 2019}{DOI}{https://doi.org/10.1016/j.celrep.2019.08.073}{We report the discovery and characterization of a small molecule, AGX51, with the surprising ability to inhibit the interaction of Id1 with E47, which leads to ubiquitin-mediated degradation of Ids.}

\newarticle{erk-reconstruction}{Sang D, Pinglay S, \underline{Wiewiora RP}, Selvan ME, Lou HJ, \jdc, Turk B, G\"{u}m\"{u}\c{s} Z, and Holt LJ.}{Ancestral reconstruction reveals mechanisms of ERK regulatory evolution}{\emph{eLife}, 2019;8:e38805}{DOI}{https://doi.org/10.7554/eLife.38805.001}{To understand how kinase regulation by phosphorylation emerged, we reconstruct the common ancestor of CDKs and MAPKs, using biochemical experiments and massively parallel molecular simulations to study how a few mutations were sufficient to switch ERK-family kinases from high- to low-autophosphorylation.}

\newarticle{setd8-msm}{\underline{Rafal P. Wiewiora}$^\dag$, Shi Chen$^\dag$, Fanwang Meng, Nicolas Babault, Anqi Ma, Wenyu Yu, Kun Qian, Hao Hu, Hua Zou, Junyi Wang, Shijie Fan, Gil Blum, Fabio Pittella-Silva, \underline{Kyle A. Beauchamp}, Wolfram Tempel, Hualing Jiang, Kaixian Chen, Robert Skene, Y. George Zheng, Peter J. Brown, Jian Jin, \jdc$^\ddag$, and Minkui Luo$^\ddag$}{The dynamic conformational landscapes of the protein methyltransferase SETD8}{\emph{eLife} 8:e45403, 2019}{DOI}{http://doi.org/10.7554/eLife.45403}{In this work, we show how targeted X-ray crystallography using covalent inhibitors and depletion of native ligands to reveal structures of low-population hidden conformations can be combined with massively distributed molecular simulation to resolve the functional dynamic landscape of the protein methyltransferase SETD8 in unprecedented atomistic detail. Using an aggregate of six milliseconds of fully atomistic simulation from Folding\@home, we use Markov state models to illuminate the conformational dynamics of this important epigenetic protein.}

\newarticle{musashi}{Minuesa G, \underline{Albanese SK}, Chow A, Schurer A, Park SM, Rotsides CZ, Taggart J, Rizzi A, Naden LN, Chou T, Gourkanti S, Cappel D, Passarelli MC, Fairchild L, Adura C, Glickman FJ, Schulman J, Famulare C, Patel M, Eibl JK, \underline{Ross GM}, Tan DS, Leslie CS, Beeming T, Golgur Y, \jdc, and Kharas MG. }{Small-molecule targeting of MUSASHI RNA-binding activity in acute myeloid leukemia}{\emph{Nature Communications} 10:2691, 2019}{DOI}{https://doi.org/10.1038/s41467-019-10523-3}{We use absolute alchemical free energy calculations to identify the likely interaction site for a small hydrophobic ligand that shows activity against MUSASHI in AML.}

%{\bf Publications prior to promotion to Associate Member} \\

\newarticle{ddr1-dfg-pmf}{\underline{Hanson SM}$^*$, Georghiou G$^*$, Miller WT, Rest JS, \jdc$\ddag$, and Seeliger MA$\ddag$}{What makes a kinase promiscuous for inhibitors?}{\emph{Cell Chemical Biology} 26:390, 2019}{DOI}{https://doi.org/10.1016/j.chembiol.2018.11.005}{Using a combination of chemogenomics, structural biology, and molecular simulation approaches, we identify a set of human kinases that are especially promiscuous binders of small molecule kinase inhibitors, and show that a prototypical member of this class, DDR1, achieves this promiscuity by virtue of its more stable Asp-DFG-out conformation.}

\newarticle{smarty}{Zanette C, Bannan CC, Bayly CI,\underline{ Fass J}, Gilson MK, Shirts MR, \jdc, Mobley DL}{Toward learned chemical perception of force field typing rules}{\emph{Journal of Chemical Theory and Computation} 15:402, 2019}{DOI}{http://doi.org/10.1021/acs.jctc.8b00821}{We show how machine learning can learn typing rules for molecular mechanics force fields within a Bayesian statistical framework.}

\newarticle{kinase-mutations.jpg}{Hauser K, Negron C, \underline{Albanese SK}, Ray S, Steinbrecher T, Abel R, and \jdc, and Wang L}{Predicting resistance of clinical Abl mutations to targeted kinase inhibitors using alchemical free-energy calculations}{\emph{Communications Biology} 1:70, 2018}{DOI}{https://doi.org/10.1038/s42003-018-0075-x}{We show how alchemical free energy calculations can be used to predict whether clinical point mutations in human kinase domains confer resistance or susceptibility to targeted kinase inhibitors.}

\newarticle{smirnoff}{Mobley DL$\ddag$, Bannan CC, \underline{Rizzi A}, Bayly CI, \jdc, Lim VT, Lim NM, Beauchamp KA, Shirts MR, Gilson MK, and Eastman PK}{Escaping atom types using direct chemical perception with SMIRNOFF v0.1}{\emph{Journal of Chemical Theory and Computation} 14:6076, 2018}{DOI}{http://doi.org/10.1021/acs.jctc.8b00640}{We describe the philosophy behind a modern approach to molecular mechanics forcefield parameterization, and present initial results for the first SMIRNOFF-encoded forcefield: SMIRNOFF99Frosst.}

\newarticle{ops}{\underline{Swenson DWH}, \underline{Prinz JH}, No\'{e} F, \jdc, and Bolhuis PG}{OpenPathSampling: A Python framework for path sampling simulations. I. Basics}{\emph{Journal of Chemical Theory and Computation} 15:813, 2019}{DOI}{https://doi.org/10.1021/acs.jctc.8b00626}{To make powerful path sampling techniques broadly accessible and efficient, we have produced a new Python framework for easily implementing path sampling strategies (such as transition path and interface sampling) in Python. This first publication describes some of the theory and capabilities behind the approach.}

\newarticle{ops}{\underline{Swenson DWH}, \underline{Prinz JH}, No\'{e} F, \jdc, and Bolhuis PG}{OpenPathSampling: A Python framework for path sampling simulations. II. Building and customizing path ensembles and sample schemes}{\emph{Journal of Chemical Theory and Computation} 15:837, 2019}{DOI}{http://doi.org/10.1021/acs.jctc.8b00627}{To make powerful path sampling techniques broadly accessible and efficient, we have produced a new Python framework for easily implementing path sampling strategies (such as transition path and interface sampling) in Python. This second publication describes advanced aspects of the theory and details of how to customize path ensembles.}

\newarticle{sampl6-host-guest-overview}{\underline{Rizzi A}, Murkli S, McNeill J, Yao W, Sullivan M, Gilson MK, Chiu MW, Isaacs L, Gibb BC, Mobley DL$\ddag$, and \jdc$\ddag$}{Overview of the SAMPL6 host-guest binding affinity prediction challenge}{\emph{Journal of Computer Aided Molecular Design} 32:937, 2018.}{DOI}{http://doi.org/10.1007/s10822-018-0170-6}{We present an overview of the host-guest systems and participant performance for the SAMPL6 host-guest blind affinity prediction challenges, assessing how well various physical modeling approaches were able to predict ligand binding affinities for simple ligand recognition problems where receptor sampling and protonation state effects are eliminated due to the simplicity of supramolecular hosts. We find that progress is now stagnated likely due to force field limitations.}

\newarticle{sampl6-pka-measurements}{\underline{Isik M}, Levorse D, \underline{Rustenburg AS}, Ndukwe IE, Wang H, Reibarkh M, Martin GE, Makarov AA, Mobley DL, Rhodes T$\ddag$, and \jdc$\ddag$}{pKa measurements for the SAMPL6 prediction challenge for a set of kinase inhibitor-like fragments}{\emph{Journal of Computer Aided Molecular Design} 32:1117, 2018}{DOI}{http://doi.org/10.1007/s10822-018-0168-0}{The SAMPL5 blind challenge exercises identified neglect of protonation state effects as a major accuracy-limiting factor in physical modeling of biomolecular interactions. In this study, we report the experimental measurements behind a SAMPL6 blind challenges in which we assess the ability of community codes to predict small molecule pKas for small molecule resembling fragments of selective kinase inhibitors.}

\newarticle{kinome-expression-tree}{\underline{Albanese SK}*, \underline{Parton DL}*, \underline{Isik M}$\dag$, \underline{Rodr\'{i}guez-Laureano} L$\dag$, \underline{Hanson SM}, Gradia S, Jeans C, Levinson NM, Seeliger M, and \jdc}{An open library of human kinase domain constructs for automated bacterial expression}{\emph{Biochemistry} 57:4675, 2018}{DOI}{http://doi.org/10.1021/acs.biochem.7b01081}{To establish a tractable experimental system for studying the biophysical determinants of selective kinase inhibitor resistance in clinical cancer mutations, we engineer a library of human kinase domains with useful bacterial expression with phosphatase coexpression. Over 350 requests have been made for these plasmids.}

\newarticle{bayesian-itc}{Nguyen TH, \underline{Rustenburg AS}, Krimmer SG, Zhang H, Clark JD, Novick PA, Branson K, Pande VS, \jdc$\ddag$, MinH DDL$\ddag$}{Bayesian analysis of isothermal titration calorimetry for binding thermodynamics}{\emph{PLOS ONE} 3(9): e0203224, 2018}{DOI}{http://dx.doi.org/10.1371/journal.pone.0203224}{We show how Bayesian inference can produce greatly improved estimates of statistical uncertainty from isothermal titration calorimetry (ITC) experiments, allowing the joint distribution of thermodynamic parameter uncertainties to be inferred.}

\newarticle{idh2-trans-mutations.jpg}{Intlekofer AM*, Shih AH*, Wang B, Nazir A, Rustenburg AS, \underline{Albanese SK}, Patel M, Famulare C, Correa FM, Arcila ME, Taylor J, Tallman MS, Roshal M,  Petsko GA, \jdc, Thompson CB$\ddag$, Levine RL$\ddag$, Stein, EM$\ddag$}{Acquired resistance to IDH inhibition through trans or cis dimer-interface mutations}{\emph{Nature} 559:125, 2018}{DOI}{https://doi.org/10.1038/s41586-018-0251-7}{Clinical double mutations acting in trans in cancer patients receiving IDH2 inhibitors act through a novel biophysical mechanism.}

\newarticle{configuration-space-error}{\underline{Fass J}, Sivak DA, Crooks GE, Beauchamp KA, Leimkuhler B, and \jdc}{Quantifying configuration-sampling error in Langvevin simulations of complex molecular systems}{\emph{Entropy} 20:318, 2018}{DOI}{https://doi.org/10.3390/e20050318}{We address a fundamental question regarding why molecular dynamics simulation works despite the fact that the use of finite timesteps leads to error in the sampled probability densities and populations, demonstrating how to measure configuration-space sampling error for an important class of Langevin integrators widely used in biomolecular simulation.}

\newarticle{saltswap.jpg}{\underline{Ross GA}, \underline{Rustenburg AS}, \underline{Grinaway PB}, \underline{Fass J}, and \jdc}{Biomolecular simulations under realistic salt conditions}{\emph{Journal of Physical Chemistry B} 122:5466, 2018}{DOI}{http://doi.org/10.1021/acs.jpcb.7b11734}{We show how NCMC can be used to implement an efficient osmostat in molecular dynamics simulations to model realistic fluctuations in ion environments around biomolecules, and illustrate how the local salt environment around biological macromolecules can differ substantially from bulk.}

\newarticle{blues}{Gill SC, Lim NM, \underline{Grinaway PB}, \underline{Rustenburg AS}, \underline{Fass J}, \underline{Ross GA}, \jdc, and Mobley DL}{Binding Modes of Ligands Using Enhanced Sampling (BLUES): Rapid Decorrelation of Ligand Binding Modes Using Nonequilibrium Candidate Monte Carlo}{\emph{Journal of Physical Chemistry} 122:5579, 2018}{DOI}{http://doi.org/10.1021/acs.jpcb.7b11820}{Nonequilibrium candidate Monte Carlo can be used to accelerate the sampling of ligand binding modes by orders of magnitude over instantaneous Monte Carlo.}

\newarticle{aurka-phosphorylation}{Ruff EF, Muretta JM, Thompson A, Lake E, Cyphers S, \underline{Albanese SK}, \underline{Hanson SM}, \underline{Behr JM}, Thomas DT, \jdc, and Levinson NM}{A dynamic mechanism for allosteric activation of Aurora kinase A by activation loop phosphorylation}{\emph{eLife} 7:e32766, 2018}{DOI}{https://doi.org/10.7554/eLife.32766}{Through a combination of FRET, IR, and EPR labeling and large-scale molecular dynamics simulations, we show that phorphorylation activates Aurora kinase by a novel mechanism that does not simply correspond to a DFG-out to DFG-in population shift, but rather reorganization of DFG-in subpopulations.}

\newarticle{nanoparticle.jpg}{Shamay Y, Shah J, Tschaharganeh DF, Roxbury D, Budhathoki-Uprety J, I\j{s}ik M, Mizrachi A, Nawaly K, Sugarman JL, Baut E, Neiman MR, Johnson DC, Sridharan R, Chu KL, Rajasekhar VK, \jdc, Lowe SW, and Heller DA}{Quantitative self-assembly prediction yields targeted nanoparticles}{\emph{Nature Materials} 17:361, 2018}{DOI}{http://doi.org/10.1038/s41563-017-0007-z}{A decision tree based on predicted physical properties and and molecular descriptors is capable of predicting the assembly of drug/dye nanoparticles that can be used in tumor-targeted selective kinase inhibitor therapy to minimize on- and off-pathway toxicity.}

\newarticle{aurora.jpg}{Cyphers S, Ruff E, Behr JM, \jdc, and Levinson NM}{A conserved water-mediated hydrogen bond network governs allosteric activation in Aurora\\ kinase A}{\emph{Nature Chemical Biology} 13:402, 2017}{DOI}{http://doi.org/10.1038/nchembio.2296}{Over 50 microseconds of aggregate simulation data on Folding@home reveal a surprisingly stable hydrogen bond network underlies allosteric activation by Tpx2.}

\newarticle{ldha-protonation-state-effects.jpg}{Intlekofer A, Wang B, Liu H, Shah H, Carmona-Fontaine C, \underline{Rustenburg AS}, Salah S, Gunner MR, \jdc, Cross JR, and Thompson CB}{Acidification enhances production of L-2-hydroxyglutarate through alternative substrate use by dehydrogenase enzymes}{\emph{Nature Chemical Biology} 13:494, 2017}{DOI}{http://doi.org/10.1038/nchembio.2307}{At low pH, metabolic enzymes lactate dehydrogenase and malate dehydrogenase undergo shifts in substrate utilization that have high relevance to cancer metabolism due to surprisingly simple protonation state effects.}

\newarticle{freesolv-update}{Matos GDR, Kyu DY, Loeffler HH, \jdc, Shirts MR, and Mobley DL}{Approaches for calculating solvation free energies and enthalpies demonstrated with an update of the FreeSolv database}{\emph{Journal of Chemical \& Engineering Data} 62:1559, 2017}{DOI}{http://dx.doi.org/10.1021/acs.jced.7b00104}{We review alchemical approaches to computing solvation free energies and update FreeSolv---the most popular database of hydration free energies of neutral molecules---with more computed and experimental properties.}

\newarticle{openmm7-logo.jpg}{Eastman P, Swails J, \jdc, McGibbon RT, Zhao Y, \underline{Beauchamp KA}, Wang LP, Simmonett AC, Harrigan MP, Brooks BR, and Pande VS}{OpenMM 7: Rapid development of high performance algorithms for molecular dynamics}{\emph{PLoS Computational Biology} 13:e1005659, 2017}{DOI}{https://doi.org/10.1371/journal.pcbi.1005659}{The latest version of the GPU-accelerated molecular simulation OpenMM features a variety of incredibly flexible but fast tools for rapidly prototyping, evaluating, and deploying new simulation algorithms.}

\newarticle{ensembler.pdf}{\underline{Parton DL}, \underline{Grinaway PB}, \underline{Hanson SM}, \underline{Beauchamp KA}, and \jdc}{Ensembler: Enabling high-throughput molecular simulations at the superfamily scale}{\emph{PLoS Computational Biology} 12:e1004728, 2016}{DOI}{http://dx.doi.org/10.1371/journal.pcbi.1004728} {We demonstrate a new tool that enables---for the first time---massively parallel molecular simulation studies of biomolecular dynamics at the superfamily scale, illustrating its application to protein tyrosine kinases, an important class of drug targets in cancer.}

\newarticle{mtor-mutations.pdf}{Xu~J, Pham~CG, \underline{Albanese~SK}, Dong~Y, Oyama~T, Lee~CH, Rodrik-Outmezguine~V, Yao~Z, Han~S, Chen~D, \underline{Parton~DL}, \jdc, Rosen~N, Cheng~EH, and Hsieh~JJ}{Mechanistically distinct cancer-associated mTOR activation clusters predict sensitivity to rapamycin}{\emph{Journal of Clinical Investigation} 126:3529, 2016}{DOI}{https://doi.org/10.1172/JCI86120}{We use massively parallel distributed molecular simulations on Folding@home to probe the mechanism activating mutations of the mTOR kinase identified in clinical populations.}

\newarticle{automatic-equilibration-detection.jpg}{\jdc}{A simple method for automated equilibration detection in molecular simulations}{\emph{Journal of Chemical Theory and Computation} 12:1799, 2016}{DOI}{http://dx.doi.org/10.1021/acs.jctc.5b00784}{We present a simple approach to automatically determining the equilibrated region of a molecular simulation, a longstanding challenge formerly without a good solution.}

\newarticle{sampl5-logd.pdf}{\underline{Rusteburg AS}, Dancer J, Lin B, Ortwine D, Mobley DL, and \jdc}{Measuring cyclohexane-water distribution coefficients for the SAMPL5 challenge}{\emph{Journal of Computer Aided Molecular Design}, 30:945, 2016}{DOI}{http://dx.doi.org/10.1007/s10822-016-9971-7}{To test the accuracy of physical modeling techniques in predicting free energies of transfer between aqueous and nonpolar solvents, we worked with Genentech to develop a new protocol to measure cyclohexane-water distribution coefficients for 53 druglike compounds at pH 7.4, fielding a blind community challenge as part of the \href{https://drugdesigndata.org/about/sampl5}{SAMPL5 exercise}. A special issue of JCAMD was published with 16 papers describing various approaches used by participants to predict this data and understand their failures.}

\newarticle{thermoml-benchmark.jpg}{\underline{Beauchamp~KA}, \underline{Behr~JM}, \underline{Rustenburg~AS}, Bayly~CI, Kroenlein~K, and \jdc. }{Towards automated benchmarking of atomistic forcefields: Neat liquid densities and static dielectric constants from the {ThermoML} data archive}{\emph{Journal of Physical Chemistry B} 199:12912, 2015}{DOI}{http://t.co/xhoJzSQKo0}{Molecular mechanics forcefields are critical to computer-guide drug design, but the benchmarking and improvement of these forcefields has been hindered by the lack of high-quality machine-readable physical property datasets. We show how the NIST-curated ThemoML Archive, which stores physical property data in an IUPAC-standard XML format, can eliminate these roadblocks and reveal issues with current generation forcefields.}

\newarticle{L-2HG.pdf}{Intlekofer AM, Dematteo RG, Venetti S, Finley LWS, Lu Chao, Judkins AR, \underline{Rutenburg AS}, \underline{Grinaway PB}, \jdc, Cross JR, and Thompson CB}{Hypoxia introduces production of {L-2-Hydroxyglutarate}}{\emph{Cell Metabolism} 22:1--8, 2015}{DOI}{http://dx.doi.org/10.1016/j.cmet.2015.06.023}{Molecular docking is used to demonstrate the potential for alternative substrate usage by isocitrate dehydrogenases under hypoxic conditions in cancer.}

\newarticle{timestep-rescaling.pdf}{Sivak DA, \jdc, and Crooks GE}{Time step rescaling recovers continuous-time dynamical properties for discrete-time Langevin integration of nonequilibrium systems}{\emph{Journal of Physical Chemistry B}, 118:6466--6474, 2014.  William C.~Swope Festschrift}{DOI}{http://dx.doi.org/10.1021/jp411770f}{We derive a simple, easy-to-implement Langevin integrator that has universally useful properties in molecular simulations.}

\newarticle{spectral-rate-theory.pdf}{\underline{Prinz J-H}, \jdc, and No\'{e} F}{Spectral rate theory for two-state kinetics}{\emph{Physical Review X} 4:011020, 2014}{DOI}{http://dx.doi.org/10.1103/PhysRevX.4.011020}{We present a new mathematical framework for unifying various two-state rate theories presented in the physical chemistry literature over many decades, and provide a quantitative way to measure reaction coordinate quality.}

\newarticle{identifying-ligand-binding-sites.pdf}{Wang K, \jdc, Yang Y, and Shirts MR}{Identifying ligand binding sites and poses using GPU-accelerated Hamiltonian replica exchange molecular dynamics}{\emph{Journal of Computer Aided Molecular Design} 27:989--1007, 2013}{DOI}{http://dx.doi.org/10.1007/s10822-013-9689-8}{We show how bound ligand poses can be identified even when the location of the binding sites are unknown using the machinery of alchemical modern free energy calculations on graphics processors.}

\newarticle{iamoeba-water}{Wang L-P, Head-Gordon TL, Ponder JW, Ren P, \jdc, Eastman PK, Martinez TJ, and Pande VS}{Systematic improvement of a classical molecular model of water}{\emph{Journal of Physical Chemistry B} 117:9956--9972, 2013}{DOI}{http://dx.doi.org/10.1021/jp403802c}{Water is the most important molecule in biology, and accurate treatment of its interactions is critical to accurate modeling for drug discovery. While polarizable models of water can achieve very high accuracies, they are both difficult to parameterize and expensive to employ. Here, we show how a high quality inexpensive polarizable model of liquid water can be derived using an automated parameterization engine.}

\newarticle{nonequilibrium-fluctuation-theorems}{Sivak DA, \jdc, and Crooks GE}{Using nonequilibrium fluctuation theorems to understand and correct errors in equilibrium and nonequilibrium discrete Langevin dynamics simulations}{\emph{Physical Review X} 3:011007, 2013}{DOI}{http://dx.doi.org/10.1103/PhysRevX.3.011007}{All molecular dynamics simulations introduce error into the sampled distribution by virtue of the finite timestep used to integrate the equations of motion on a digital computer. While traditional approaches to analyzing this error are extremely complicated, we show how interpreting finite-timestep integrators as a form of nonequilibrium driving leads to simple, straightforward schemes for assessing the impact of these errors, as well as correcting for them.}

\newarticle{openmm-logo.pdf}{Eastman P, Friedrichs MS, \jdc, Radmer RJ, Bruns CM, Ku JP, Beauchamp KA, Lane TJ, Wang L, Shukla D, Tye T, Houston M, Stich T, Klein C, Shirts MR, and Pande VS}{OpenMM 4: A reusable, extensible, hardware independent library for high performance molecular simulation}{\emph{Journal of Chemical Theory and Computation} 9:461, 2012}{DOI}{http://dx.doi.org/10.1021/ct300857j}{Inexpensive consumer GPUs promise a 100-fold increase in simulation power by problems that can effectively exploit their highly specialized structure. Here, we describe the latest advances in an extremely high performance, open-source, extensible GPU-accelerated library and toolkit for molecular simulation.}

\newarticle{constant-force-feedback.pdf}{Elms PJ, \jdc, Bustamante CJ, Marqusee S}{The limitations of constant-force-feedback experiments}{\emph{Biophysical Journal} 103:1490, 2012}{DOI}{http://dx.doi.org/10.1016/j.bpj.2012.06.051}{Popular constant-force-feedback single-molecule experiments can cause severe artifacts in single-molecule force spectroscopy data.  We demonstrate a simple alternative that eliminates these artifacts.}

\newarticle{molten-globule-state.pdf}{Elms PJ, \jdc, Bustamante C, Marqusee S}{The molten globule state is unusually deformable under mechanical force}{\emph {Proceedings of the National Academy of Sciences} 109:3796, 2012}{DOI}{http://dx.doi.org/10.1073/pnas.1115519109}{We measure the physical properties of the molten globule state of apo-myoglobin, and show that it is unusually deformable compared to typical protein native states.}

\newarticle{experimental-observables.pdf}{Pitera JW and \jdc}{On the use of experimental observations to bias simulated ensembles}{\emph{Journal of Chemical Theory and Computation} 8:3445, 2012}{DOI}{http://dx.doi.org/10.1021/ct300112v}{We show how the concept of maximum entropy can be used to recover unbiased conformational distributions from experimental data, and how this concept relates to the popular `ensemble refinement' schemes for NMR data analysis.}

\newarticle{ribosome-modulates.pdf}{Kaiser CM, Goldman DH, \jdc, Tinoco I, Jr., and Bustamante C}{The ribosome modulates nascent protein folding}{\emph{Science} 334:1723, 2011}{DOI}{http://dx.doi.org/10.1126/science.1209740}{Using single-molecule force spectroscopy, we show how the ribosome itself modulates the folding dynamics of nascent protein chains emerging from the exit tunnel.}

\newarticle{splitting-probabilities.pdf}{\jdc and Pande VS}{Splitting probabilities as a test of reaction coordinate choice in single-molecule experiments}{\emph{Physical Review Letters} 107:098102, 2011}{DOI}{http://dx.doi.org/10.1103/PhysRevLett.107.098102}{We demonstrate a simple test for identifying poor reaction coordinates in single-molecule experiments.}

\newarticle{itc-enthalpogram.pdf}{Tellinghuisen JT and \jdc}{Systematic errors in isothermal titration calorimetry: Concentrations and baselines}{\emph{Analytical Biochemistry} 414:297, 2011}{DOI}{http://dx.doi.org/10.1016/j.ab.2011.03.024}{A word of caution about large errors in isothermal titration calorimetry measurements arising from ligand concentration errors.}

\newarticle{multiple-time-slices.pdf}{Minh DDL, \jdc}{Estimating equilibrium ensemble averages using multiple time slices from driven nonequilibrium processes: Theory and application to free energies, moments, and thermodynamic length in single-molecule pulling experiments}{\emph{Journal of Chemical Physics} 134:024111, 2011}{DOI}{http://dx.doi.org/10.1063/1.3516517}{We derive a new estimator for estimating equilibrium expectations from nonequilibrium experiments, and show how it can be used to estimate a variety of useful quantities in simulated single-molecule force spectroscopy experiments.}

\newarticle{ncmc.pdf}{Nilmeier JP, Crooks GE, Minh DDL, and \jdc}{Nonequilibrium candidate Monte Carlo is an efficient tool for equilibrium simulation}{\emph{Proceedings of the National Academy of Sciences} 108:E1009, 2011}{DOI}{http://dx.doi.org/10.1073/pnas.1106094108}{We present a significant generalization of Monte Carlo methods that provide an enormously useful tool for enhancing the efficiency of molecular simulations and enabling molecular design.}

\newarticle{parallel-tempering.pdf}{Prinz J-H, \jdc, Pande VS, Smith JC, and No\'{e} F}{Optimal use of data in parallel tempering simulations for the construction of discrete-state Markov models of biomolecular dynamics}{\emph{Journal of Chemical Physics} 134:244108, 2011}{DOI}{http://dx.doi.org/10.1063/1.3592153}{We demonstrate how multitemperature data from parallel tempering simulations can be used to construct fully temperature-dependent models of the dynamics of biomolecular systems.}

\newarticle{dynamical-reweighting.pdf}{\jdc, Swope WC, No\'{e} F, Prinz J-H, Shirts MR, and Pande VS}{Dynamical reweighting: Improved estimates for dynamical properties from simulations at multiple temperatures}{\emph{Journal of Chemical Physics} 134:244107, 2011}{DOI}{http://dx.doi.org/10.1063/1.3592152}{We describe how reweighing techniques can provide optimal estimates of temperature-dependent dynamical properties from simulations conducted at multiple temperatures.}

\newarticle{dynamical-fingerprints.pdf}{No\'{e} F, Doose S, Daidone I, L\"{o}llmann M, Sauer M, \jdc, and Smith JC}{Dynamical fingerprints: A theoretical framework for understanding biomolecular processes by combination of simulation and kinetic experiments}{\emph{Proceedings of the National Academy of Sciences} 108:4822, 2011}{DOI}{http://dx.doi.org/10.1073/pnas.1004646108}{We present a new framework for comparing essential features of the dynamics between experiment and simulation to identify the kinetics processes contributing to individual relaxation timescales in perturbation-response or correlation spectroscopy experiments.}

\newarticle{gibbs-sampling.pdf}{\jdc and Shirts MR}{Replica exchange and expanded ensemble simulations as Gibbs sampling:\\ Simple improvements for enhanced mixing}{\emph{Journal of Chemical Physics} 135:194110, 2011}{DOI}{http://dx.doi.org/10.1063/1.3660669}{We show how a simple change to the way exchanges are handled in the popular replica-exchange simulation methodology can astronomically increase efficiency at no increase in computational cost.}

\newarticle{current-status-amoeba.pdf}{Ponder JW, Wu C, Ren P, Pande VS, \jdc, Mobley DL, Schnieders MJ, Haque I, Lambrecht DS, DiStasio RA Jr., Head-Gordon M,  Clark GNI, Johnson ME, and Head-Gordon T}{Current status of the AMOEBA polarizable force field}{\emph{Journal of Physical Chemistry B} 114:2549, 2010}{DOI}{http://dx.doi.org/10.1021/jp910674d}{The AMOEBA polarizable force field is able to reproduce a diverse set of physical chemical phenomenon to high accuracy.}

\newarticle{observable-uncertainty.pdf}{\jdc and No\'e F}{Probability distributions of molecular observables computed from Markov models.\\ II.~Uncertainties in observables and their time-evolution}{\emph{Journal of Chemical Physics} 133:105102, 2010}{DOI}{http://dx.doi.org/10.1063/1.3463406}{A simple Bayesian approach for the modeling of statistical uncertainties in kinetic and equilibrium quantities computed from Markov state models of biomolecular dynamics.}

\newarticle{pcna-cover.pdf}{Adelman JL, \jdc, Kuo IW, Miller TF, and Barsky D}{The mechanical properties of PCNA: Implications for the loading and function of a DNA sliding clamp}{\emph{Biophysical Journal} 98:3062, 2010}{DOI}{http://dx.doi.org/10.1016/j.bpj.2010.03.056}{Molecular simulations of the PCNA clamp responsible for DNA polymerase processivity show a surprisingly small energetic penalty for the deformation required for clamp loading.  Featured on issue cover.}

\newarticle{bayesian-comparison-markov-models.pdf}{Bacallado S, \jdc, and Pande VS}{Bayesian comparison of Markov models of molecular dynamics with detailed balance constraint}{\emph{Journal of Chemical Physics} 131:045106, 2009}{DOI}{http://dx.doi.org/10.1063/1.3192309}{A Bayesian scheme for comparing state space decompositions for Markov state models of biomolecular dynamics that incorporates the fact that physical systems must obey detailed balance.  This paper utilizes recent results from Markov chain theory on edge-reinforced random walks.}

\newarticle{optimal-estimates-nonequilibrium.pdf}{Minh DDL, \jdc}{Optimal estimators and asymptotic variances for nonequilibrium path-ensemble averages}{\emph{Journal of Chemical Physics} 131:134110, 2009}{DOI}{http://dx.doi.org/10.1063/1.3242285}{We derive an optimal estimator and corresponding statistical uncertainties for inferring expectations of bidirectional nonequilibrium processes.  These estimators have widespread applicability in single-molecule biophysical force-spectroscopy experiments and nonequilibrium molecular simulations.}

\newarticle{mbar-equation.pdf}{Shirts MR, \jdc}{Statistically optimal analysis of samples from multiple equilibrium states}{\emph{Journal of Chemical Physics} 129:124105, 2008}{DOI}{http://dx.doi.org/10.1063/1.2978177}{We present a highly general, statistically optimal approach for producing estimates of free energies and equilibrium expectations from multiple simulations that provably extracts all useful information from the data.}

\newarticle{small-molecule-solvation.pdf}{Nicholls A*, Mobley DL*, Guthrie JP, \jdc, and Pande VS}{Predicting small-molecule solvation free energies: A blind challenge test for computational chemistry}{\emph {Journal of Medicinal Chemistry} 51:769, 2008}{DOI}{http://dx.doi.org/10.1021/jm070549+}{A blind evaluation of the accuracy of alchemical free energy methods for computing gas-to-water transfer free energies (solvation free energies) of small molecules demonstrates that modern forcefields are likely sufficiently accurate to be useful in drug design.}

\newarticle{entropy-solvation.pdf}{Mobley DL, Dill KA, and \jdc}{Treating entropy and conformational changes in implicit solvent simulations of small molecules}{\emph{Journal of Physical Chemistry B} 112:938, 2008}{DOI}{http://dx.doi.org/10.1021/jp0764384}{An quantitative examination of how much conformational entropy contributes to hydration free energies of small molecules, with implications for ligand binding.}

\newarticle{long-range-dispersion-correction.pdf}{Shirts MR*, Mobley DL*, \jdc, and Pande VS}{Accurate and efficient corrections for missing dispersion interactions in molecular simulations}{\emph{Journal of Physical Chemistry B} 111:13052, 2007}{DOI}{http://dx.doi.org/10.1021/jp0735987}{We identify a major source of systematic error in absolute alchemical free energy calculations of ligand binding and show how a simple procedure can inexpensively and accurately eliminate it.}

\newarticle{charge-models.pdf}{Mobley DL, Dumont E, \jdc, Bayly CI, Cooper MD, and Dill KA}{Comparison of charge models for fixed-charge force fields: Small-molecule hydration free energies in explicit solvent}{\emph{Journal of Physical Chem B} 111:2242, 2007}{DOI}{http://dx.doi.org/10.1021/jp0667442}{We compare a number of popular methods for deriving charge models for small molecules, deriving lessons about best practices for accurate simulations.}

\newarticle{t4-lysozyme-l99a.pdf}{Mobley DL, Graves AP, \jdc, McReynolds AC, Shoichet BK, and Dill KA}{Predicting absolute ligand binding free energies to a simple model site}{\emph{Journal of Molecular Biology} 371:1118, 2007}{DOI}{http://dx.doi.org/10.1016/j.jmb.2007.06.002}{We show how alchemical free energy calculations are capable of accurate blind prediction of small-molecule binding affinities to a simple model protein binding site.}

\newarticle{t4-lysozyme-val111}{Mobley DL, \jdc, and Dill KA}{Confine-and-release method: Obtaining correct binding free energies in the presence of protein conformational change}{\emph{Journal of Chemical and Theoretical Computation} 3:1231, 2007}{DOI}{http://dx.doi.org/10.1021/ct700032n}{We present a general scheme for obtaining correct ligand binding affinities when protein conformational change is implicated in ligand binding.}

\newarticle{automatic-state-decomposition-trpzip2.jpg}{\jdc*, Singhal N*, Swope WC, Pitera JW, Pande VS, and Dill KA}{Automatic discovery of metastable states for the construction of Markov models of macromolecular conformational dynamics}{\emph{Journal of Chemical Physics} 126:155101, 2007}{DOI}{http://dx.doi.org/10.1063/1.2714538}{Proposing one of the first automated algorithms for discovering kinetically metastable states of biomolecules from molecular simulations, this paper shows how many biomolecules can possess numerous distinct long-lived conformational states even though the the equilibrium populations of these states may too small for standard structural biology techniques to detect.}

\newarticle{zipping-assembly.pdf}{Ozkan SB, Wu GA, \jdc, and Dill KA}{Protein Folding by Zipping and Assembly}{\emph{Proceedings of the National Academy of Sciences} 104:11987, 2007}{DOI}{http://dx.doi.org/10.1073/pnas.0703700104}{A review of the utility of the proposed zipping and assembly mechanism for the concomitant formation of secondary and tertiary structure in protein folding for predicting folding pathways and native structures.}

\newarticle{alanine-dipeptide-2dpmf.pdf}{\jdc, W. C. Swope, J. W. Pitera, C. Seok, and K. A. Dill}{Use of the weighted histogram analysis method for the analysis of simulated and parallel tempering simulations}{\emph{Journal of Chemical Theory and Computation} 3:26, 2007}{DOI}{http://dx.doi.org/10.1021/ct0502864}{The weighted histogram analysis method (WHAM), a mainstay of molecular dynamics simulation analysis, is thoroughly explained and modernized for the analysis of simulated and parallel tempering simulation data.}

\newarticle{orientational-restraints.pdf}{Mobley DL, \jdc, and Dill KA}{On the use of orientational restraints and symmetry corrections in alchemical free energy calculations}{\emph{Journal of Chemical Physics} 125:084902, 2006}{DOI}{http://dx.doi.org/10.1063/1.2221683}{We illustrate how orientational restraints can be used to greatly reduce the computational effort in alchemical calculations of ligand binding free energies, and clarify how symmetry corrections are necessary when molecules contain symmetric or pseudosymmetric substituents.}

\newarticle{alanine-dipeptide.pdf}{\jdc, Swope WC, Pitera JW, and Dill KA}{Long-time protein folding dynamics from short-time molecular dynamics simulations}{\emph{Multiscale Modeling and Simulation} 5:1214, 2006}{DOI}{http://dx.doi.org/10.1137/06065146X}{We show how the long-time dynamics of biomolecular systems can be recapitulated from statistics collected from short molecular simulations sampling transitions between kinetically metastable states.}

\newarticle{moped.pdf}{Seok C, Rosen JB, \jdc, Dill KA}{MOPED: Method for optimizing physical energy parameters using decoys}{\emph{Journal of Computational Chemistry} 24:89, 2003}{DOI}{http://dx.doi.org/10.1002/jcc.10124}{We propose a new way to optimize parameters for a physical energy function using decoy structures for protein folding studies.}

\newarticle{odcase.pdf}{Lee TS$^*$, Chong LT$^*$, \jdc, and Kollman PA}{An alternative explanation for the catalytic proficiency of orotidine 5'-phosphate decarboxylase}{\emph{Journal of the American Chemical Society} 123:12837, 2001}{DOI}{http://dx.doi.org/10.1021/ja011096f}{A combined QM and MD analysis of potential plausible mechanisms to explain the enormous catalytic acceleration of one of the most proficient enzymes known.}

\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% OTHER ACTIVITIES
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Peer reviewer for scientific journals}

Bioinformatics,
Biopolymers,
Chemical Physics,
Computation,
Drug Discovery Today,
European Biophysics Journal,
Entropy,
International Journal of Molecular Sciences,
Journal of the American Chemical Society,
Journal of Chemical Theory and Computation,
Journal of Computer-Aided Molecular Design,
Journal of Computational Chemistry,
Journal of Computational Physics,
Journal of Physical Chemistry,
Journal of Physical Chemistry Letters,
Molecular Physics,
Multiscale Modeling \& Simulation,
Nature Chemistry,
Nature Communications,
Nature Physics,
Pacific Symposium in Biocomputing,
PLoS Computational Biology,
PLoS One,
Proceedings of the National Academy of Sciences,
Science,
Scientific Reports,
Structure

%\vfill{}
%\hrulefill

% FILL IN THE FULL URL TO YOUR CV
%\begin{center}
%{\footnotesize 
%Latest version available online at \href{http://www.choderalab.org}{http://choderalab.org} --- this version current as of \today.\\
%The template for this CV is available at \href{https://github.com/jchodera/latex-cv}{https://github.com/jchodera/latex-cv}}
%\end{center}

%\eject


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% ACADEMIC SERVICE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\section*{Service}

%\talk{2012}{MSKCC}Established High Performance GPU Research Computing Resource \\ \vspace{0.5ex}

%\talk{2018--}{~}Open Force Field Consortium Governing Board \\ {\small \url{http://openforcefield.org/consortium}}\vspace{0.5ex}

%\talk{2012--2015}{~}MSKCC High Performance Computing Steering Committee \\ {\small Member, Co-founder}


%\vspace{1.5ex}
%
%\color{red}
%What else should be listed here? 
%\begin{itemize}
%\item Participation in graduate programs? 
%\item Departmental hiring committees? 
%\item CBM curriculum committees? 
%\item ACE and dissertation committees?
%\end{itemize}
%\color{black}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% GRADUATE PROGRAMS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Graduate Programs}

\talk{}{2013--}Program in Physiology, Biophysics, and Systems Biology (PBSB)\vspace{1ex}

\talk{}{2013--}Tri-Institutional PhD Program in Chemical Biology (TPCB)\vspace{1ex}

\talk{}{2013--}Tri-Institutional Program in Computational Biology and Medicine (CBM)\vspace{1ex}

\talk{}{2015--}Gerstner Sloan Kettering Graduate Program (GSK)\vspace{1ex}

%\eject

%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% TEACHING
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Teaching}

\talk{Summer 2017}{Virginia Tech}MolSSI Software Summer School Instructor \\ {\small Molecular Software Sciences Institute}\vspace{0.5ex}

\talk{2015--2018}{Tri-I}CBM Journal Club Moderator \\ {\small Tri-I Computational Biology and Medicine Program}\vspace{0.5ex}

\talk{2018--}{WCMC}BCMB Biochemistry \\ {\small WCMC Graduate School of Medical Sciences (statistical mechanics and thermodynamics unit, 3.0 lecture hours)}\vspace{0.5ex}

\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% GRANT REVIEWS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Grant Reviews and Study Sections}

\subsection*{NIH}

Macromolecular Structure and Function B [MSFB] (\emph{early career reviewer})

\subsection*{NSF}

Ad hoc reviewer and virtual panel member for Chemical Theory, Models, and Computational Methods (CTMC)

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% CONFERENCES ORGANIZED
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Conferences organized}

\talk{2020--}{\,}Pause due to COVID-19 pandemic \vspace{0.5ex}

\talk{Sep 2019}{London, UK}CompBioMed 2019\\ {\small IET London}\vspace{0.5ex}

\talk{May 2019}{Berlin, Germany}MolKin 2019: Molecular Kinetics: Sampling, Design and Machine Learning\\ {\small Freie Universit\"{a}t Berlin, Germany}\vspace{0.5ex}

\talk{May 2019}{G\"{o}ttingen, Germany}Alchemical Free Energy Methods Workshop \\ {\small Novartis, Cambridge, MA}\vspace{0.5ex}

\talk{May 2018}{Boston, MA}Free Energy Methods and Molecular Kinetics in Drug Design Workshop \\ {\small Novartis, Cambridge, MA}\vspace{0.5ex}

\talk{Sep 2017}{San Francisco, CA}MolSSI Workshop on Workflows in Biomolecular Simulation \\ {\small Autodesk}\vspace{0.5ex}

\talk{Jan 2017}{Berkeley, CA}SMML//2017: Statistical Mechanics // Machine Learning \\ {\small University of California, Berkeley}\vspace{0.5ex}

\talk{May 2016}{Boston, MA}Free Energy Methods in Drug Design Workshop \\ {\small Vertex Pharmaceuticals, Boston, MA}\vspace{0.5ex}

\talk{May 2016}{Cambridge, MA}Markov State Models in Drug Design Workshop \\ {\small Novartis, Cambridge, MA}\vspace{0.5ex}

\talk{Sep 2015}{Berlin, Germany}World Molecular Kinetics Workshop 2015 \\ {\small Freie Universit\"{a}t Berlin, Germany} \vspace{0.5ex}

\talk{May 2014}{Boston, MA}Free Energy Methods in Drug Design Workshop \\ {\small Vertex Pharmaceuticals, Boston, MA}\vspace{0.5ex}

\talk{Sep 2013}{Berlin, Germany}World Molecular Kinetics Workshop 2013 \\ {\small Freie Universit\"{a}t Berlin, Germany} \vspace{0.5ex}

\talk{May 2012}{Cambridge, MA}Free Energy Methods in Drug Design Workshop \\ {\small Vertex Pharmaceuticals, Cambridge, MA}\vspace{0.5ex}

\talk{Sep 2011}{Berlin, Germany}World Molecular Kinetics Workshop 2011 \\ {\small Freie Universit\"{a}t Berlin, Germany} \vspace{0.5ex}

\talk{May 2010}{Cambridge, MA}Free Energy Methods in Drug Design Workshop \\ {\small Vertex Pharmaceuticals, Cambridge, MA}\vspace{0.5ex}

\talk{May 2009}{Berlin, Germany}World Molecular Kinetics Workshop 2009 \\ {\small Freie Universit\"{a}t Berlin, Germany}

%% TALKS

%\section*{Invited presentations}
%
%\numberedtalk{Jul 2013}{Mount Snow, VT}Computer Aided Drug Discovery GRC\\ {\small Experimental (T)error: Strategies for coping with data in an uncertain world} \vspace{0.5ex}
%
%\numberedtalk{Jul 2013}{Snowmass, CO}Free energy workshop.\\ {\small Redesigning drug design} \vspace{0.5ex}
%
%\numberedtalk{Apr 2013}{Merced}Chemistry Departmental Seminar, UC Merced.\\ {\small Redesigning drug design} \vspace{0.5ex}
%
%\numberedtalk{Jan 2013}{College Park}Statistical Mechanics Seminar Series, University of Maryland.\\ {\small New applications of nonequilibrium fluctuation theorems in single-molecule biophysics and molecular simulation} \vspace{0.5ex}
%
%\numberedtalk{Sep 2012}{Marconi Center}UC Berkeley Biophysics Retreat.\\ {\small Redesigning drug design} \vspace{0.5ex}
%
%\numberedtalk{Sep 2012}{Skytop}MSKCC cBio Retreat.\\ {\small Redesigning drug design} \vspace{0.5ex}
%
%\numberedtalk{Sep 2012}{Zurich}CECAM Protein Folding Workshop, ETH Zurich.\\ {\small New techniques for extracting insight from single-molecule force spectroscopy} \vspace{0.5ex}
%
%\numberedtalk{Feb 2012}{Schr\"dingier}Schr\"{o}dinger, New York\\ {\small Redesigning drug design} \vspace{0.5ex}
%
%\numberedtalk{Feb 2012}{Berlin}Freie Universit\"{a}t Berlin, MATHEON\\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\numberedtalk{Jan 2012}{Berkeley}Physics Departmental Seminar, UC Berkeley \\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\numberedtalk{Jan 2012}{Berkeley}Berkeley Mini Stat Mech Meeting\\ {\small Unraveling the statistical mechanics of biomolecules with force spectroscopy} \vspace{0.5ex}
%
%\numberedtalk{Sep 2011}{Berlin}World Molecular Kinetics Workshop, Berlin\\ {\small New techniques for nonequilibrium force spectroscopy} \vspace{0.5ex}
%
%\numberedtalk{Aug 2011}{Johns Hopkins}Johns Hopkins University\\ {\small Toward multiscale modeling of the effects of small molecules on cellular pathways}\vspace{0.5ex}
%
%\numberedtalk{Mar 2011}{UCSF}University of California, San Francisco\\ {\small Pushing the boundaries of single-molecule force spectroscopy} \vspace{0.5ex}
%
%\numberedtalk{Jan 2011}{MIT}Physical Chemistry Seminar, MIT \\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\numberedtalk{Jul 2010}{New York}Schr\"{o}dinger, New York \\ {\small Alchemical free energy methods for drug discovery} \vspace{0.5ex}
%
%\numberedtalk{Jun 2010}{Edinburgh}Multiscale Molecular Modeling (M3) meeting, Edinburgh \\ {\small Optimal estimates of dynamical properties} \vspace{0.5ex}
%
%\eject
%
%\numberedtalk{Jan 2010}{Berkeley}Berkeley Mini Stat Mech Meeting, TPS Satellite Meeting, Berkeley \\ {\small Optimal estimation from multiple path sampling simulations} \vspace{0.5ex}
%
%\numberedtalk{Dec 2009}{Cambridge}Vertex Pharmaceuticals, Cambridge MA \\ {\small Entropy-enthalpy compensation: Not even wrong?} \vspace{0.5ex}
%
%\numberedtalk{Apr 2009}{Purdue}Purdue University\\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\numberedtalk{Apr 2009}{Notre Dame}University of Notre Dame\\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\numberedtalk{Apr 2009}{Berlin}Freie Universit\"{a}t Berlin, MATHEON\\ {\small Estimators for equilibrium and nonequilibrium experiments} \vspace{0.5ex}
%
%\numberedtalk{Mar 2009}{Miami}SIAM National Meeting, Miami\\ {\small } {\small Bridging between atomistic simulation and physical experiments with master equation models} \vspace{0.5ex}
%
%\numberedtalk{Feb 2009}{UCLA}IPAM Rare Events in High-Dimensional Systems, UCLA\\ {\small Bridging between atomistic simulation and physical experiments with master equation models}\vspace{0.5ex}
%
%\numberedtalk{Jul 2008}{Telluride}Telluride meeting on Enhanced Sampling Methods\\ {\small Statistically optimal estimates from multiple equilibrium samples} \vspace{0.5ex}
%
%\numberedtalk{Jun 2008}{Banff}BIRS Workshop on Mathematical Methods for Free Energy Calculations in Molecular Systems\\ {\small Statistically optimal estimates from equilibrium and nonequilibrium measurements} \vspace{0.5ex}
%
%\numberedtalk{Apr 2008}{Stanford}Gromacs Workshop, Stanford\\ {\small Alchemical free energy calculations with {\tt gromacs}}\vspace{0.5ex}
%
%\numberedtalk{Feb 2008}{Vienna}ESI Workshop on Metastability and Rare Events in Complex Systems, Vienna\\ {\small Master equation models of protein folding and dynamics from atomistic molecular dynamics simulation} \vspace{0.5ex}
%
%\numberedtalk{Aug 2007}{Boston}ACS National Meeting, Boston\\ {\small Bridging timescales between atomistic simulation and experiments} \vspace{0.5ex}
%
%%\numberedtalk{Mar 2007}{Chicago}ACS National Meeting\\ {\small Accuracy and reliability in molecular simulation. \emph{(Substituting for Vijay S. Pande)}} \vspace{0.5ex}
%
%\numberedtalk{Sep 2006}{Heidelberg}MMS06 Workshop, Heidelberg\\ {\small Describing peptide and protein folding dynamics by discrete-state master equation models} \vspace{0.5ex}
%
%\numberedtalk{Dec 2003}{San Diego}Burroughs Wellcome Fund, Interfaces in Science Program retreat, San Diego\\ {\small Constructing master equation models of protein dynamics from atomistic simulation} \vspace{0.5ex}
%
%\numberedtalk{Mar 2003}{New York}ACS National Meeting, New York\\ {\small Extracting dynamical information from parallel tempering simulations}
%
%%%%%%%%%%%

%\talk{Sep 2011}{Berlin}World Molecular Kinetics Workshop\\ {\small New techniques for nonequilibrium force spectroscopy} \vspace{0.5ex}
%
%\talk{Aug 2011}{Johns Hopkins}Invited talk, Johns Hopkins University\\ {\small Toward multiscale modeling of the effects of small molecules on cellular pathways}\vspace{0.5ex}
%
%\talk{Mar 2011}{UCSF}Invited talk, UCSF\\ {\small Pushing the boundaries of single-molecule force spectroscopy} \vspace{0.5ex}
%
%\eject
%
%\talk{Jan 2011}{MIT}Physical Chemistry Seminar, MIT \\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\talk{Jul 2010}{New York}Invited talk, Schr\"{o}dinger \\ {\small Alchemical free energy methods for drug discovery} \vspace{0.5ex}
%
%\talk{Jun 2010}{Edinburgh}Multiscale Molecular Modeling (M3) meeting \\ {\small Optimal estimates of dynamical properties} \vspace{0.5ex}
%
%\talk{Jan 2010}{Berkeley}Berkeley Mini Stat Mech Meeting, TPS Satellite Meeting \\ {\small Optimal estimation from multiple path sampling simulations} \vspace{0.5ex}
%
%\talk{Dec 2009}{Boston}Vertex Pharmaceuticals \\ {\small Entropy-enthalpy compensation: Not even wrong?} \vspace{0.5ex}
%
%\talk{Apr 2009}{Purdue}Invited talk, Purdue University\\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\talk{Apr 2009}{Notre Dame}Invited talk, University of Notre Dame\\ {\small How do biomolecules move?} \vspace{0.5ex}
%
%\talk{Apr 2009}{Berlin}Freie Universit\"{a}t Berlin, MATHEON\\ {\small Estimators for equilibrium and nonequilibrium experiments} \vspace{0.5ex}
%
%\talk{Mar 2009}{Miami}SIAM National Meeting - State Space Decomposition Methods in Molecular Simulations\\ {\small } {\small Bridging between atomistic simulation and physical experiments with master equation models} \vspace{0.5ex}
%
%\talk{Feb 2009}{UCLA}IPAM Rare Events in High-Dimensional Systems\\ {\small Bridging between atomistic simulation and physical experiments with master equation models}\vspace{0.5ex}
%
%\talk{Jul 2008}{Telluride}Telluride meeting on Enhanced Sampling Methods \\ {\small Statistically optimal estimates from multiple equilibrium samples.} \vspace{0.5ex}
%
%\talk{Jun 2008}{Banff}BIRS Workshop on Mathematical Methods for Free Energy Calculations in Molecular Systems\\ {\small Statistically optimal estimates from equilibrium and nonequilibrium measurements.} \vspace{0.5ex}
%
%\talk{Apr 2008}{Stanford}Gromacs Workshop\\ {\small Alchemical free energy calculations with {\tt gromacs}}\vspace{0.5ex}
%
%\talk{Feb 2008}{Vienna}ESI Workshop on Metastability and Rare Events in Complex Systems\\ {\small Master equation models of protein folding and dynamics from atomistic molecular dynamics simulation} \vspace{0.5ex}
%
%\talk{Aug 2007}{Boston}ACS National Meeting\\ {\small Bridging timescales between atomistic simulation and experiments.} \vspace{0.5ex}
%
%%\talk{Mar 2007}{Chicago}ACS National Meeting\\ {\small Accuracy and reliability in molecular simulation. \emph{(Substituting for Vijay S. Pande)}} \vspace{0.5ex}
%
%\talk{Sep 2006}{Heidelberg}MMS06 Workshop\\ {\small Describing peptide and protein folding dynamics by discrete-state master equation models} \vspace{0.5ex}
%
%\talk{Dec 2003}{San Diego}Burroughs Wellcome Fund, Interfaces in Science Program retreat.\\ {\small Constructing master equation models of protein dynamics from atomistic simulation.} \vspace{0.5ex}
%
%\talk{Mar 2003}{New York}ACS National Meeting\\ {\small Extracting dynamical information from parallel tempering simulations.}
%

%\section*{Teaching experience}
%
%\noindent\years{\scshape 2013-}Faculty, Tri-Institutional Training Program in Computational Biology and Medicine (CBM)\\
%\noindent\years{\scshape 2013-}Faculty, Tri-Institutional Training Program in Chemical Biology (TPCB)\\
%\noindent\years{\scshape Fall 2006}MathBio Boot Camp, UCSF Biophysics. \\
%\noindent\years{\scshape Fall 2003}Teaching Assistant, UCSF BP 206, Computation of Biological Molecules, Matthew P. Jacobson. \\
%\noindent\years{\scshape Fall 2000}Teaching Assistant, UCSF Chem 241, Molecular Thermodynamics / Stat Mech, Ken A. Dill.

%\section*{Research and Computing Grants}
%
%\noindent {\bf Astra-Zeneca iMED Collaboration Award} (2015--2016) \\
%{\small Understanding slow off-rate inhibition in CK2 and SYK} 
%
%\noindent {\bf Functional Genomics Initiative (FGI) Award} (2015--2017) \\
%{\small Characterization of cancer-derived mTOR mutants for precision therapeutics} 
%
%\noindent {\bf Functional Genomics Initiative (FGI) Rapid Response Award} (2015--2016) \\
%{\small Assessing the biophysical impact of clinical kinase mutations on drug binding affinity} 
%
%\noindent {\bf Starr Cancer Foundation} (2015--2016) \\
%{\small co-PIs: Minkui Luo (MSKCC)} 
%
%\noindent {\bf Louis V.~Gerstner Young Investigator Award} (2013--2016)
%
%\noindent {\bf INCITE Allocation for ORNL TITAN} (2014--2015) \\
%{\small co-PIs: Jeremy Smith (ORNL/UT); Jeome Baudry (ORNL/UT)} \\
%{\small Targeting Ras with allosteric modulators using the ORNL TITAN supercomputer} 
%
%\noindent {\bf Louis V.~Gerstner Young Investigator Award} (2013--2016) 
%
%\noindent {\bf Google Exacycle Grant for Visiting Faculty} (2013--2014) \\
%{\small Over 100 million core-hours to benchmark accuracies for protein-small molecule binding free energy computation.} 
%
%\noindent {\bf Teragrid large-scale allocation on NCSA Lincoln} (2010, renewed 2011) \\
%{\small 1.3 million service units in 2011. Grant number TG-MCB100015.} \\
%{\small co-PIs: Michael R. Shirts (University of Virginia); David L. Mobley (University of New Orleans)} \\
%{\small GPU-accelerated calculation of ligand binding affinities to biological macromolecules}

%\section*{Community software and resources}
%
%\noindent {\bf Co-editor}, alchemistry.org. \href{http://www.alchemistry.org}{http://www.alchemistry.org}\\
%{\small A community site for alchemical free energy calculations.} \\
%\noindent {\bf Developer}, pyMBAR. \href{http://www.simtk.org/home/pymbar}{http://www.simtk.org/home/pymbar}.\\
%{\small A free Python implementation of the multistate Bennett acceptance ratio (MBAR) estimator for analysis of computer simulations and single-molecule experiments.} \\
%\noindent {\bf Developer}, MMTOOLS. \href{http://www.simtk.org/home/mmtools}{http://www.simtk.org/home/mmtools} \\
%{\small A toolkit for automated simulation setup and interoperability.} \\
%\noindent {\bf Developer}, YANK. \href{http://www.simtk.org/home/yank}{http://www.simtk.org/home/yank}\\
%{\small A GPU-enabled code for alchemical binding free energy calculations.}\\
%\noindent {\bf Developer}, BHMM. \href{http://www.simtk.org/home/bhmm}{http://www.simtk.org/home/bhmm}.\\
%{\small A Bayesian hidden Markov model (BHMM) analysis Matlab toolkit for single-molecule experiments.} \\
%\noindent {\bf Developer}, Bayesian ITC. \href{http://www.simtk.org/home/bayesian-itc}{http://www.simtk.org/home/bayesian-itc}\\
%{\small A Python-based Bayesian analysis package for isothermal titration calorimetry (ITC) experiments.}\\
%%\vspace{1cm}


\eject
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% TALKS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Recent Invited Talks}

\vspace{-0.1in}

\subsection*{Seminars}

\vspace{-0.1in}

\talk{9 Mar 2023}{San Francisco}The COVID Moonshot: Open science discovery of a novel SARS-CoV-2 antiviral \\ {\small UCSF Biophysics}\vspace{0.5ex}

\talk{15 Feb 2023}{Berlin}Integrating physical modeling and machine learning to enable rational kinase inhibitor polypharmacology \\ {\small DigiDrug}\vspace{0.5ex}

\talk{15 Feb 2023}{Berlin}Teaching free energy calculations to learn \\ {\small Freie Universit\"{a}t Berlin}\vspace{0.5ex}

\talk{14 Feb 2023}{ZOOM}Teaching free energy calculations to learn \\ {\small IST India}\vspace{0.5ex}

\talk{5 Dec 2023}{San Francisco}Teaching free energy calculations to learn \\ {\small COMP Together / Gilead Sciences}\vspace{0.5ex}

\talk{18 Nov 2022}{University of Edinburgh}The COVID Moonshot: Open science discovery of a novel SARS-CoV-2 antiviral \\ {\small University of Edinburgh Biomedical AI CDT Program}\vspace{0.5ex}

\talk{17 Nov 2022}{University of Edinburgh}Deep dive into biophysical modeling and drug discovery \\ {\small University of Edinburgh Biomedical AI CDT Program}\vspace{0.5ex}

\talk{6 Jun 2022}{Toronto}The COVID Moonshot: Open science discovery of a novel SARS-CoV-2 antiviral \\ {\small University of Toronto}\vspace{0.5ex}

\talk{14 Apr 2022}{ZOOM}The future of free energy: Calculations that can learn from experiment \\ {\small OpenEye Scientific}\vspace{0.5ex}

\talk{12 Apr 2022}{ZOOM}The future of free energy: Calculations that can learn from experiment \\ {\small Amgen}\vspace{0.5ex}

\talk{3 Feb 2022}{ZOOM}Open science antiviral discovery with the COVID Moonshot and the open source discovery ecosystem \\ {\small NIH BISTI Seminar}\vspace{0.5ex}

\talk{16 Dec 2021}{ZOOM}The COVID Moonshot: How we got here and where we're going \\ {\small COVID Moonshot Livestream}\vspace{0.5ex}

\talk{8 Dec 2021}{ZOOM}Redesigning drug design \\ {\small University of Toronto}\vspace{0.5ex}

\talk{8 Dec 2021}{ZOOM}Frontiers in structure-based drug discovery \\ {\small Rutgers IQB Seminar (celebrating 50 years of the PDB)}\vspace{0.5ex}

\talk{4 Oct 2021}{ZOOM}Next-generation technologies for computer-driven drug discovery: Teaching free energy calculations to learn, scaling predictions across the kinome, and automating discovery \\ {\small Genentech}\vspace{0.5ex}

\talk{14 Jun 2021}{ZOOM}Redesigning drug design \\ {\small Abbvie}\vspace{0.5ex}

\talk{23 Apr 2021}{ZOOM}The COVID Moonshot: An open science collaboration to develop an orally bioavailable inhibitor of SARS-CoV-2 main viral protease \\ {\small Laufer Center, Stony Brook University}\vspace{0.5ex}

\talk{12 Apr 2021}{ZOOM}Next-generation strategies for designing novel kinase inhibitors and predicting the emergence of cellular resistance \\ {\small Albert Einstein College of Medicine}\vspace{0.5ex}

\talk{31 Mar 2021}{ZOOM}Redesigning drug design \\ {\small Oxford Centre for Medicines Discovery}\vspace{0.5ex}

\talk{16 Dec 2020}{ZOOM} Folding@home: Crowdsourcing computing for a cure \\ {\small COVID Moonshot: State of the Moonshot Livestream}\vspace{0.5ex}

\talk{8 Apr 2020}{ZOOM}The COVID Moonshot: Crowdsourcing a COVID-19 Cure \\ {\small Folding@home 20th Anniversary Livestream}\vspace{0.5ex}

\talk{8 Apr 2020}{ZOOM}COVID Moonshot: Docking and free energy calculations \\ {\small BioSolveIT webinar}\vspace{0.5ex}

\talk{8 Apr 2020}{ZOOM}Drug discovery and how to fix it \\ {\small Google X}\vspace{0.5ex}

\talk{3 Mar 2020}{San Francisco, CA}Accelerating molecular discovery of small molecule modulators of protein-protein interactions with computational chemistry \\ {\small Foresite Labs}\vspace{0.5ex}

\talk{25 Feb 2020}{New York, NY}Redesigning drug design \\ {\small Albert Einstein College of Medicine}\vspace{0.5ex}

\talk{29 Jan 2020}{San Francisco, CA}Building predictive models of antibody:antigen mutations \\ {\small Vir Biotechnology}\vspace{0.5ex}

\talk{13 Nov 2019}{Berlin, Germany}Redesigning drug design \\ {\small Berlin Pharmaceutical Research Colloquium}\vspace{0.5ex}

\talk{7 Nov 2019}{Biberach, Germany}Toward adaptive free energy calculations and physically-informed machine learning for predicting affinity, selectivity, and resistance \\ {\small Boehringer Ingelheim}\vspace{0.5ex}

\talk{6 Sep 2019}{Pittsburgh, PA}Harnessing 100,000 computers around the world to probe kinase activation, inhibition, and resistance \\ {\small University of Pittsburgh}\vspace{0.5ex}

\talk{6 Aug 2019}{Edinburgh, UK}Alchemical free energy calculations, adaptive sampling, and machine learning \\ {\small University of Edinburgh}\vspace{0.5ex}

\talk{2 Jun 2019}{Dortmund, Germany}Redesigning drug design \\ {\small Dortmund Universit\"{a}t}\vspace{0.5ex}

\talk{31 May 2019}{Slough, UK}Predictive models for molecular design \\ {\small UCB Pharma}\vspace{0.5ex}

\talk{29 May 2019}{Wuppertal, Germany}Predictive models for molecular design  \\ {\small Bayer}\vspace{0.5ex}

\talk{14 May 2019}{Cambridge, MA}Predictive models for molecular design \\ {\small XtalPi}\vspace{0.5ex}

\talk{7 May 2019}{Tampa, FL}Alchemical free energy calculations for biomolecular design\\ {\small Vir Biotechnology}\vspace{0.5ex}

\talk{29 Apr 2019}{Tampa, FL}Harnessing 100,000 computers around the world to probe kinase activation, inhibition, and resistance \\ {\small University of South Florida}\vspace{0.5ex}

\talk{17 Apr 2019}{Tempe, AZ}Redesigning drug design\\ {\small Arizona State University}\vspace{0.5ex}

\talk{15 Apr 2019}{Cambridge, MA}Redesigning drug design: New predictive modeling tools for designing selective small molecules with targeted properties\\ {\small Celgene}\vspace{0.5ex}

\talk{15 Nov 2018}{Bethesda, MD}Advancing quantitative biophysical predictions: How can we move things forward?\\ {\small National Institute for Advancing Translational Sciences (NIH NCATS)}

\talk{4 Oct 2018}{Philadelphia, PA}What will it take to predict kinase inhibitor selectivity and resistance using computational physical modeling?\\ {\small Fox Chase Cancer Center}

\talk{15 Aug 2018}{Waltham, MA}Predicting porin permeation for antimicrobial drug development \\ {\small Entasis Therapeutics}

\talk{18 Jul 2018}{G\"{o}ttingen, Germany}Redesigning drug design \\ {\small Max Planck Institute for Biophysical Chemistry}

\talk{17 Jul 2018}{Berlin, Germany}Redesigning drug design: Selectively targeting the kinome and beyond \\ {\small Charit\'{e}}

\talk{8 Jul 2018}{Darmstadt, Germany}Can free energy really be free? The state of open source absolute and relative free energy calculations \\ {\small Merck KGaA}

\talk{22 May 2018}{Cambridge, MA}What will it take to predict kinase inhibitor selectivity and resistance using computational physical modeling? \\ {\small Blueprint Medicines}

\talk{21 May 2018}{Cambridge, MA}Redesigning drug design: Selectively targeting the kinome and beyond \\ {\small Pfizer}

\talk{12 Apr 2018}{San Francisco, CA}Redesigning drug design: Selectively targeting the kinome and beyond \\ {\small Pharmaceutical Chemistry Departmental Seminar, UCSF}

\talk{9 Feb 2018}{Portland, OR}Selective kinase inhibition and emergence of resistance mutations in cancer:\\ Probing driving forces with physical modeling and automated biophysical experiments \\ {\small Biomedical Engineering Seminar, OHSU}

\talk{14 Dec 2017}{Minneapolis, MN}What will it take to predict kinase inhibitor selectivity and resistance using computational physical modeling? \\ {\small Department of Pharmacology, University of Minnesota}\vspace{0.5ex}

\talk{27 Oct 2017}{Stanford, CA}How can physical modeling play a major role in drug discovery and biomedicine? \\ {\small Vijay Pande group, Stanford University}\vspace{0.5ex}

\talk{13 Oct 2017}{New York, NY}Markov chain Monte Carlo and nonequilibrium statistical mechanics in drug discovery\\ {\small Applied Math Seminar, NYU Courant}\vspace{0.5ex}

\talk{18 Sep 2017}{Darmstadt, Germany}Advancing drug design with alchemical free energy calculations\\ {\small Merck Darmstadt}\vspace{0.5ex}

\talk{16 Jun 2017}{Mountain View, CA}Redesigning drug design\\ {\small Google Advanced Sciences}\vspace{0.5ex}

\talk{9 May 2017}{Rahway, NJ}Advancing quantitative biophysical predictions with blind challenges\\ {\small Merck Formulations}\vspace{0.5ex}

\talk{10 Apr 2017}{San Francisco, CA}Redesigning drug design\\ {\small Diane Barber lab, UCSF}\vspace{0.5ex}

\talk{14 Feb 2017}{Boston, MA}Is there a use for nonequilibrium statistical mechanics in drug design?\\ {\small Harvard Widely Applied Math Seminar}\vspace{0.5ex}

\talk{15 Dec 2016}{Collegeville, PA}Redesigning drug design\\ {\small GlaxoSmithKline}\vspace{0.5ex}

\talk{8 Dec 2016}{Bethesda, MD}Redesigning drug design\\ {\small Bernie Brooks group, NHLBI National Institutes of Health}\vspace{0.5ex}

\talk{28 Oct 2016}{Stanford, CA}Redesigning drug design\\ {\small Vijay Pande group, Stanford University}\vspace{0.5ex}

\talk{15 Sep 2016}{Barcelona, Spain}Redesigning drug design\\ {\small Universitat Pompeu Fabra}\vspace{0.5ex}

\talk{13 Sep 2016}{Slough, UK}Redesigning drug design\\ {\small UCB Pharma}\vspace{0.5ex}

\talk{6 Sep 2016}{Darmstadt, Germany}Alchemical free energy calculations for drug discovery\\ {\small Merck KGaA}\vspace{0.5ex}

\talk{29 Oct 2015}{San Francisco, CA}Redesigning drug design\\ {\small Autodesk}\vspace{0.5ex}

\talk{8 Oct 2015}{Rahway, NJ}Redesigning drug design\\ {\small Merck}\vspace{0.5ex}

\talk{24 Jul 2015}{Cambridge, MA}Redesigning drug design\\ {\small Novartis}\vspace{0.5ex}

\talk{3 Jun 2015}{Tokyo, Japan}Redesigning drug design\\ {\small University of Tokyo / RCAST}\vspace{0.5ex}

\talk{15 May 2015}{Waltham, MA}Redesigning drug design\\ {\small AstraZeneca}\vspace{0.5ex}

\talk{13 May 2015}{New Haven, CT}Redesigning drug design\\ {\small Yale Theoretical Chemistry Seminar}\vspace{0.5ex}

\talk{15 Apr 2015}{Chicago, IL}Redesigning drug design\\ {\small Illinois Institute of Technology}\vspace{0.5ex}

\talk{8 Apr 2015}{New Brunswick, NJ}Making the most of limited data in biophysics:\\Challenges in single-molecule biophysics, nonequilibrium statistical mechanics, and drug discovery\\ {\small Rutgers --- Department of Statistics and Biostatistics Seminar}\vspace{0.5ex}

%\color{red}
%[Lots more to be filled in for 2016 and earlier]
%\color{black}

\subsection*{Meetings and Conferences}

\talk{29 Mar 2023}{Chicago}The COVID Moonshot: Open science discovery of a novel SARS-CoV-2 antiviral \\ {\small Caltech-UChicago AI+SCIENCE}\vspace{0.5ex}

\talk{28 Feb 2023}{Marburg}Open antiviral discovery with the COVID Moonshot and ASAP \\ {\small DRUID Antiviral Discovery Symposium}\vspace{0.5ex}

\talk{16 Mar 2023}{Santa Fe}Teaching free energy calculations to learn \\ {\small OpenEye CUP XXII}\vspace{0.5ex}

\talk{31 Aug 2022}{Toronto, Canada}Enabling autonomous decisionmaking in drug discovery \\ {\small ACCELERATE}\vspace{0.5ex}

\talk{15 Jul 2022}{Barcelona, Spain}Teaching free energy calculations to learn \\ {\small MLSMM}\vspace{0.5ex}

\talk{2 Jun 2022}{Boston, MA}Next-generation technologies for structure-enabled drug design \\ {\small MPM Capital Scientific Advisors Meeting}\vspace{0.5ex}

\talk{23 May 2022}{San Diego, CA}The COVID Moonshot: Open science discovery of a novel SARS-CoV-2 antiviral \\ {\small RECOMB}\vspace{0.5ex}

\talk{29 Apr 2022}{ZOOM}Teaching free energy calculations to learn \\ {\small ICML MLDD}\vspace{0.5ex}

\talk{24 Mar 2022}{ZOOM}New frontiers in machine learning for drug discovery \\ {\small Acceleration Consortium}\vspace{0.5ex}

\talk{20 Mar 2022}{ZOOM}The future of free energy: Calculations that can learn from experiment \\ {\small ACS San Diego: Kate Holloway Award Symposium}\vspace{0.5ex}

\talk{10 Mar 2022}{Santa Fe, NM}A Moonshot on theory and practice \\ {\small OpenEye CUP XXI}\vspace{0.5ex}

\talk{21 Apr 2021}{ZOOM}The COVID Moonshot: An open science collaboration to develop an orally bioavailable inhibitor of SARS-CoV-2 main viral protease \\ {\small OpenEye Spring MiniCUP 2021}\vspace{0.5ex}

\talk{5 Apr 2021}{ZOOM}The COVID Moonshot: An open science collaboration to develop an orally bioavailable inhibitor of SARS-CoV-2 main viral protease \\ {\small ACS Spring 2021 Meeting}\vspace{0.5ex}

\talk{16 Mar 2021}{ZOOM}The COVID Moonshot: Closing in on an orally-bioavailable non-peptidomimetic small molecule inhibitor of SARS-CoV-2 Mpro with an open science collaboration \\ {\small Winter RosettaCon}\vspace{0.5ex}

\talk{26 Feb 2021}{ZOOM}The COVID Moonshot: Closing in on an orally-bioavailable non-peptidomimetic small molecule inhibitor of SARS-CoV-2 Mpro with an open science collaboration \\ {\small Biophysical Society Annual Meeting}\vspace{0.5ex}

\talk{29 Jan 2021}{ZOOM}OpenMM: Integrating ML to transform drug discovery \\ {\small NVIDIA Round Table on AI/ML Workloads for MD}\vspace{0.5ex}

\talk{26 Jan 2021}{ZOOM}The COVID Moonshot: Closing in on an orally-bioavailable non-peptidomimetic small molecule inhibitor of SARS-CoV-2 Mpro with an open science collaboration \\ {\small CSHL COVID/SARS-CoV-2 Rapid Research Reports 5}\vspace{0.5ex}

\talk{8 Dec 2020}{ZOOM}OpenMM: Key infrastructure for biomolecular modeling and simulation \\ {\small CZI Essential Open Source Software for Science}\vspace{0.5ex}

\talk{25 Nov 2020}{ZOOM}Learning biomolecular potentials for drug discovery \\ {\small NeurIPS}\vspace{0.5ex}

\talk{5 May 2020}{ZOOM}Future directions in force field science \\ {\small Open Force Field Initiative Virtual Meeting}\vspace{0.5ex}

\talk{14 Mar 2020}{Santa Fe, NM}Analysis, Interpretation, and Presentation of Free Energy Results \\ {\small OpenEye CUP XX}\vspace{0.5ex}

\talk{13 Mar 2020}{Santa Fe, NM}Update on the Open Force Field Initiative \\ {\small OpenEye CUP XX}\vspace{0.5ex}

\talk{21 Nov 2019}{Cambridge, MA}Setting free energy calculations free: Opening up the free energy ecosystem \\ {\small OpenEye MiniCUP Cambridge MA}\vspace{0.5ex}

\talk{17 Nov 2019}{College Park, MD}Powering the next generation of research in machine learning in chemistry with OpenMM, the Open Force Field Initiative, and Blind Predictive Challenges \\ {\small MolSSI Workshop in Machine learning in Chemistry}\vspace{0.5ex}

\talk{5 Nov 2019}{Mainz, Germany}Looking to the future of drug discovery: Integrating free energy calculations, machine learning, and autonomous discovery to drive molecular design \\ {\small German Conference on Cheminformatics}\vspace{0.5ex}

\talk{5 Nov 2019}{Mainz, Germany}Looking to the future of drug discovery: Integrating free energy calculations, machine learning, and autonomous discovery to drive molecular design \\ {\small German Conference on Cheminformatics}\vspace{0.5ex}

\talk{21 Oct 2019}{St Louis, MO}Folding@home Consortium: Perspectives from the Chodera lab \\ {\small Folding@home Consortium Meeting, Washington University, St Louis}\vspace{0.5ex}

\talk{23 Aug 2019}{La Jolla, CA}Looking forward to the future of assessment of predictive modeling and blinded prediction challenges \\ {\small SAMPL/D3R Workshop}\vspace{0.5ex}

\talk{15 Jul 2019}{Mount Snow, VT}Alchemical free energy calculations: Differentiating approaches, measuring impact, and surveying the open source ecosystem \\ {\small Computer Aided Drug Discovery GRC}\vspace{0.5ex}

\talk{27 Jun 2019}{Telluride, CO}Protonation state effects in small molecule recognition and design \\ {\small TSRC Protein Electrostatics}\vspace{0.5ex}

\talk{19 Jun 2019}{Berlin, Germany}Molecular kinetics, free energy calculations, and machine learning for drug discovery \\ {\small MolKin 2019}\vspace{0.5ex}

\talk{7 Jun 2019}{Mainz, Germany}Molecular kinetics, free energy calculations, and machine learning for drug discovery \\ {\small Mainz Materials Simulation Days 2019}\vspace{0.5ex}

\talk{27 May 2019}{G\"{o}ttingen, Germany}What's next for alchemical free energy calculations? \\ {\small Alchemical Free Energy Calculations in Drug Discovery 2019}\vspace{0.5ex}

\talk{26 Apr 2019}{New York, NY}Redesigning drug design (together)\\ {\small Mid-Atlantic Computational Chemistry / NY Area Group in Informatics and Modeling}\vspace{0.5ex}

\talk{8 Mar 2019}{Santa Fe, NM}Free energy calculations on Orion: Research and Development for 2019\\ {\small OpenEye CUP XIX Free Energy Summit}\vspace{0.5ex}

\talk{5 Mar 2019}{Baltimore, MD}Closing the loop in automated design and measurement: Scalable inference for biophysical experiments\\ {\small Biophysical Society Meeting: Symposium on Bayesian methods in biophysics}\vspace{0.5ex}

\talk{27 Nov 2018}{Stockholm}Streamlining molecular simulation data\\ {\small BioExcel Workshop on Interoperability and Reproducibility in Molecular Simulations}\vspace{0.5ex}

\talk{11 Nov 2018}{Berlin}Redesigning drug design\\ {\small Future Medicine Berlin}\vspace{0.5ex}

\talk{7 Oct 2018}{Tempe, AZ}What can protein folding teach us about drug discovery?\\ {\small Statistical Physics in Biology}\vspace{0.5ex}

\talk{2 Oct 2018}{Vienna, Austria}Rare events and large-scale conformational changes in drug discovery\\ {\small ECAM Workshop: Large scale activated event simulations}\vspace{0.5ex}

\talk{12 Jun 2018}{Biddeford, ME}What can physical modeling tell us about mutational mechanisms of kinase inhibitor resistance? \\ {\small Human Genetic Variation GRC}\vspace{0.5ex}

\talk{8 Mar 2018}{Santa Fe, NM}Testing the Friesner Conjecture: Are relative free energy calculations always more efficient than absolute? \\ {\small OpenEye CUP XVIII}\vspace{0.5ex}

\talk{22 Feb 2018}{San Diego, CA}The SAMPL6 Challenges: Advancing molecular modeling accuracy with targeted blind challenges \\ {\small Drug Design Data Resource (D3R) Workshop}\vspace{0.5ex}

\talk{13 Nov 2017}{New York, MA}Bridging scales in cancer\\ {\small Simons MPS Conference}\vspace{0.5ex}

\talk{9 Nov 2017}{Cambridge, MA}Building next-generation forcefields and moving free energy calculations to the cloud\\ {\small OpenEye MiniCUP}\vspace{0.5ex}

\talk{20 Sep 2017}{Berlin, Germany}Next-generation small molecule forcefields\\ {\small RDKit User Group Meeting}\vspace{0.5ex}

\talk{24 Aug 2017}{Gaithersberg, MD}Challenges and opportunities for reproducibility and reliability in molecular simulations\\ {\small MolSSI Best Practices Workshop, NIST}\vspace{0.5ex}

\talk{15 Aug 2017}{Leiden, Netherlands}Developing GPU-accelerated molecular simulation tools with OpenMM and the Omnia ecosystem\\ {\small ECAM Software Workshop on Path Sampling}\vspace{0.5ex}

\talk{14 Jul 2017}{Telluride, CO}Toward realizing the dream of free-energy based small molecule design\\ {\small Telluride Free Energy Workshop}\vspace{0.5ex}

\talk{2 Jun 2017}{New York, NY}What can physical modeling tell us about mutational mechanisms of kinase inhibitor resistance?\\ {\small NYU Genomics Symposium}\vspace{0.5ex}

\talk{29 May 2017}{Toronto, Canada}The statistical mechanics of drug discovery\\ {\small Canadian Society of Chemistry 100th Anniversary National Meeting}\vspace{0.5ex}

\talk{3 May 2017}{Waltham, MA}What can physical modeling tell us about mutational mechanisms of kinase inhibitor resistance?\\ {\small EMBL-EBI Informatics and -omics for Oncology Drug Resistance}\vspace{0.5ex}

\talk{8 Mar 2017}{Santa Fe, NM}Can free energies really be free? Alchemical free energy calculations in the cloud\\ {\small OpenEye CUP XVII}\vspace{0.5ex}

\talk{8 Oct 2016}{Houston, TX}Advancing Quantitative Biophysical Predictions\\ {\small Molecular Software Sciences Institute --- Biomolecular Simulation}\vspace{0.5ex}

\talk{14 Sep 2016}{London, UK}Redesigning drug design\\ {\small CCPBioSim --- Kings College London}\vspace{0.5ex}

\talk{23 Aug 2016}{Philadelphia, PA}What is required for alchemical free energy calculations to be useful for predicting drug polypharmacology?\\ {\small ACS Fall National Meeting}\vspace{0.5ex}

\talk{22 Aug 2016}{Philadelphia, PA}Advancing quantitative biophysical predictions: What can be gained from industry-academic data sharing?\\ {\small ACS Fall National Meeting}\vspace{0.5ex}

\talk{17 May 2016}{Boston, MA}How can we combine computation and experiment to move the field forward?\\ {\small Free Energy Calculations in Drug Discovery, Vertex}\vspace{0.5ex}

\talk{16 May 2016}{San Diego, CA}Redesigning drug design\\ {\small ACS San Diego}\vspace{0.5ex}

\talk{4 Mar 2016}{Edinburgh, UK}Redesigning drug design\\ {\small International Centre for Mathematical Sciences (ICMS)}\vspace{0.5ex}

\talk{1 Mar 2016}{La Jolla, CA}What will it take to design kinase inhibitors with desired selectivity and resistance profiles?\\ {\small GTC 11th Protein Kinases in Drug Discovery Meeting}\vspace{0.5ex}

\talk{19 Oct 2015}{Edinburgh, UK}Biomolecular software interoperability\\ {\small CECAM Software Interoperability Workshop}\vspace{0.5ex}

\talk{18 Sep 2015}{Vienna, Austria}Reaction coordinates in drug discovery\\ {\small ESI Insight from Molecular Simulations}\vspace{0.5ex}

\talk{8 Sep 2015}{Berlin, Germany}Redesigning drug design\\ {\small World Molecular Kinetics Meeting}\vspace{0.5ex}

\talk{5 Jun 2015}{Tokyo, Japan}Redesigning drug design\\ {\small OpenEye JCUP VI}\vspace{0.5ex}

\talk{21 Jan 2015}{New York, NY}Redesigning drug design\\ {\small New York Structural Biology Discussion Group}\vspace{0.5ex}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\subsection*{Lab Member Talks}
%
%\talk{22 Feb 2018}{Mehtap Isik}The SAMPL6 p$K_a$ challenge\\ {\small Drug Design Data Resource (D3R) Workshop, La Jolla, CA}\vspace{0.5ex}
%
%\talk{22 Feb 2018}{Andrea Rizzi}The SAMPL6 host-guest and SAMPLing challenges\\ {\small Drug Design Data Resource (D3R) Workshop, La Jolla, CA}\vspace{0.5ex}
%
%\talk{20 Feb 2018}{Rafal Wiewiora}Conformational Dynamics of Histone Methyltransferase SET8 Probed by Millisecond-Timescale Molecular Dynamics, Markov State Modeling and Biochemical Experiments\\ {\small Biophysical Society Meeting, San Francisco, CA}\vspace{0.5ex}
%
%\talk{8 Jan 2018}{Bas Rustenburg}Protonation state and tautomer sampling for small molecules in biomolecular complexes using OpenMM\\ {\small Amsterdam Center for Multiscale Modeling}\vspace{0.5ex}
%
%\talk{29 Nov 2017}{Chaya Stern}Bayesian inference in computational chemistry\\ {\small PyData NYC, New York, NY}\vspace{0.5ex}
%
%\talk{25 Oct 2017}{Chaya Stern}Is the force with us? Quantifying model uncertainty in molecular simulations\\ {\small NYU Center for Data Science Seminar, New York, NY}\vspace{0.5ex}
%
%\talk{13 Sep 2017}{Steven Albanese}Using physical modeling to direct biophysical experiments and predict the functional impact of clinical cancer mutations in kinases on drug susceptibility\\ {\small Med Into Grad Symposium: Precision Medicine, Columbia University, New York, NY}\vspace{0.5ex}
%
%\talk{6 Oct 2017}{Levi Naden}Computational Free Energy For Everyone (not just Experts)\\ {\small MSKCC csBioMed Work in Progress Talk, MSKCC, New York, NY}\vspace{0.5ex}
%
%\talk{8 Sep 2017}{Steven Albanese}Using physical modeling to direct biophysical experiments and predict the functional impact of clinical cancer mutations in kinases on drug susceptibility\\ {\small EMBO Workshop: Integrating genomics and biophysics to comprehend functional genetic variation, Turin, Italy}\vspace{0.5ex}
%
%\talk{7 Sep 2017}{Patrick Grinaway}Molecular Design using Reversible Jump MCMC and Stochastic Approximation\\ {\small Computational and Systems Biology Retreat}\vspace{0.5ex}
%
%\talk{29 Aug 2017}{Levi Naden}A Feature Complete YANK and Data on FXR\\ {\small Merck KGaA, Darmstadt, Germany}\vspace{0.5ex}
%
%\talk{23 Aug 2017}{Andrea Rizzi}Coding binding free energy calculations: Lesson learned and best practices\\ {\small Coding Solvation Workshop, Livorno, Italy }\vspace{0.5ex}
%
%\talk{15 Feb 2017}{Levi Naden}YANK at Merck: What is it and how does it Work?\\ {\small Merck KGaA, Darmstadt, Germany}\vspace{0.5ex}
%
%\talk{11 Feb 2017}{Sonya Hanson}Can we automatically detect biologically relevant order parameters in molecular simulation?\\ Comparing long timescale simulations of multiple kinases\\ {\small Biophysical Society 61st Annual Meeting, New Orleans, LA}\vspace{0.5ex}
%
%\talk{19 Apr 2017}{Patrick Grinaway}Scalable Free Energy Calculations and Molecular Design\\ {\small Silicon Therapeutics, Boston, MA}\vspace{0.5ex}
%
%\talk{28 Oct 2016}{Sonya Hanson}Open science in drug design:\\ Analysis and visualization of an open experimental dataset of FDA approved cancer drugs\\ {\small Mozilla Science Festival, London, UK}\vspace{0.5ex}
%
%\talk{15 Sep 2016}{Levi Naden}Predicting Thermodynamic Properties over Combinatorially Large Chemical Spaces\\ {\small Merck KGaA, Darmstadt, Germany}\vspace{0.5ex}
%
%\talk{10 Aug 2016}{Josh Fass}Progress and Challenges in Parameterizing and Validating Markov State Models\\ {\small Computational Biology and Medicine Retreat}\vspace{0.5ex}
%
%\talk{22 Jun 2016}{Sonya Hanson}From TRP channels to kinase inhibitors: Combining computation and experiment to redesign drug design\\ {\small NIH Oxford-Cambridge 2016 Global Doctoral Partnerships Annual Workshop, Oxford, UK}\vspace{0.5ex}
%
%\talk{19 May 2016}{Andrea Rizzi}Running alchemical free energy calculations with YANK\\ {\small Kinetics and Markov State Models in Drug Design Workshop, Boston, MA}\vspace{0.5ex}
%
%\talk{17 Mar 2016}{Sonya Hanson}How computer programs can help us design better cancer drugs\\ {\small Genspace, New York, NY}\vspace{0.5ex}
%
%\talk{10 Mar 2016}{Bas Rustenburg}Experimental determination of the SAMPL5 cyclohexane-water distribution coefficients\\ {\small SAMPL5/D3R Workshop, San Diego, CA}\vspace{0.5ex}
%
%\talk{7 Feb 2015}{Sonya Hanson}Developing high-throughput fluorescence-based assays for measuring kinase inhibitor free energies of binding\\ {\small Biophysical Society 59th Annual Meeting, Baltimore, MD}\vspace{0.5ex}
%
%\talk{10 Aug 2014}{Sonya Hanson}Tackling complex problems in small molecule recognition using computation and automated experiment\\ {\small Telluride TSRC Molecular Recognition Workshop, Telluride, CO}\vspace{0.5ex}

%\color{red}
%[Lots more to be filled in for 2016 and earlier]
%\color{black}

%\subsection*{Lab Member Talks}
%
%\color{red}
%[To be filled in]
%\color{black}
%
\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% FUNDING AND PERSONNEL LISTS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Funding}

All funding sources are publicly available online at \url{http://choderalab.org/funding}

\subsection*{Active}

\subsubsection*{Single-Investigator Grants}

%\emph{Federal}
%\vspace{1.5ex}

%\eject

% ACTIVE COLLABORATIVE GRANTS
\subsubsection*{Collaborative Grants}

NIH NIAID U19 1U19AI171399 (MPI: Chodera [Contact PI]; Alpha Lee, PostEra; Peter Sj\"{o}, DNDi) \\
{\bf AI-driven Structure-enabled Antiviral Platform (ASAP)}
co-Is: Karla Kirkegaard (Stanford), Matt Bogyo (Stanford), Jesse Bloom (Fred Hutch), Frank von Delft (Diamond Light Source), Martin Walsh (Diamond Light Source), Alpha Lee (PostEra), Nir London (Weizmann), Ed Griffen (MedChemica), Laurent Fraisse (DNDi), Haim Barr (Weizmann), Daren Fearon (Diamond Light Source), Kris White (Mount Sinai), Adolfo Garc\'{i}a-Sastre (Mount Sinai), Randy Albrecht (Mount Sinai) \\
05/16/2022 -- 04/30/2025\\
\emph{The goal of this antiviral drug discovery (AViDD) Center is to use an AI-accelerated structure-enabled discovery platform to build a pipeline of novel oral antivirals against coronaviruses, flaviviruses, and picornaviruses using an open science, IP-free discovery paradigm to act as a nexus for global antiviral discovery.}\\
Total federal funds awarded: \$67,662,387\\
Total direct costs: \$49,000,085\\
%Total direct costs less consortium/subaward costs: \$23,689,478\\
Total F\&A for MSKCC: \$18,662,302
Eligible for non-competive renewal for two more years for an additional \$42M.
%Total federal funds requested: \$113,076,459\\
%Total direct costs: \$86,402,219\\
%Total direct costs less consortium F\&A: \$75,000,000\\
%Total F\&A for MSKCC: \$24,415,485 \\
%Note: Notice of Award has not yet arrived but we are told by NIAID to expect it prior to press release on 18 May 2022.

\vspace{1.5ex}

NIH R01 1R01GM132386 (MPI: John Chodera; Michael Shirts, University of Colorado [Contact PI]) \\
{\bf Open data-driven infrastructure for building biomolecular force fields for predictive biophysics and drug design} \\
co-Is: David Mobley (UCI), Lee-Ping Wang (UC Davis), Michael Gilson (UCSD) \\
The goal of this project is to develop a modern open infrastructure for building fully consistent biopolymer and small molecule force fields for drug design and predictive biophysical modeling. \\
3/1/2020 -- 2/29/2024 \\
\$20,684 annual direct costs to the Chodera lab\\
\emph{Note:} Chodera was a PI of this multi-PI/PD grant for the first year, but moved to co-I status in subsequent years due to NIGMS limit two active GMS PI grants.

\vspace{1.5ex}

NIH 3R01GM132386-02S1 (PI: Michael Shirts, University of Colorado Boulder)\\
{\bf Supplement: Open data-driven infrastructure for building biomolecular force fields for predictive biophysics and drug design}\\
co-Is: David Mobley (UCI), Lee-Ping Wang (UC Davis), Michael Gilson (UCSD) \\
The goal of this project is to support continued development of OpenMM, transition to a community governance and distributed development model, and extension to integrate physical and machine learning models to enable genomic-scale biomolecular modeling, simulation, and prediction.\\
3/1/2021 -- 2/29/2024\\
\$132,694 annual direct costs for this supplement\\
\emph{JDC wrote this proposal to secure additional funding for a critical community resource (QCArchive/QCFractal) he will help direct.}

\vspace{1.5ex}

NIH 1R01GM140090 (PI: Tom Markland, Stanford University)\\
{\bf OpenMM: Scalable biomolecular modeling, simulation, and machine learning}\\
co-I: Gianni de Fabritiis (Universitat de Pompeu Fabra, Barcelona)\\
The goal of this project is to support continued development of OpenMM, transition to a community governance and distributed development model, and extension to integrate physical and machine learning models to enable genomic-scale biomolecular modeling, simulation, and prediction.\\
7/1/2021 -- 3/31/2025\\
\$36,685 annual direct costs to the Chodera lab


%\vspace{1.5ex}
%\emph{Industry}

\vspace{1.5ex}

% Fix : Split into two awards
BIH Einstein Visiting Professorship at the Charit\'{e} Berlin\\
{\bf Computational polypharmacology: A new paradigm for selectively promiscuous kinase inhibitors}\\
The goal of this project is to develop structure-informed machine learning approaches to predicting kinase inhibitor polypharmacology for computer-guided small molecule design.\\
%1/1/2019 -- 6/30/2022\\
1/1/2019 -- 12/31/2024\\
EUR150,000 annual direct costs to Chodera (via Charit\'{e}) for Berlin working group (with six-month no-cost extension)

%\vspace{1.5ex}
%
%{\bf The Open Force Field Consortium}\\
%An academic-industry consortium to develop open biomolecular force fields\\
%\url{http://openforcefield.org/consortium}\\
%10/1/2018 -- (renewed annually)\\
%\$600,000 annual total costs to the Consortium

% ACTIVE FELLOWSHIPS
\vspace{-0.1in} 

\subsubsection*{Fellowships}

Sukrit Singh // Postdoctoral Fellow \\
{\bf Damon Runyon Quantitative Biology Fellowship}\\
5/1/2022 -- 4/30/2025\\
\$240,000 total stipend support

\vspace{1.5ex}

Benjamin Kaminow // CBM Graduate Student \\
{\bf NSF Graduate Research Predoctoral Fellowship (GRFP)}\\
9/1/2023 -- 8/31/2026\\
\$49,000 annual stipend support

%\vfill

\eject
%%%%%%%%%%%%%%%%%%
\subsection*{Pending}

\subsubsection*{Single-Investigator Grants and Contracts}

NIH NIGMS R35 GM152017-01 (PI: Chodera)\\
12/01/2023 -- 11/30/2028\\
{\bf Teaching free energy calculations to learn}\\
\emph{This proposal would support the development and integration of advanced machine learning methods for alchemical free energy calculations}\\
\$499,344 annual direct costs to the Chodera lab\\
{\bf Impact Score: 18}

\vspace{1.5ex}

NIH DP1OD037515-01  (PI: Chodera)\\
8/01/2024 -- 7/31/2029\\
{\bf Redesigning drug design}\\
\emph{This proposal would support the development of autonomous small molecule drug discovery technologies}\\
\$700,000 annual direct costs to the Chodera lab

\vspace{1.5ex}

HHMI 2024 Investigator Competition (PI: Chodera)\\
9/1/2024 -- 8/31/2031\\
TBD annual direct costs to the Chodera lab over seven years

%(Note: JDC has already reached the NIH GMS limit of two funded grants on which he serves as Principal Investigator.)

%\vspace{1.5ex}
%
%Foresite Labs Sponsored Research Agreement \\
%{\bf Predicting the binding of cancer-associated substrates to ubiquitin ligases}\\
%The goal of this project is to develop and apply open source tools to predict the binding of cancer-associated substrates to ubiquitin ligases.\\
%8/1/2020 -- 7/30/2022\\
%\$119,500 annual direct costs to the Chodera lab
%
%\vspace{1.5ex}
%
%Silicon Therapeutics Sponsored Research Agreement \\
%{\bf Investigating the efficiency of algorithms for relative free energy calculations}\\
%The goal of this project is to assess the relative efficiencies of various algorithms for relative free energy calculations for drug discovery.\\
%9/1/2020 -- 8/31/2022\\
%\$94,680 annual direct costs to the Chodera lab

% TODO: HHMI

%2019 Pershing Square Sohn Prize for Young Investigators in Cancer Research\\
%{\bf Predicting mutational resistance and susceptibility to targeted kinase inhibitors for cancer}\\
%The goal of this project is to develop structure-informed machine learning approaches to predicting kinase inhibitor polypharmacology for computer-guided small molecule design.\\
%1/1/2019 -- 12/31/2021\\
% \$200,000 annual direct costs for three years
% 
%\vspace{1.5ex}

%Simons Investigator in the Mathematical Modeling of Living Systems Award\\
%{\bf Modeling the emergence of drug resistance mutations with equilibrium and nonequilibrium statistical\\ mechanics}\\
%The goal of this project is to develop and use principles of equilibrium and nonequilibrium statistical mechanics to model the emergence of drug resistance mutations in the target of therapy.\\
%8/1/2019 -- 7/31/2024\\
%\$132,000 annual direct costs

%Cycle for Survival Equinox Innovation \\
%{\bf Predicting susceptibility and resistance to targeted kinase inhibitors in rare cancers}\\
%The goal of this project is to develop physical modeling approaches for predicting kinase inhibitor sensitivity and resistance for rare cancers.\\
%10/1/2019 -- 9/30/2021\\
%\$200K annual direct costs to Chodera

% SIMONS MMLS


%\vspace{1.5ex}

\subsubsection*{Collaborative Grants}

NIH NIGMS R01 HG013328-01A1 (PI: Morris)\\
7/1/2023 -- 6/30/2027\\
{\bf Post-transcriptional Regulatory Networks}\\
\emph{The role of the Chodera lab in this project is to co-supervise a student in the development of machine learning models for protein:RNA structure and affinity prediction}\\
\$15,749 annual direct costs to the Chodera lab\\
\emph{Impact score 25, Percentile 13th, NHGRI council meeting scheduled for 5/2023}

\vspace{1.5ex}

NIH NCI R01 GM145739‑01 (MPI: Chiosis/Gewirth)\\
4/1/2022 -- 3/31/2027\\
{\bf Mechanistic investigations into Glyc62GRP94 epichaperomes}\\
\emph{The role of the Chodera lab in this project is to use MD simulations to uncover the effect of the N-glycan on GRP94 structure and ligand binding.}\\
\$50,000 annual direct costs to the Chodera lab\\
\emph{Impact Score:28; Percentile:16, NIGMS declined to fund due to "well-funded lab" rule; A1 submission with PI: Gewirth scored and moved to "for consideration", with funding decision 6/2023}

%\vspace{1.5ex}
%
%NIH 1P01 CA262136-01A1 (PI: Li) \\
%7/1/2022 ‑- 6/30/2027\\
%{\bf GRP94 in Cancer Biology and Immuno‑oncology (Project 1; Project leader: Chiosis)}\\
%\emph{The role of the Chodera lab in this project is to use MD simulations to discover improved inhibitors that target this cancer-specific GRP94 variant.}\\
%\$50,000 annual direct costs to the Chodera lab

%\vspace{1.5ex}
%
%Einstein Visiting Fellowship at the Charit\'{e} Berlin\\
%{\bf Computational polypharmacology: A new paradigm for selectively promiscuous kinase inhibitors}\\
%The goal of this project is to develop structure-informed machine learning approaches to predicting kinase inhibitor polypharmacology for computer-guided small molecule design.\\
%7/1/2022 -- 6/30/2024\\
%EUR115,000 annual direct costs to Chodera (via Charit\'{e}) for Berlin working group

%NIH COVID-19 R01 1R01AI161245 (PI: John Chodera, MSKCC) \\
%{\bf Development of small molecule SARS-CoV-2 Mpro inhibitors via large-scale crystallography and computation} \\
%co-Is: Frank von Delft (Oxford), Annette von Delft (Oxford), Nir London (Weizmann) \\
%The goal of this project is to develop a small molecule inhibitor for SARS-CoV-2 Mpro using a combination of large-scale free energy calculations and high-throghput X-ray crystallography. \\
%9/1/2020 -- 8/31/2023 \\
%\$292,186 annual direct costs to the Chodera lab

%\vspace{1.5ex}

%NIH R01 (PI: Daniel Heller, MSKCC) \\
%{\bf Targeted delivery of kinase inhibitors to RAS-driven GI tumors} \\
%The goal of this project is to develop predictive computational models for the design and optimization of nanoparticles for the targeted delivery of selective kinase inhibitors. \\
%4/1/2020 -- 3/31/2025 \\
%\$471,771 annual direct costs / \$98,218 annual direct costs to Chodera lab
%
%\vspace{1.5ex}

%Wellcome Trust UNS95190 (PI: Frank von Delft, Oxford/DiamondMX) \\
%{\bf Effecting a transformation in compound discovery through open, experiment-coupled AI-led drug design}\\
%The goal of this proposal is to develop computational techniques for autonomously designing small molecules and interpreting experimental assays utilizing the DiamondMX high-throughput beamline.\\
%3/1/2020 -- 2/28/2025 \\
%\$955,159 annual direct costs / \$91,980 annual direct costs to Chodera lab
%
%\vspace{1.5ex}

%NSF Expeditions in Computing (PI: Edward Dougherty, Texas A\&M) \\
%NSF 18-528 -- CCF -- Experimental Expeditions\\
%{\bf Expeditions: Achieving Computational and Engineering Objectives Under Uncertainty -- \\ Optimal Operational and Experimental Design} \\
%The goal of this project is to develop optimal adaptive methods for experimental design under uncertainty using operator methods, with multiple applications including drug discovery. \\
%12/1/2019 -- 11/30/2024 \\
%\$72,801 annual direct costs to Chodera
%
%\vspace{1.5ex}

%\color{red}
%NIH 1R01CA222843-01 (PI: James Hsieh, MSKCC) \\
%{\bf Characterization of Cancer-derived mTOR Mutations for Precision Therapeutics} \\
%The goal of this project is to understand the mechanism underlying clinically-identified mTOR activating mutants and evaluate the potential for rapalog therapy to aid the 2\% of cancer patients harboring mTOR mutations. \\
%09/01/2017 -- 08/31/2022 \\
%\$408,993
%\color{black}

%\vspace{1.5ex}

%Simons Collaborations in Mathematics and the Physical Sciences\\
%{\bf Statistical mechanics and statistical inference for systems driven out of equilibrium}\\
%lead PI: Aaron Dinner (University of Chicago)\\
%direct costs: \$2,500,000/year (shared among all investigators)\\
%09/01/2016--08/31/2020

%\vspace{1.5ex}
%
%NIH NCI R01 resubmission\\
%{\bf Characterization of cancer-derived mTOR mutations for precision therapeutics}\\
%lead PI: James Hsieh\\
%direct costs: \$262,915/year (shared among all investigators)\\
%04/01/2016--03/30/2020

%\vspace{1.5ex}
%
%\color{red}
%NIH NIGMS R01\\
%{\bf The role of protonation state effects in selective inhibitor recognition}\\
%lead PI: John Chodera\\
%direct costs: \$250,000/year\\
%submission close date: 2/01/2016
%\color{black}

%\color{red}
%NIH U54 Cancer Systems Biology\\
%{\bf Systems Biology of Cancer Immunobiology}\\
%lead PI: Christina Leslie\\
%direct costs: \$XXX,XXX/year\\
%submission close date: 11/05/2015
%\color{black}

%Chan Zuckerberg Initiative -- Essential Open Source Software for Science (PI: Tom Markland, Stanford University)\\
%co-I: Gianni de Fabritiis (Universitat de Pompeu Fabra, Barcelona)\\
%{\bf OpenMM: Scalable biomolecular modeling, simulation, and machine learning}\\
%The goal of this project is to support continued development of OpenMM, transition to a community governance and distributed development model, and extension to integrate physical and machine learning models to enable genomic-scale biomolecular modeling, simulation, and prediction.\\
%12/1/2019 -- 11/30/2020\\
%\$100,000 annual direct costs to the Chodera lab

%\vspace{1.5ex}
%
%NSF CMI (PI: David D.\ L.\ Minh, IIT)\\
%{\bf Collaborative Research: CDS\&E: Elucidating Binding using Bayesian Inference to Integrate Multiple Data Sources}\\
%9/1/2019 -- 8/31/2022\\
%\$126,779 annual direct costs / \$52,438 annual direct costs to Chodera

%\vspace{1.5ex}
%
%NIH 1R01GM124150-01A1 (PI: Nicholas Levinson, UMN) \\
%{\bf Unraveling multifaceted allosteric regulation in Aurora kinase through coupled experiment and theory} \\
%The goal of this project is to understand the mechanism of complementary mechanisms of Aurora activation \\
%12/1/2017 -- 11/30/2022 \\
%\$372,047 annual direct costs / \$90,878 annual direct costs to Chodera
%
%\vspace{1.5ex}
%
%NIH 1R01GM123183-01A1 \\
%{\bf Protonation state effects in selective kinase inhibitor recognition} \\
%The goal of this project is to characterize the role of protonation state effects in selective kinase inhibitor recognition and explore ramifications for selective inhibitor design. \\
%co-Is: Markus Seeliger (Stony Brook); Marilyn Gunner (CCNY) \\
%7/1/2017 -- 6/30/2022 \\
%\$344,109 annual direct costs / \$187,109 annual direct costs to Chodera

\subsubsection*{Fellowships}

None

\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Completed}

\subsubsection*{Single-Investigator Grants and Contracts}

Vir Biosciences Sponsored Research Agreement SK2019-0582 \\
{\bf Development of alchemical free energy methods for predicting the impact of point mutations on antigen and antibody affinities}\\
The goal of this project is to develop alchemical free energy methods for predicting the impact of point mutations on antigen and antibody affinities for SARS-CoV-2.\\
8/1/2019 -- 7/31/2023\\
\$55,556 annual direct costs

\vspace{1.5ex}

Interline Therapeutics Sponsored Research Agreement SK2020-0898 \\
{\bf Identification of key biophysical interactions in ligase-substrate recognition} \\
The goal of this project is to identify key biophysical interactions necessary for ligase-substrate recognition and developing quantitative accurate open source tools for predicting changes in affinity associated with point mutations in either ubiquitin ligases or their substrate peptides. \\
1/15/2021 -- 4/15/2023\\
\$187,998 direct costs

\vspace{1.5ex}

NIH R01 GM121505 \\
{\bf The role of reorganization energy in achieving selective kinase inhibition} \\
The goal of this project is to probe the role of protein reorganization energy in achieving selectivity in targeted kinase inhibition with small molecules. \\
9/1/2017 -- 8/31/2022 \\
\$196,250 annual direct costs to the Chodera lab

\vspace{1.5ex}

Relay Therapeutics Collaboration SK2018-0162\\
{\bf Development of efficient open source cloud-enabled free energy based lead optimization algorithms and integrative Bayesian model of experimental biophysical and molecular simulation data} \\
The goal of this project is to develop open source scalable cloud workflows for lead optimization using relative alchemical free energy calculations, as well as Bayesian integrative modeling techniques for experimental biophysical data and molecular simulations. \\
4/5/2018 -- 4/4/2023 \\
\$124,828 annual direct costs

\vspace{1.5ex}

Bayer Sponsored Research Agreement SK2019-1289 \\
{\bf Development of a combined machine learning / alchemical free energy approach to predict the impact of mutations on small molecule kinase inhibitor binding} \\
The goal of this project is to develop open source scalable free energy and machine learning workflows in collaboration with the Volkamer lab (Charit\'{e} Berlin) to predict kinase mutations small molecule therapeutics may be effective against. \\
3/1/2020 -- 4/30/2022 \\
\$117,105 annual direct costs to the Chodera lab

\vspace{1.5ex}

NSF RAPID CHE2033426\\
{\bf RAPID: Identifying Biophysical Determinants of Binding to the SARS-CoV-2 Main Viral Protease}\\
The goal of this proposal is to assess the accuracy of large-scale alchemical free energy calculations in identifying potent inhibitors of the SARS-CoV-2 main viral protease (Mpro) with the COVID Moonshot. \\
7/1/2020 -- 6/30/2021\\
\$112,994 direct costs to Chodera lab

\vspace{1.5ex}

Entasis Therapeutics Sponsored Research Agreement SK2018-0163 \\
{\bf Development of efficient equilibrium and nonequilibrium algorithms for predicting small molecule porin permeation with potential of mean force methods} \\
The goal of this project is to develop algorithms and open source tools for efficient equilibrium and nonequilibrium potential of mean force calculations for bacterial porins.\\
4/5/2018 -- 4/5/2019 \\
\$74,074 annual direct costs to the Chodera lab

\vspace{1.5ex}

Parker Institute for Cancer Immunotherapy Pilot Grant \\
{\bf Physics-based computational prediction of cancer-associated mutant MHC class II epitopes} \\
The goal of this project is to develop algorithms and open source tools for efficient computation of MHC class II epitope binding affinities.\\
4/2/2018 -- 4/1/2019 \\
\$75,000 annual direct costs to the Chodera lab

\vspace{1.5ex}

NSF D3SC EAGER CTMC \\
{\bf A probabilistic framework for automated forcefield parameterization from experimental datasets} \\
The goal of this project is to develop a scalable Bayesian inference infrastructure to parameterize molecular mechanics forcefields using experimental datasets. \\
9/1/2017 -- 8/31/2019 \\
\$52,030 annual direct costs to the Chodera lab

\vspace{1.5ex}

Silicon Therapeutics Sponsored Research Agreement \\
{\bf Development of efficient open source free energy based lead optimization algorithms} \\
The goal of this project is to develop open source scalable parallel workflows for lead optimization using relative alchemical free energy calculations. \\
5/17/2017 -- 11/17/2018\\
\$48,655 annual direct costs to the Chodera lab

\vspace{1.5ex}

Merck KGaA Sponsored Research Agreement \\
{\bf Developing automated workflows for absolute alchemical free energy calculations} \\
The goal of this project is to develop open source automated workflows using absolute alchemical free energy calculations for use in prioritizing compounds for synthesis in drug discovery applications. \\
7/26/2016 -- 1/25/2019 \\
\$90,273 annual direct costs to the Chodera lab

\vspace{1.5ex}

Astra-Zeneca iMed Sponsored Research Agreement \\
{\bf Evaluating the potential for Markov state models of conformational dynamics to advance quantitative prediction of thermodynamics and kinetics of selective kinase inhibitors} \\
The goal of this project is to evaluate the potential for Markov state models of conformational dynamics to quantitatively predict the thermodynamics and kinetics of selective kinase inhibitors to CK2 and SYK. \\
7/30/2015 -- 1/30/2017 \\
%\color{red}
\$117,505 annual direct costs to the Chodera lab
%\color{black}

\vspace{1.5ex}

Gerstner Family Foundation \\
{\bf Louis V. Gerstner, Jr.~Young Investigator Award} \\
Dates: 2/1/2013 -- 1/30/2016 \\
\$75,000 annual direct costs

\vspace{1.5ex}

MSKCC Functional Genomics Initiative (FGI) Rapid Response Grant \\
{\bf Biophysical characterization of clinically-identified kinase mutations} \\
The goal of this project is to quantify the impact of clinically-identified kinase domain mutations on FDA-approved selective kinase inhibitor binding. \\
4/1/2016 -- 3/31/2016 \\
\$25,000 annual direct costs

\vspace{1.5ex}

MSKCC Functional Genomics Initiative (FGI) Rapid Response Grant \\
{\bf Biophysical characterization of clincally-identified K-Ras mutants} \\
The goal of this project is to biophysically characterize clinically-identified K-Ras mutants. \\
7/1/2016 -- 6/30/2016 \\
\$25,000 annual direct costs

\subsubsection*{Fellowships}

Dominic Rufa // TPCB Graduate Student\\
{\bf MolSSI Seed Software Fellowship}\\
1/1/2022 -- 6/30/2022\\
\$20,000 stipend support

\vspace{1.5ex} 

Yuanqing Wang // PBSB Graduate Student\\
{\bf Anagenex Open Science Fellowship}\\
9/1/2021 -- 2/28/2022\\
\$21,750 stipend support

\vspace{1.5ex} 

Karmen \v{C}ondi\'{c}-Jurki\'{c}, Postdoctoral Research Fellow\\
{\bf Open Force Field Consortium Postdoctoral Fellowship}\\
4/1/2020 -- 3/31/2021\\
\$88,864.10 stipend, supplies, and travel support

\vspace{1.5ex}

Ivy Zhang CBM Graduate Student\\
{\bf MolSSI COVID-19 Seed Software Fellowship}\\
7/1/2020 -- 12/31/2020\\
\$20,000/year stipend support

\vspace{1.5ex}

Hannah Bruce Macdonald, Postdoctoral Fellow\\
{\bf MolSSI Investment Software Fellowship}\\
7/1/2019 -- 12/31/2020\\
\$50,000/year stipend support

\vspace{1.5ex}

Marcus Wieder, Postdoctoral Research Fellow\\
{\bf Austrian Schr\"{o}dinger Stipendium}\\
10/1/2018 -- 9/30/2019\\
Fellowship recipient direct paid

\vspace{1.5ex}

Simon Boothroyd, Postdoctoral Research Fellow\\
{\bf XtalPi / Open Force Field Consortium Distinguished Postdoctoral Fellowship}\\
10/1/2018 -- 9/30/2019\\
Fellowship recipient direct paid

\vspace{1.5ex}

Andrea Rizzi, CBM Graduate Student\\
{\bf MolSSI Open Force Field Predoctoral Fellowship}\\
1/1/2019 -- 6/30/2019\\
\$20,000/six months stipend support

\vspace{1.5ex}

Rafal P. Wiewiora, TPCB Graduate Student\\
{\bf DOD Peer-Reviewed Cancer Research Program Award}\\
8/15/2017 -- 8/14/2019\\
 \$75,000/year for stipend and supplies

\vspace{1.5ex}

Karmen \v{C}ondi\'{c}-Jurki\'{c}, Postdoctoral Research Fellow\\
{Open Force Field Consortium Postdoctoral Fellowship}\\
1/1/2019 -- 6/30/2019\\
\$60,628 stipend and travel support

\vspace{1.5ex}

Chaya Stern, TPCB Graduate Student\\
{\bf MolSSI Phase II Fellowship}\\
7/1/2018 -- 12/31/2018\\
\$40,000/year stipend support

\vspace{1.5ex}

Chaya Stern, TPCB Graduate Student\\
{\bf MolSSI Phase I Fellowship}\\
7/1/2018 -- 12/31/2018\\
\$40,000/year stipend

\vspace{1.5ex}	

Chaya Stern, TPCB Graduate Student\\
{\bf NSF Graduate Research Fellowship}\\
8/1/2015 -- 7/30/2018\\
\$32,000/year stipend support

\vspace{1.5ex}

Mehtap Isik, TPCB Graduate Student\\
{\bf Doris J.~Hutchison Fellowship}\\
7/1/2017 -- 6/30/2018\\
\$38,668/year stipend support

\vspace{1.5ex}

Gregory A. Ross, Postdoctoral Fellow\\
{\bf Molecular Software Sciences Institute (MolSSI) Graduate Research Fellowship}\\
\$50,000/year stipend support

\vspace{1.5ex}

%\color{red}
Steven Albanese, GSK Graduate Student\\
{\bf Summer Internship, Schr\"{o}dinger}\\
Summer 2017

\vspace{1.5ex}

%\color{red}
Ari\"{e}n S. Rustenburg, PBSB Graduate Student\\
{\bf Summer Internship, Genentech}\\
Summer 2015

\vspace{1.5ex}

Andrea Rizzi, CBM Graduate Student\\
{\bf Tri-I CBM Student Stipend Assistance}\\
\$13,500/year stipend support

\color{black}

\subsubsection*{Collaborative Grants}

NIH R01 GM124270 (PI: David Mobley, UCI) \\
{\bf Advancing predictive physical modeling through focused development of model systems\\ to drive new modeling innovations}\\
The goal of this project is to enable blind community challenges that drive progress toward quantitative accuracy in the field of computational physical modeling of drug-receptor interactions.\\
9/10/2018 -- 8/31/2023\\
\$42,711 annual direct costs to the Chodera lab

\vspace{1.5ex}

NSF CHI 1904822 (MPI: John Chodera, David Minh) \\
{\bf Collaborative Research: CDS\&E: Elucidating Binding using Bayesian Inference to Integrate Multiple Data Sources}\\
The goal of this project is to develop a scalable approach to Bayesian inference from biophysical experiments\\
9/1/2019 -- 8/31/2023\\
\$47,327 annual direct costs to the Chodera lab

\vspace{1.5ex}

Chan Zuckerberg Initiative -- Essential Open Source Software for Science (PI: Tom Markland, Stanford University)\\
co-I: Gianni de Fabritiis (Universitat de Pompeu Fabra, Barcelona)\\
{\bf OpenMM: Key infrastructure for biomolecular modeling and simulation}\\
The goal of this project is to support continued development of OpenMM, transition to a community governance and distributed development model, and extension to integrate physical and machine learning models to enable genomic-scale biomolecular modeling, simulation, and prediction.\\
9/1/2020 -- 4/1/2021\\
\emph{JDC wrote this proposal to secure funding for a key community infrastructure project that his lab depends upon.}\\
\$180,000 total direct costs to support this project

Functional Genomics Initiative (FGI)  \\
{\bf Integrated Approaches Annotate Functions of Cancer-associated H3K36 Methyltransferases} \\
The goal of this project is to develop approaches for using large-scale physical modeling and biochemical experiments to annotate the functions of clinical cancer mutations.\\
1/1/2019 -- 12/31/2020 \\
co-PI: Minkui Luo \\
\$125,000 annual direct costs to the Chodera lab

\vspace{1.5ex}

2014 Functional Genomics Initiative (PI: James Hsieh, MSKCC) \\
{\bf Characterization of Cancer-derived mTOR Mutations for Precision Therapeutics} \\
The goal of this project is to understand the mechanism underlying clinically-identified mTOR activating mutants and evaluate the potential for rapalog therapy to aid the 2\% of cancer patients harboring mTOR mutations. \\
5/1/2015 -- 4/30/2017 \\
%\color{red}
\$118,000
\color{black}

\vspace{1.5ex}

2014 STARR Cancer Consortium (PI: Minkui Luo, MSKCC) \\
{\bf Designing sinefungin scaffolds as specific protein methyltransferase inhibitors} \\
The goal of this project is to use computational techniques that explicitly incorporate protein flexibility to design selective inhibitors for protein lysine methyltransferases. \\
1/1/2015 -- 6/30/2017 \\
%\color{red}
\$125,000
\color{black}

\eject

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% PERSONNEL
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Personnel}

\subsection*{Current members}

\subsubsection*{Postdocs}

\begin{itemize}
  \item Sukrit Singh, PhD Washington University St.\ Louis (6/1/2021--) \\
  Damon Runyon Quantitative Biology Fellow\\
  \emph{Selective inhibition and drug resistance in clinical kinase inhibitors}

\end{itemize}

\subsubsection*{Graduate Students}

\begin{itemize}
  \item Dominic Rufa (4/1/2019--)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  \emph{Machine learning for alchemical free energy calculations}

  % check this
  \item Alexander Payne (8/1/2020--) [co-supervised with Rich Hite, MSKCC Structural Biology Program] \\
  Program in Physiology, Biophysics, and Systems Biology (PBSB)\\
  \emph{Combining cryo-EM with Markov state models, virtual screening, and free energy calculations for drug discovery}
  
  % check this
  \item Viktor Belay (8/1/2020--) [co-supervised with Rich Hite, MSKCC Structural Biology Program] \\
  Program in Physiology, Biophysics, and Systems Biology (PBSB)\\
  \emph{Combining cryo-EM with Markov state models, virtual screening, and free energy calculations for drug discovery}

  % check this
  \item Michael Retchin (7/1/2021--) \\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  \emph{New technologies for autonomous drug discovery}

  % check this
  \item Benjamin Kaminow (1/1/2022--)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  \emph{Structure-enabled machine learning for drug discovery}
  
  % check this
  \item Jessica White (7/1/2022--)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  \emph{Structure-enabled machine learning for kinase-directed drug discovery and resistance}  
  
  % check this
  \item Kendall Lemons (7/1/2022--)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  \emph{Structure-enabled machine learning for antiviral drug discovery}  

\end{itemize}

\subsubsection*{Software Scientists}

\begin{itemize}
  \item Iv\'{a}n Pulido (8/2021--)\\
  \emph{Alchemical free energy calculation software infrastructure; software infrastructure support for ASAP}

  \item Jenke Scheen (8/2022--) \emph{via \href{http://omsf.io}{OMSF}}\\
  \emph{Computer-aided drug discovery software infrastructure lead for ASAP}

  \item Mike Henry (1/2021--)\\
  \emph{Alchemical free energy calculation software infrastructure; software infrastructure support for ASAP}

  \item David Dotson (1/2021--) \emph{via \href{http://omsf.io}{OMSF}}\\
  \emph{Folding\@home distributed computing and alchemical free energy software infrastructure support for ASAP}
  
  \item Hugo MacDermott-Opeskin (1/2023--) \emph{via \href{http://omsf.io}{OMSF}}\\
  \emph{Folding\@home distributed computing and alchemical free energy software infrastructure support for ASAP}
  
\end{itemize}

% TODO: Add contractors?

\subsubsection*{Research Associates}

\begin{itemize}
  \item Erica Goldberger (7/1/2019--)\\
  \emph{Lab manager; Automated measurements of small molecule binding affinities to model drug targets for improving quantitative predictions for drug discovery}

\end{itemize}

\eject

\subsection*{Past trainees}

My lab trains interdisciplinary scientists to tackle the most challenging problems in drug discovery.

\subsubsection*{Postdocs}

\begin{itemize}
  \item Melissa Boby (8/1/2016--11/8/2023)\\
  %Pharmacology PhD Program\\
  %\emph{Computer-aided drug discovery}
Current position: \emph{Drug Discovery Fellow}, \href{https://www.deshawresearch.com/}{DE Shaw Research}

  \item Ivy Zhang (8/1/2018--1/16/2024)\\
  %Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  %\emph{Scalable free energy calculations for predicting the impact of mutations on protein-protein interactions}
  Current position: \emph{Scientist, Computer Aided Drug Design}, \href{https://www.bms.com/}{Bristol Myers Squibb}
 

  % check departure date
  \item David Schaller, PhD Freie Unviersitat Berlin (7/1/2020--7/31/2022) {\bf (Berlin group)}\\
  %\emph{Scalable alchemical free energy calculations for drug discovery}
  Current position: \emph{Associate Scientist}, \href{https://www.nuvisan.com/}{Nuvisan}

  \item Ana Silveira, PhD Federal University of Rio de Janeiro (8/1/2018--7/31/2019)\\
  %\emph{Integrative modeling of allosteric inhibition; modeling antimicrobial permeation of bacterial porins}
  %Current position: \emph{Investigator}, \href{https://silicontx.com/}{Silicon Therapeutics}
  Current position: \emph{Head of Molecular Simulation}, \href{https://psivant.com/}{Psivant Therapeutics}  

  \item Jiaye Guo, PhD Stony Brook University (10/1/2018--5/31/2021)\\
  %\emph{Design of selective kinase inhibitors and emergence of resistance}
  Current position: \emph{Senior Scientist I}, \href{http://schrodinger.com}{Schrodinger}
  
  \item Simon Boothroyd, PhD Lancaster University (11/1/2018--10/31/2019)\\
  %\emph{Development of advanced biomolecular force fields for drug discovery}
  %Next position: \emph{XtalPi Open Force Field Distinguished Postdoctoral Fellow} with \href{https://www.colorado.edu/lab/shirtsgroup/}{{\bf Michael Shirts}} at the University of Colorado, Boulder
  %Current position: \emph{Science Lead}, \href{http://openforcefield.org}{Open Force Field Initiative}
  Current position: \emph{Senior Researcher, Scientific Programmer}, \href{https://psivant.com/}{Psivant Therapeutics}  
  
  \item William Glass, PhD Southampton (5/1/2020--7/31/2021) \\
  %\emph{Scalable alchemical free energy calculations for predicting the impact of resistance mutations in kinase inhibition}
  Current position: \emph{Molecular Dynamics Research Scientist}, \href{https://www.exscientia.ai/}{Exscientia}

  \item Jaime Rodr\'{i}guez-Guerra Pedregal, PhD Autonomous University of Barcelona (4/1/2019--5/31/2021) {\bf (Berlin group)}\\
  %\emph{Scalable machine learning and alchemical free energy calculations for drug discovery}
  Current position: \emph{Software Engineer}, \href{https://quansight.com/}{Quansight}

  \item Karmen \v{C}ondi\'{c}-Jurki\'{c}, PhD Friedrich Alexander University, Erlangen (4/6/2019--)\\
  %\emph{Biomolecular force fields; open science; interoperability in molecular simulations}
  %Current position: \emph{Project Manager}, \href{http://openforcefield.org}{Open Force Field Initiative}; \emph{Board of Directors}, \href{http://omsf.io}{Open Molecular Software Foundation}
  Current position: \emph{Executive Director and Co-Founder}, \href{http://omsf.io}{Open Molecular Software Foundation}

  \item Hannah Bruce Macdonald, PhD University of Southampton (10/1/2018--9/30/2020)\\
  %\emph{Advanced alchemical free energy methods for drug discovery}
  Current position: \emph{Senior Scientist I}, \href{http://www.merck.com}{Merck Sharp \& Dohme (MSD)}
  
  \item Marcus~Wieder, PhD University of Vienna (9/1/2018--4/16/2020)\\
  %\emph{Modeling tautomer shifts upon binding of small druglike molecules}
  Current position: \emph{Faculty}, \href{https://www.univie.ac.at/en/}{University of Vienna}
  
  \item Levi N. Naden (until 8/10/2018)\\
  Current position: \emph{Software Scientist} at the NSF \href{http://molssi.org}{Molecular Sciences Software Institute (MolSSI)}
  
  \item Sonya M. Hanson (until 6/30/2017)\\
  Next position: \emph{Postdoctoral Researcher with Joachim Frank}, \href{http://franklab.cpmc.columbia.edu/franklab/}{{Columbia University}}\\
  %Current position: Humboldt Research Fellow with \href{https://www.biophys.mpg.de/en/hummer.html}{{\bf Gerhard Hummer}} at MPI Biophysics, Frankfurt
  Current position: Group Leader and Research Scientist, Center for Computational Biology, \\ \href{https://www.simonsfoundation.org/flatiron/}{Flatiron Institute}, Simons Foundation, NYC
 
  \item Gregory Ross (until 6/30/2017)\\
  Current position: \emph{Principal Scientist}, \href{http://www.schrodinger.com}{{ Schr\"{o}dinger}}
  
  \item David W.~H.~Swenson (until 12/30/2015)\\
  %Current position: \emph{Postdoctoral Researcher}, \href{http://www.acmm.nl/molsim/users/swenson/index.html}{{\bf Universiteit von Amsterdam}}
  Current position: \emph{Software Scientist}, \href{http://openfree.energy}{Open Free Energy Consortiun}
  
  \item Daniel L.~Parton (until 8/31/2015)\\
  Current position: \emph{Lead Data Scientist}, \href{http://www.bardess.com/}{{Bardess Group}}
  
  \item Kyle A.~Beauchamp (until 6/12/2015)\\
  Current position: \emph{Senior Director of Data Science}, \href{http://www.tempus.com}{{Tempus}}
  
  \item Jan-Hendrik Prinz (until 1/30/2015)\\
  Current position: \emph{Digital Solutions Architect}, \href{http://www.keylight.de/}{{Keylight GmbH}}
  
  \item Sarah E.~Boyce (until 10/31/2013)\\
  Current position: \emph{Director},  \href{http://www.schrodinger.com}{{Schr\"{o}dinger}}

\end{itemize}

\subsubsection*{Graduate Students}

\begin{itemize}

  \item Yuanqing Wang (7/1/2018--3/13/2022)\\
  Program in Physiology, Biophysics, and Systems Biology (PBSB)\\
  \emph{Free and open source parameterization of protein-small molecule force fields}\\
  Current position: Independent Simons Postdoctoral Fellow, Department of Chemistry, NYU; Schmidt Science Fellow

  \item Talia Kimber (4/1/2019--2/28/2023) [co-supervised with Andrea Volkamer, Charit\'e Berlin] \\
  Charit\'e Universit\"{a}tmedizin (supervisor: Andrea Volkamer)\\
  Freie Universit\"{a}t Berlin, Computer Science PhD program (supervisor: Frank No\'{e})\\
  \emph{Machine learning for optimizing affinity/selectivity and predicting resistance}\\
  Current position: Data Scientist, Swiss Federal Government
  
  % check this
  \item Joshua Fass (9/1/2015----11/9/2020)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
%  \emph{Machine learning and statistical inference for biomolecular dynamics and forcefield parameterization}
  Current position: \emph{Scientist}, \href{https://relaytx.com/}{Relay Therapeutics}
  
  % check this
  \item Mehtap Isik (6/1/2015--9/11/2020 anticipated)\\
  Tri-Institutional Program in Chemical Biology (TPCB)\\
%  \emph{Model experiemental systems for protein-small molecule recognition; targeted nanoparticle design}
  Current position: \emph{Scientist, Computational Chemistry}, \href{https://www.modernatx.com/}{Moderna}
  
  \item Rafal Wiewiora (6/1/2015--\emph{8/31/2020})\\
  Tri-Institutional Training Program in Chemical Biology (TPCB)\\
  %\emph{Design of selective inhibitors for protein methyltransferases; Markov state modeling}
  Current position: \emph{Scientist}, \href{redesignscience.com}{Roivant Sciences}
  
  \item Ari\"{e}n Sebastian Rustenburg (10/1/2013--10/19/2018)\\
  Program in Physiology, Biophysics, and Systems Biology (PBSB)\\
%  \emph{Constant-pH dynamics; protonation states effects in kinase inhibitors via experiment and theory}
  Current position: \emph{DevOps Engineer, Software for Chemistry \& Materials}, \href{https://www.scm.com/}{SCM}

  \item Andrea Rizzi (9/1/2015--12/31/2019)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  %\emph{Quantitatively accurate alchemical free energy calculations for kinase inhibitor design}
  Current position: \emph{Postdoctoral Fellow, Michele Parrinello Group}, \href{https://parrinello.ethz.ch/}{ETH Z\"urich}

  \item Chaya Stern (4/01/2014--6/30/2020)\\
  Tri-Institutional Training Program in Chemical Biology (TPCB)\\
  %\emph{Analysis of single-molecule biophysical experiments; Bayesian inference for forcefield parameterization}
  Current position: \emph{Senior Principal Scientist},  \href{https://odysseytx.com/}{Odyssey Therapeutics}
 
  \item Steven Albanese (6/1/2015--3/31/2019)\\
  Gerstner Graduate Program (GSK)\\
  Current position: \emph{Principal Scientist II},  \href{http://www.schrodinger.com}{{Schr\"{o}dinger}}
  
  \item Patrick B.~Grinaway (7/1/2013--12/31/2018)\\
  Program in Physiology, Biophysics, and Systems Biology (PBSB)\\
  Current position: \emph{Scientist, \href{https://www.onai.com}{Onai}}

  \item Julie M.~Behr (1/10/2014--12/31/2016)\\
  Tri-Institutional Program in Computational Biology and Medicine (CBM)\\
  Current position: \emph{Transferred to Imielinski lab after publishing three papers with the Chodera lab} 

\end{itemize}
  
\subsubsection*{Technicians}

\begin{itemize}

  \item Liza Casella (1/19/2021--11/6/2021)\\
  Current position: \emph{Research Associate}, \href{https://www.kingdomsupercultures.com/}{Kingdom Supercultures}
  
  \item Binisha Karki (8/1/2019--8/28/2020)\\
  %\emph{Scalable biophysical measurements of model protein-ligand systems for advancing computational chemistry}
  Current position: \emph{Research Associate}, \href{http://biontech.de}{BioNTech}

  \item Lucelenie Rodriguez (7/1/2015--12/31/2016; 12/4/2017--7/20/2018)\\
  Current position: \emph{Medical student}, \href{https://med.nyu.edu/education}{NYU School of Medicine}

  \item Erin Grundy (6/1/18--7/31/2019)\\
  Current position: \emph{Graduate student}, \href{https://smhs.gwu.edu/ibs/program/microbio-immunology}{George Washington University}
\end{itemize}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% REFERENCES
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\eject
%
%\section*{References}
%
%\begin{itemize}
%  \item {\bf Ken A. Dill} \\
%  Director, Laufer Center for Physical and Quantitative Biology \\
%  Professor, Physics and Chemistry, Stony Brook University \\
%  email: \href{mailto:dill@laufercenter.org}{dill@laufercenter.org}\\
%  url: \href{http://dillgroup.stonybrook.edu}{http://dillgroup.stonybrook.edu}
%  
%  \item {\bf Cecilia Clementi} \\
%  Professor of Chemistry and Chemical and Biomolecular Engineering, Rice University \\
%  Co-Director, Molecular Sciences Software Institute (MolSSI) \\
%  Senior Scientist, Center for Theoretical Biological Physics \\
%  Einstein Visiting Fellow, Freie Universität Berlin \\
%  email: \href{mailto:cecilia@rice.edu}{cecilia@rice.edu}\\
%  url: \href{http://clementiresearch.rice.edu}{http://clementiresearch.rice.edu}
%
%  \item {\bf Vijay S. Pande} \\
%  Camille and Henry Dreyfus Distinguished Chair in Chemistry; \\
%  Professor of Structural Biology and of Computer Science;  \\
%  General Partner, Andreessen Horowitz \\
%  email: \href{mailto:pande@stanford.edu}{pande@stanford.edu}\\
%  url: \href{https://www.pandelab.org}{https://www.pandelab.org}
%  
%  \item {\bf Susan Marqusee} \\
%  Director of California Institute for Quantitative Biosciences (QB3) \\
%  Eveland Warren Endowed Chair Professor of Biochemistry, Biophysics and Structural Biology \\
%  email: \href{marqusee@berkeley.edu}{marqusee@berkeley.edu} \\
%  url: \href{https://zebra.berkeley.edu}{https://zebra.berkeley.edu}
%
%  \item {\bf Adrian Roitberg} \\
%  Full Professor, Department of Chemistry, University of Florida \\
%  Affiliate Full Professor, Department of Physics, University of Florida \\
%  University of Florida Research Foundation Professor, Department of Chemistry, University of Florida \\
%  email: \href{mailto:roitberg@ufl.edu}{roitberg@ufl.edu} \\
%  url: \href{https://roitberg.chem.ufl.edu}{https://roitberg.chem.ufl.edu}
%  
%  \item {\bf Beno\^{i}t Roux} \\
%  Professor, Department of Biochemistry and Molecular Biophysics \\
%  Department of Pediatrics, Institute of Molecular Pediatric Sciences  \\
%  Committee on Cancer Biology \\
%  email: \href{mailto:roux@uchicago.edu}{roux@uchicago.edu} \\
%  url: \href{http://thallium.bsd.uchicago.edu/RouxLab}{http://thallium.bsd.uchicago.edu/RouxLab}
%
%  \item {\bf Frank No\'{e}} \\
%  Professor for Mathematical Modeling in the Life Sciences, Departments of Mathematics, Physics, and Chemistry, Freie Universität Berlin \\
%  Dean of Studies, Masters program in Computational Sciences, FU Berlin \\
%  Adjunct Professor, Department of Chemistry, Rice University \\
%  email: \href{frank.noe@fu-berlin.de}{frank.noe@fu-berlin.de} \\
%  url: \href{http://www.mi.fu-berlin.de/en/math/groups/comp-mol-bio/index.html}{http://www.mi.fu-berlin.de/en/math/groups/comp-mol-bio/index.html}
%
%  \item Chris Sander (Director, cBio Center, Dana-Farber Cancer Institute) \\
%  email: \href{mailto:chris@sanderlab.org}{chris@sanderlab.org}\\
%  url: \href{http://sanderlab.org}{http://sanderlab.org}
%  
%  \item Gunnar R\"{a}tsch (Head, Biomedical Informatics Group, ETH Z\"{u}rich) \\
%  email: \href{mailto:raetsch@inf.ethz.ch}{raetsch@inf.ethz.ch}\\
%  url: \href{http://ratschlab.org}{http://ratschlab.org}
%  
%\end{itemize}

\end{document}
